{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6fde72a5",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "\n",
    "df = pd.read_pickle('pmid.pkl')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "b28a9338",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0         3155693\n",
       "1         3927294\n",
       "2         3927293\n",
       "3         3927161\n",
       "4         3927160\n",
       "           ...   \n",
       "77815    32513656\n",
       "77816    32512455\n",
       "77817    32512004\n",
       "77818    32607171\n",
       "77819    32962630\n",
       "Name: Id, Length: 77820, dtype: int64"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e8bd5316",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-33-2773d8056fd7>:8: DeprecationWarning: use options instead of chrome_options\n",
      "  chrome= webdriver.Chrome('chromedriver',chrome_options = options)\n"
     ]
    }
   ],
   "source": [
    "from selenium import webdriver\n",
    "from selenium.webdriver.chrome.options import Options\n",
    "from selenium.webdriver.support.ui import Select\n",
    "\n",
    "options= Options()\n",
    "options.add_argument(\"--disable-notifications\")\n",
    "options.add_argument(\"headless\")\n",
    "chrome= webdriver.Chrome('chromedriver',chrome_options = options)\n",
    "\n",
    "content_list=[]\n",
    "for i in range(len(df)):\n",
    "    chrome.get(\"https://www.ncbi.nlm.nih.gov/pmc/articles/pmid/%s\"%(df))\n",
    "\n",
    "    try:        \n",
    "        content = chrome.find_element_by_id('mc')\n",
    "        content_list.append(content)\n",
    "    except:\n",
    "        content_list.append(None)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "12aa4740",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "content_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "d0cebef8",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-22-c2453881f117>:8: DeprecationWarning: use options instead of chrome_options\n",
      "  chrome= webdriver.Chrome('chromedriver',chrome_options = options)\n"
     ]
    }
   ],
   "source": [
    "from selenium import webdriver\n",
    "from selenium.webdriver.chrome.options import Options\n",
    "from selenium.webdriver.support.ui import Select\n",
    "\n",
    "options= Options()\n",
    "options.add_argument(\"--disable-notifications\")\n",
    "options.add_argument(\"headless\")\n",
    "chrome= webdriver.Chrome('chromedriver',chrome_options = options)\n",
    "chrome.get(\"https://www.ncbi.nlm.nih.gov/pmc/articles/pmid/33942874\")\n",
    "content_list=[]\n",
    "content = chrome.find_element_by_id('mc')\n",
    "content_list.append(content)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "a4cc6536",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Database (Oxford). 2021; 2021: baab025.\\nPublished online 2021 May 18. doi: 10.1093/database/baab025\\nPMCID: PMC8094437\\nPMID: 33942874\\nHuman IRES Atlas: an integrative platform for studying IRES-driven translational regulation in humans\\nTzu-Hsien Yang, Chung-Yu Wang, Hsiu-Chun Tsai, and Cheng-Tse Liu\\nAuthor information Article notes Copyright and License information Disclaimer\\nGo to:\\nAbstract\\nIt is now known that cap-independent translation initiation facilitated by internal ribosome entry sites (IRESs) is vital in selective cellular protein synthesis under stress and different physiological conditions. However, three problems make it hard to understand transcriptome-wide cellular IRES-mediated translation initiation mechanisms: (i) complex interplay between IRESs and other translation initiation–related information, (ii) reliability issue of in silico cellular IRES investigation and (iii) labor-intensive in vivo IRES identification. In this research, we constructed the Human IRES Atlas database for a comprehensive understanding of cellular IRESs in humans. First, currently available and suitable IRES prediction tools (IRESfinder, PatSearch and IRESpy) were used to obtain transcriptome-wide human IRESs. Then, we collected eight genres of translation initiation–related features to help study the potential molecular mechanisms of each of the putative IRESs. Three functional tests (conservation, structural RNA–protein scores and conditional translation efficiency) were devised to evaluate the functionality of the identified putative IRESs. Moreover, an easy-to-use interface and an IRES–translation initiation interaction map for each gene transcript were implemented to help understand the interactions between IRESs and translation initiation–related features. Researchers can easily search/browse an IRES of interest using the web interface and deduce testable mechanism hypotheses of human IRES-driven translation initiation based on the integrated results. In summary, Human IRES Atlas integrates putative IRES elements and translation initiation–related experiments for better usage of these data and deduction of mechanism hypotheses.\\nDatabase URL: http://cobishss0.im.nuk.edu.tw/Human_IRES_Atlas/\\nGo to:\\nBackground\\nCells in proliferation adapt their metabolism to environmental stimuli through the regulation of protein synthesis, and translational regulation can provide instant protein level adjustment for handling sudden environmental changes and stress responses (1). Importantly, translation initiation is widely considered to be a vital step in translational regulation (2). For eukaryotes, the complex phase of cellular translation initiation can be categorized into two different types: cap-dependent translation initiation and cap-independent translation initiation (3). Under normal situations, eukaryotic canonical cap-dependent translation initiation is started by the pre-initiation complex formed with initiators tRNA, eIF2, eIF3 and eIF4 to scan for the start codon. However, when facing stressed environments or other physiological challenges, cellular translation can be initiated in a cap-independent manner via the help of the internal ribosome entry site (IRES) elements. IRES-mediated translational initiation was first discovered in poliovirus (4). Since then, scientists have found different IRES elements in other viruses (5) and in cellular genes (6). Cellular IRES elements are now known to be closely related to selective cellular protein synthesis under stress and physiopathological conditions (7) and can have profound effects on human tumor cells (8). Therefore, it is crucial to investigate cellular IRES elements, thus unraveling the nature of cellular stress responses and tumor therapeutic targets.\\nCurrently known cellular IRES elements are mostly found in genes involved in stress, mitosis or apoptosis (9). They usually bare sequence or structural motifs that can recruit the translation machinery to initiate the translation initiation stage (10). Although cellular IRESs generally show lower structural similarities to one another and are less structured than viral IRESs, they can be roughly categorized into three different functional genres with some identifiable patterns (11). First of all, IRES structural forms within 5ʹ untranslated regions (UTRs) can interact with RNA-binding proteins known as IRES trans-acting factors (ITAFs) to remodel their structures, thus opening the hindrance to ribosome recruitment. Second, upstream open reading frames (uORFs) in the neighborhood of an IRES can retain ribosomes in the uORF regions to transform the IRES structure into an active form using amino acid stress. The resulting active conformed IRES then helps presume the cap-independent translation initiation. Third, IRES can modulate different regulation signals around 5ʹ UTR and recruit internal ribosomes accordingly. Some known translational regulation signals such as cap inhibition, uORF translation, ITAFs or RNA G-quadruplexes can trigger the bypassing of canonical cap-dependent translation initiation. These basic known cellular IRES functions provide clues to select the transcriptome-wide IRES sequences in humans.\\nVarious translation initiation–related cellular features are associated with the aforementioned cellular IRES functions. They should be simultaneously considered to elucidate IRES-mediated translational regulation mechanisms. According to the known cellular IRES functions, IRESs reveal complicated interplay with the data of uORFs around IRES sequences, ITAF-binding targets, the IRES structural forms, conservation of the functional sequences and the TIS triplets around IRESs. Therefore, to understand IRES-mediated gene regulation and deduce testable molecular mechanisms, there is an urgent need to articulate the interconnections between IRES elements and these translation initiation–related features. While many of these translation regulation features have been measured and studied, most of the resources are still fragmentary over diverse literature or supplementary repositories. Some of the valuable experimental results require tedious data pre-processing to be used in a comparative analysis. Furthermore, current bicistronic reporter assays for discovering novel IRES elements require series of experiments to rule out the noises from cryptic promoters or spicing activities (12), leading to the time-consuming, small-scale and labor-intensive nature of these experimental approaches. Systematic identification of IRES elements currently resorts mainly to in silico methods to first narrow down the search space. As a consequence, studying IRES in humans requires the following two advances: (i) evaluate the functionality of the in silico predicted putative IRESs (13, 14) and (ii) integrate IRES elements and translation initiation–related information to elucidate and propose testable regulatory mechanisms for each identified IRES.\\nThe complicated connections between IRES elements and translation initiation–related features bring about an immediate demand to construct an integrative IRES database. This database should perform a systematic transcriptome-wide IRES identification and integrate the comprehensive translational regulatory features to help biologists convey researches on IRES-driven translational regulation. In this research, we constructed a database called Human IRES Atlas to accomplish these goals. Human IRES Atlas is designed for easier understanding and evaluating the functionality of IRES candidates. It aims to reduce the long computing time of large-scale IRES screening on the human transcriptome and integrate the translation-related information for better hypothesis generation and evaluation. We first performed comprehensive putative IRES searching on the human transcriptome using currently available IRES prediction tools. To assess the functionality of the putative IRES, we considered their sequence conservation, structural RNA–protein interaction (RPI) scores and conditional translation efficiency. For elucidating the complex connections between IRES elements and translation initiation–related features, we integrated eight types of datasets that probe vital translation initiation information to help biologists infer possible IRES-mediated translational regulatory mechanisms. For analyzing the interconnections between IRES and translation initiation–related features for individual gene transcripts, an easy-to-use interface and interaction maps were constructed. Human IRES Atlas provides the functionality to search for an IRES of interest or browse the human transcriptome for putative IRES lists satisfying the user-customized filtering criteria. In summary, this database is designed in an integrative manner for simultaneous consideration of putative IRESs and translation initiation features. Many critical observations and testable mechanism hypotheses can be retrieved from this integration. Human IRES Atlas is available online at http://cobishss0.im.nuk.edu.tw/Human_IRES_Atlas/.\\nGo to:\\nConstruction and contents\\nConstruction of Human IRES Atlas\\nHuman IRES Atlas deposits systematically identified putative IRES elements and integrates comprehensive translation initiation–related features to assist the study of IRES-driven translational regulation in humans. The overall database contents and construction can be roughly divided into four parts (see Figure 1). First, the transcriptome-wide putative human IRES screening was performed (Figure 1a). Currently available and suitable IRES prediction tools were used to identify the putative IRES elements in the human transcriptome. To evaluate their potential as functional IRESs, three functionality tests were conducted for each of the putative cellular IRES elements: sequence conservation (among different mammalian species), structural RPI scores and conditional translation efficiency (Figure 1b). Users can assess the functionality of the putative IRES for the queried gene based on these computed False Discovery Rate (FDR)-corrected test P-values. Third, eight genres of translation initiation–related datasets were collected and integrated into the database to facilitate biologists to form testable mechanism hypotheses for IRES-driven translational regulation (Figure 1c). In the final stage of database construction, an easy-to-use interface to query and browse the potential IRESs with the associated translation initiation features was constructed (Figure 1d). A transcript detail page that presents the complicated connections between IRES elements and translation initiation features can be found for the selected transcript. Testable functional hypotheses of the putative IRESs identified within the transcript can be inferred from the IRES structure conformation and other probed translation initiation–related features in the detail page. Overall, Human IRES Atlas aims to deposit transcriptome-wide putative human IRESs and articulate the complicated interplay between IRESs and translation initiation features. This integration helps suggest testable molecular mechanism hypotheses for subsequent IRES-mediated translation regulation experiments.\\nFigure 1.\\nOverview of the Human IRES Atlas database.\\nThe construction of Human IRES Atlas can be divided into four different parts. First, the systematic transcriptome-wide IRES identification on all human transcripts was performed based on the evaluated screening tools (part a). The IRES functionality from sequence conservation, structural RPI score and conditional translation efficiency was then estimated for each identified putative IRES (part b). To further provide potential translational regulation information, eight types of datasets were collected and integrated into the repository to assist the deduction of IRES-mediated translational regulation mechanism hypotheses (part c). Finally, an easy-to-use query interface and a transcript detail page with translation initiation interaction maps were implemented to help biologists infer testable hypotheses for subsequent IRES-driven translation initiation research experiments (part d).\\nTranscriptome-wide human IRES screening\\nWe first performed transcriptome-wide IRES element identification on all human 5ʹ UTRs. The human 5ʹ UTR sequences were downloaded from the UCSC Genome Browser database (15) (Dec. 2013 assembly of the human genome, hg38). 53\\u2009971 messenger RNA (mRNA) transcripts for 19\\u2009367 genes were available in total. The sequences before the coding sequence (CDS) regions of each of these transcripts were defined as the 5ʹ UTR sequences. Based on the defined 5ʹ UTR sequences, currently available computational IRES scanning algorithms were evaluated to be used in this research. Five IRES prediction tools [IRESfinder (16), PatSearch (17), RNAz (18), IRESPred (19) and IRESpy (20)] are still currently available. To evaluate these tools, we collected a ground truth dataset with verified IRES sequences as the positive set and random sequences as the negative set. The positive set of 190 known IRES elements is obtained from IRESbase (21) and the review work of Baird et al. (22). Since it is known that house-keeping genes depend mainly on cap-dependent translation initiation (23), we randomly sampled 190 sequences from the 5ʹ UTRs of the house-keeping genes annotated by Eisenberg and Levanon (24) to be the negative set. We forced the negative set to be of the same sequence length distribution as the positive set. Using the receiver operating characteristic curve technique (see Figure 2), we can see that RNAz and IRESPred have performance similar to random guessing, and IRESfinder, IRESpy and PatSearch have area under curve larger than 0.5. Based on this analysis, only IRESpy, PatSearch and IRESfinder are suitable for obtaining potential IRES elements in 5ʹ UTRs. Therefore, IRESpy, PatSearch and IRESfinder were applied in the screening of possible IRESs in 5ʹ UTRs. The thresholds used in the database construction are 0.7 and 0.9 for IRESpy and IRESfinder, respectively.\\nFigure 2.\\nROC curve analysis for currently available IRES prediction tools on 5ʹ UTRs.\\nWe used the ROC curve analysis technique to evaluate the results of different IRES prediction tools. From the ROC curve, RNAz and IRESPred have performance similar to random guessing. IRESfinder, IRESpy and PatSearch have AUC larger than 0.5. Based on this analysis, only IRESpy, PatSearch and IRESfinder are suitable for obtaining potential IRES elements in 5ʹ UTRs.\\nCurrently, most computational IRES identification tools assumed input sequences to be of medium lengths (<174 bps) in their model training process (25). Hence, in this research, we applied the sliding window approach proposed by Zhao et al. (16) to systematically identify transcriptome-wide IRES elements. We first divide the 5ʹ UTR sequence into segments of 174 nts long with a step size of 10 nts between consecutive overlapping segments. Each of these fragments was then fed separately into different IRES identification tools to calculate the probability of the sequence to have a potential functional IRES element. In the last stage of the IRES scanning process, consecutive potential IRES elements identified by the same IRES prediction tool were concatenated to form the final functional IRES elements. 17\\u2009689 IRES sequence segments were identified in human 5ʹ UTRs using these three IRES screening tools.\\nIt is known that the false discovery rate of the prediction process can be reduced if a potential IRES sequence is identified by more than one IRES screening tool (13). Hence to decrease the false positives in IRES screening, we adopted the nested minimal sequence definition to cross-reference the identified IRESs from different screening tools. For a group of nested potential IRES elements, we classified these IRESs into nested sets and took the minimal sequences in the nested sets as the representative IRES sequence segments. Among the 17\\u2009689 IRES elements identified by the three IRES screening tools, 17\\u2009013 minimal IRES elements in 5ʹ UTRs were cross-referenced and deposited in Human IRES Atlas.\\nAlthough most well-known cellular IRESs were found in the 5ʹ UTRs, some IRESs in CDSs and 3ʹ UTRs were also reported to drive the translation of novel proteins or protein isoforms (26–28). IRES screening experiments revealed that 3ʹ UTRs also contained many IRESs (12). Therefore, we have also extended the Human IRES Atlas database to include the potential IRES elements in CDSs (including the main ORF start codon) and 3ʹ UTRs of mRNAs. In the IRES identification beyond 5ʹ UTRs, only IRESpy was used. For PatSearch and IRESfinder, the designed features and the training process were performed on only 5ʹ UTR sequences. Thus, these two tools are not suitable for the IRES identification process on regions other than 5ʹ UTRs. Since no existing tool was trained on Long non-coding RNA (lncRNAs), it is currently unavailable to predict IRESs on lncRNAs and will be carried out when further advances are made in model construction. Extra 1868 IRESs on the CDSs/3ʹ UTRs were included. In total, there are 18\\u2009881 IRESs deposited in Human IRES Atlas.\\nData collection and pre-processing of translation initiation feature data\\nEight genres of translation initiation feature data were gathered and integrated into the database to assist biologists in understanding the interplay between IRES elements and translation initiation. The integration also helps users propose testable molecular mechanism hypotheses for IRES-driven translational regulation. The following contents were collected and integrated in this repository: translation initiation site (TIS) probing experiments, uORF sequences, sequence conservation scores (among mammalian species), ribosome profiling data, structure information of the IRES elements, experimentally identified ITAF target sequences, IRES activity bicistronic measurements and literature-curated IRES verification experiment evidence. Details of the deposited datasets are described in the following subsections.\\nIn Human IRES Atlas, the putative human IRES elements were systematically identified by computational methods. To form reliable IRES-driven translation initiation mechanism hypotheses, we are required to understand the functionality of these putative IRES identifications. Recent studies on cellular IRES have revealed that known human IRES elements are rich in their GC contents to form some special ITAF interacting sequences or structures (11, 22). However, there are still some IRESs that are characterized by their high AU-rich contents (29). These AU-rich IRES elements interact with specific ITAF chaperones that shape the RNA structure into forms that facilitate translation initiation (30). Functional sequences in the genome are usually believed to be conserved among different species (31, 32). Based on the understanding of cellular IRES elements, three functionality tests are also performed with the collected data in this database: conservation, structural RPI scores and conditional translation efficiency between stressed and normal states. Note that the thresholds of the functional tests should be treated as statistical type I error control for the functional aspects. It should not be misinterpreted as the prediction confidence since it is reported that some human IRESs are not conserved in mammals or lack complete structural forms (30).\\nData of sequence conservation between mammalian species\\nFunctional elements are often evolutionary conserved among different closely related species (31, 32). Hence when the specified sequences are to carry out vital cellular functions, they tend to be significantly more conserved among different related species (33). The base-by-base sequence conservation level can be captured by the phastCons conservation scores (34). The phastCons conservation scores were based on multiple alignments of human and other 19 mammalian genomes (baboon, bushbaby, bonobo, chimp, crab-eating macaque, gibbon, golden snub-nosed monkey, gorilla, green monkey, marmoset, mouse lemur, orangutan, proboscis monkey, rhesus, squirrel monkey, tarsier, tree shrew, mouse and dog). We mapped the conservation scores in the genome to transcripts using the transcript mapping annotated by the UCSC Genome Annotation database (15). To test the conservation significance of a putative IRES element, we used the rank-sum test (35) to calculate the P-value against the hypothesis that an IRES element of interest is more conserved than its corresponding transcript in phastCons conservation scores. Multiple hypotheses testing bias was corrected by the FDR control approach for the calculated P-values among all IRES elements.\\nRibosome profiling data and conditional translation efficiency\\nTranslation initiation in eukaryotes proceeds in a series of complicated steps that lead to the final ribosome complex formation. Hence, the ribosome-binding sites located within the 5ʹ UTR sequences under different cellular conditions can reveal important translational regulation hints for active translation programs (36). The experimental procedure called ribosome profiling (ribo-seq) can help identify the transcriptome-wide in vivo ribosome existence and can be used to indicate whether the given sequences show higher translation efficiency in the stress condition. To help users obtain the conditional translation efficiency for a given transcript, we collected the ribosome profiling data of tunicamycin-treated (SRX870808 and SRX870809) and untreated (SRX870804 and SRX870805) HEK293T cells (37). Tunicamycin is a toxin that blocks N-linked glycosylation of Endoplasmic Reticulum (ER)-residing proteins and induces cell depth, leading to ER stress. ER stress induces translational reprogramming that favors alternative translation such as IRES mechanisms. From these ribo-seq experiment results, users can recognize if the transcript is differentially expressed between normal and stress conditions. By simultaneously considering the ITAF data and the ribo-seq results, users can postulate if the stress response of the transcript is mainly regulated by IRES-mediated (38), uORF-mediated (39) or IRES–uORF-interplay (40) translation initiation. To integrate the ribosome profiling results, we first used the NCBI SRA Toolkit to download the sequencing results and then transformed the results into the fastq format. We followed the peak calling procedure used in the HRPDviewer (41) to obtain the transcript nucleotide-level reads for both conditions. First, the adaptor linker sequences or poly-(A) tails of the short reads are trimmed by Cutadapt (42). These trimmed reads are then mapped to the hg38 reference transcriptome using RSEM (43) with the bowtie engine. Finally, the reads per million mapped reads (RPM) values are calculated on all transcript nucleotide positions. By performing the rank-sum test on the RPM values between the stressed and normal conditions, users can get the idea if the identified IRES is stress-induced. Multiple hypotheses testing bias was corrected by the FDR control approach for the calculated P-values among all IRES elements.\\nRPI scores and structural information within IRESs\\nStructural features have been found in several well-studied cellular IRESs (44, 45). Although the structure–function relationship seems to be less rigid in cellular IRESs than in the viral ones, current investigation of known cellular IRESs has indicated that at least some structural modules play important roles with other ITAFs in cellular IRES-mediated translation initiation (11, 30). Hence, studying the potential regulatory IRES structures can help infer the potential functional molecular mechanisms. 12 currently available RNA secondary structure prediction tools [RNAFold (46), RNALfold (47), RNAprob (48), MaxExpect (49), RNAalifold (50), TurboFold (51), SPARSE (52), MXSCARNA (53), aliFreeFold (54), LocaRNA (55), comRNA (56) and RME (57)] were used to comprehensively identify the candidate secondary structures of the putative IRES elements. We further implemented an RPI score to estimate the given IRES’s tendency to interact with proteins. This score is derived from a comparison between the queried sequence and verified RPIs curated in the RNAInter database (58).\\nStructurally similar non-coding RNA (ncRNA) usually carry out similar roles in RNA–protein complexes. Thus, proteins showing interactions with them tend to work in similar biological processes. For estimating the tendency of a given RNA candidate structure to interact with a protein, we resorted to the human ncRNAs having verified RPIs curated in the RNAInter database (58). Specifically, by BLASTing these ncRNAs to the BRAlibaseII structure benchmark dataset (59), we collected a reference set with known structures and their corresponding interacting protein set. To obtain the potential IRES structural forms, the abovementioned 12 different RNA structure prediction tools were integrated in this research to produce candidate structures for a given IRES sequence. For each candidate structure, an ncRNA set consisting of the top 10 structurally similar ncRNAs from the constructed reference set was extracted. The similarity between a candidate structure and a reference ncRNA structure was measured by the general RNA structure distance defined by Jiang et al. (60). The defined general RNA structure distance considers both sequence mismatches and structure pairing mismatches. From the extracted structurally similar ncRNA set of a candidate structure, proteins interacting with these structurally similar ncRNAs were listed based on the data from the RNAInter database. On the listed protein set, we performed the protein interaction prevalence test and the ontology enrichment test as defined in previous researches (61). In brief, the protein interaction prevalence test evaluates if the protein set forms an enriched protein complex, and the ontology enrichment test checks if the protein set has statistically significant annotated functional ontology terms. The enrichment P-values are defined as the RPI scores. A given IRES’s candidate structure with the lowest RPI score among its structure predictions was selected as its representative IRES structure. Lower RPI scores of the predicted structure indicate that the queried sequence is structurally similar to ncRNAs selected from the reference RNA collection and is more likely to interact with a protein complex that regulates a particular cellular function(s). The RPI scores among all putative IRESs in Human IRES Atlas were corrected by the FDR multiple hypothesis control approach.\\nData of ITAF-binding events\\nCellular IRESs in eukaryotes are reported to cooperate with specialized RNA-binding protein called ITAFs in the regulation of the ribosome translation machinery (9, 23). By acting as molecular chaperones or RNA structure remodelers, ITAFs identify particular sequences or structures in IRES elements to help facilitate the translation initiation factor recruitment and initiation complex formation (38). Hence, investigating the connections between ITAF-binding sites and IRES locations can help understand the potential cellular translational regulation mechanisms. Various cellular ITAFs have been reported in humans. Some of their binding targets have been experimentally probed through the transcriptome-wide cross-linking and immunoprecipitation (CLIP) or the photoactivatable-ribonucleoside enhanced cross-linking and immunoprecipitation (PAR-CLIP) techniques at a nucleotide-level resolution. We collected the CLIP and PAR-CLIP-binding events for the following ITAFs from the doRiNA database (62) and remapped them into the hg38 transcriptome: 4\\u2009001\\u2009271 FUS-binding sites, 4899 hnRNPC-binding sites, 30\\u2009593 hnRNPL-binding sites, 621\\u2009116 HuR/ELAVL1-binding sites, 806 p54nrb-binding sites and 1\\u2009367\\u2009825 eIF4A-3-binding sites. Simultaneously considering the predicted IRES elements, the identified ITAF-binding events and the potential IRES structural forms can help users infer possible IRES-mediated translational regulation molecular mechanisms.\\nData of translation initiation site probing data\\nThe translation machinery is completely formed to start polypeptide chain synthesis on special triplets called TISs. Canonical translation initiation uses the AUG triplet as the normal start codon. However, many non-AUG triplets have been shown to provide alternative starting sites for initiating uORF translation or the IRES-driven initiation (63). Hence, the TIS signals around or within IRES elements can provide translation initiation evidence that supports the regulatory effects of these IRES elements. Two different types of TIS data were collected and mapped to human transcript 5ʹ UTRs. The first type that indicates translation initiation is the TIS peaks probed by sequencing techniques. The high-throughput transcriptome-wide TIS positioning for humans is collected from the global translation initiation sequencing (GTI-seq) performed in the work by Lee et al. (64). In their work, two different ribosome E-site translation inhibitors, namely the cycloheximide (CHX) and lactimidomycin (LTM), were used in HEK293 cells. CHX reports elongating ribosomes by blocking translation elongation and LTM probes initiating ribosomes since it preferentially binds to the TIS, while the elongating ribosomes run off the mRNAs. We followed the same analysis procedure used by TISDB (65) to map their sequencing results to the human hg38 transcriptome. In brief, the GTI-seq data were first mapped to the hg38 genome and transcriptome by Tophat (66). The 13th nucleotide of a uniquely mapped read was identified as the start codon recognized in translation initiation. The initiation sites with statistically significant read counts were then determined using the zero-truncated binomial negative model. In total, 11\\u2009395 TIS peaks were identified in the 5ʹ UTRs of the transcripts. These experimentally probed TISs are called experimentally probed translation initiation site (eTIS) in this database. The eTIS experimental contexts suggest either uORF start codons resulting from cap-dependent scanning or cap-independent IRES activity under growth conditions in these particular cells. The second group of translation initiation data is the non-canonical start codon predicted by the PreTIS software (67). PreTIS constructed from a linear regression approach was trained on different ribosome profiling datasets, significant Position Weight Matrix (PWM) and biologically motivated features. Using the PreTIS tool, the potential non-canonical TISs were generated. We adopted a confidence level of 0.9 in this research to obtain 13\\u2009900 predicted non-canonical TIS within the human 5ʹ UTR sequences. The collected non-canonical TIS is called nTIS in this database.\\nData of uORF sequences\\nResearchers have reported that the stalling of ribosomes in the uORF regions can regulate IRES-driven translation initiation (68). uORF translations can mediate 5ʹ UTR structural remodeling to conform the regulated IRES element into the active form (69) or unfold the IRES structure to repress IRES activity (40). Hence, considering the connections between IRES elements and uORFs within 5ʹ UTRs could help indicate the potential regulatory mechanisms of IRESs. We adopted the analysis procedure proposed by Ye et al. (70) to obtain the uORFs in each of the transcripts. uORFs were identified by scanning the 5ʹ UTRs of the transcripts to obtain the sequences that start with the start codon AUG and continue with a three-nucleotide reading frame until the stop codon is encountered. In total, 131\\u2009612 uORFs starting from human 5ʹ UTRs were collected in this database.\\nEvidence of IRES activity\\nRecently, Weingarten-Gabbay et al. (12) devised a systematic bicistronic assay coupled with fluorescence-activated cell sorting to uncover the human transcript sequences with cap-independent translation activity. In their work, they measured the enhanced green fluorescent protein (eGFP) expression scores as the IRES-mediated translation activity in the prepared oligo pEF1 vector library consisting of the virus genome, human genome and reported IRES mutations. They performed the bicistronic experiments for 6946 5ʹ UTRs of 5058 genes in humans, about one-fourth of all human genes. These IRES activity values can indicate the existence of IRES elements for the given sequence segments. Following the experimental design settings of Weingarten-Gabbay et al., the IRES activity was designated as background expression when the oligos were not detected in adjacent eGFP bins during GFP sorting but had >100 reads in the full mRFP-oligos-eGFP library, indicating these background oligos did not convey cap-independent translation signals in the experiments. The oligos were assigned ‘NaN’ if they got <100 reads in the full monomeric red fluorescent protein (mRFP)-oligos-eGFP library, representing a lack of identification confidence. Other normal real numbers represent the extent of IRES activity of the regulatory sequences. When values larger than the background signal are reported for a given identified IRES, subsequent verification experiments to rule out cryptic promoters and splicing activity can be designed to confirm the existence of IRES functionality. In total, we collected and mapped the probed human oligo sequences with IRES activity measurement to 39\\u2009700 transcript segments (E-value\\u2009<1E-6, identity percentage\\u2009>85%, query cover\\u2009>85%, promoter activity\\u2009<0.2 and splicing activity >−2.5) in this database.\\nCollection of literature-curated IRESs\\nSince the first discovery of the viral IRES sequence in poliovirus, various researchers have studied and described diverse cellular IRES elements in the literature. These results have been collected in various review papers. Some databases were also constructed to deposit IRES verification experiments. However, all of these databases either stopped updating the contents or were no longer available to the public. In this research, we collected those literature-verified human IRESs from the IRESite database (71), the IRESbase database (21) and the review works of Baird et al. (22), Leppek et al. (11) and Stoneley et al. (30). These verified IRES sequences were mapped back to the transcripts of the reported IRES-regulating genes by BLASTN (E-value\\u2009<1E-6, identity percentage\\u2009>85% and query cover\\u2009>85%). In total, 659 IRESs with experimental evidence were retrieved from IRESite, IRESbase and the literature.\\nImplementation of the Human IRES Atlas web interface\\nIRES identification and the translation initiation features data pre-processing were carried out using Python. The web interface of the database is constructed using the PHP CodeIgniter MVC framework. IRES data and the related datasets were deposited through MySQL.\\nGo to:\\nUtility and discussion\\nDatabase interface\\nIn Human IRES Atlas, four essential database functions are implemented to help select IRES elements of interest: (i) search function: users can search the putative IRESs for different transcripts of a given gene. (ii) Filter and browse function: users are allowed to browse the deposited transcriptome-wide human IRESs categorized by chromosomes or KEGG pathway annotations of genes regulated by them. The KEGG pathway annotation helps users identify the IRES elements that regulate genes involved in the same biological pathway. This criterion helps biologists check if a particular biological pathway consists of different potential IRES regulation mechanisms. To enable users to select only the putative IRESs with significant q-values for the three functional tests, we have further implemented the q-value filters in the browse page. Users can set the intended q-value threshold(s) for the functional test(s), and the database will show only the potential IRES elements with significant q-values. The resulting IRES list can be downloaded in the browse page. (iii) Transcript detail page: after selecting the transcript of interest in the query results, a detailed translation mechanism deduction page is provided. In this page, the detailed IRES summary and translation initiation–related features are integrated to help understand and infer the testable mechanism hypotheses for each IRES. (iv) Download function: the integrative IRES–translation initiation interaction map and the supplemented experimental datasets can be downloaded as plain text files for subsequent analyses. An easy-to-use web interface was built to facilitate these functions.\\nUsers can type in a particular gene name or its aliases in the search page (Figure 3a). Users can also browse the database to find IRESs categorized by chromosomes and/or KEGG pathway annotations of genes regulated by them (Figure 3b). The putative IRESs and the summary of the corresponding translation initiation–related information are then presented in a tabular form (Figure 3c). For each transcript of the input gene, Human IRES Atlas summaries the brief transcript information, the putative IRES elements found on the transcripts, the number of prediction tools that specify this sequence to be a potential IRES, the functionality test results for each identified IRES element and finally the summaries of other translation initiation–related data observed within IRES regions. Three tests (conservation significance, RPI score significance and conditional translation efficiency significance) are provided to assess the functionality of each identified IRES. The summary of the translation initiation–related data matched to an IRES region includes the following information: (i) the number of eTIS (experimentally probed TIS) found in the upstream to the downstream 100 bps of the IRES element, (ii) the number of nTIS (non-canonical TIS) predicted in the upstream to the downstream 100 bps of the IRES element, (iii) the number of uORF observed in the upstream and downstream 100 bps of the IRES element, (iv) the number of ITAF-binding events that overlap and interact with the IRES element, (v) IRES activity measured for the IRES element and (vi) the number of literature IRES evidence that overlaps with the IRES prediction. Some IRES elements have been shown to use the cis uORFs and TISs to mediate their activity (45, 69). The amino acid stress or the stalling ribosome near an IRES can trigger a structural switch of the IRES to an active conformation, leading to IRES-mediated translation initiation. Therefore, we included extra 100 bps upstream and downstream of an IRES when associating TISs and uORFs to it. For ITAFs, they usually bind and remodel IRES structures to set up the ribosome-binding platform (38).\\nFigure 3.\\nThe search result page for the identified IRESs of a given gene.\\n(a) The search mode: users can type in the gene of interest to search for the identified IRESs on the given gene transcripts. (b) The browse mode: users can filter out the IRESs categorized by their residing chromosomes, annotated KEGG pathways of the regulated genes and the thresholds of IRES functionality q-values. (c) The result table for the identified IRES elements in all transcript isoforms of the queried gene. In this table, three IRES functionality test results and other translational regulatory features for each identified IRES element are summarized. (d) The link to the detail page of the selected transcript.\\nBelow the summary table of the translation initiation–related features, an interaction map between translation initiation–related features for the transcript can be viewed in this table for better data visualization (Figure 3c). For each transcript, one can click the ‘detail’ hyperlink (Figure 3d) to obtain the detail page that lists the complete information of the putative IRES elements, the visualized IRES–translation initiation interaction maps and the translational regulation information at a single nucleotide resolution. Both the genomic chromosomal coordinates and the transcript locations for different translation initiation–related information are listed in the detail page. Moreover, we have also provided the .bed files for the putative IRES track and different transcription-related information tracks. These files can facilitate users to integrate the application with the genome browser. In the last part, the implemented transcript detail page assists in forming molecular mechanism hypotheses for IRES-driven translation initiation. The walkthrough of the detail page is provided along with the case study.\\nCase study\\nHuman IRES Atlas aims to integrate the transcriptome-wide human IRESs and translational regulatory features. The integration provides ways to infer testable mechanism hypotheses for IRES-mediated translation regulation. One can use the IRES–translation initiation interaction map and the queried transcript’s detailed information to deduce such hypotheses. We discuss the IRES-mediated regulatory program of human FGF9 in detail to demonstrate the biological applicability of the constructed database. Human Fibroblast Growth Factor 9 (FGF9) is the gene that transcribes and translates to a protein that acts as an autocrine/paracrine growth factor in lung branching morphogenesis (72), bone development (73) and even cancer progression (74). It has been reported that the FGF9 mRNA is expressed in embryos while the FGF9 protein is identified to be of low level in most cells and is restrictedly expressed in specific cell organs (75). Hence, it is of interest to biologists that how the FGF9 transcripts are translationally regulated in human cells. In Human IRES Atlas, the FGF9 transcript map of IRES–translation initiation interactions provides clues for deducing the testable mechanism hypotheses. There is an identified overlapping IRES region (IRES2: 746.840) that was found by at least two IRES screening tools and is calculated to be conserved and significant in functional RPI scores (q\\u2009=\\u20092.83E-13*) (Figure 4a and b). In the FGF9 IRES–translation initiation interaction map, very low levels of TIS and ribosome occupancy signals were found (Figure 4c), suggesting the low translation level in normal conditions. Further, uORFs were discovered in proximity to these structurally significant IRES elements. Finally, HuR-binding signals were observed to be overlapped with the putative IRES. Details of the translation initiation–related data found around each IRES element are listed in the fourth part of the page (Figure 4d). A link to download the text files in the .csv format that contain the full information of the detail page is provided for users (Figure 4e).\\nFigure 4.\\nThe transcript detail page showing the potential IRES-driven translation regulatory program of the given transcript.\\n(a) The brief information table of the queried gene transcript. (b) The translation initiation–related feature summary table for the identified IRES elements within the queried gene transcript. (c) The IRES–translation initiation interaction map for the queried gene transcript. (d) The detail of the translation initiation–related feature data found for each IRES. (e) The download function to obtain the .csv or .bed formats of all information in the detail page.\\nBy careful consideration of the integrated information, we can deduce that FGF9 transcripts could be translationally regulated through the interplay between the structural IRES, uORF segments and ITAFs (see Figure 5). By the effort of molecular biologists, it is now verified that there is indeed a functional IRES structure near the end of the 5ʹ UTR of the FGF9 transcript. The uORF segments found around the IRES elements were shown to maintain the low protein expression level (40). The IRES structure induced under hypoxia recruits the ribosome to the CDS and activates the FGF9 protein synthesis with IRES–ITAF interactions. Although it is later found that the ITAF HuR may bind to the AU-rich elements in FGF9 transcripts (76), further studies and experiments should be carried out to figure out whether other ITAFs interact with these IRES elements. These findings conform to the hypotheses suggested by the constructed database. In conclusion, translation regulatory programs can be inferred with the facilitation of Human IRES Atlas.\\nFigure 5.\\nHuman IRES Atlas helps suggest testable IRES-mediated translational regulation mechanism hypotheses.\\nFor the FGF9 5ʹ UTR, the IRES–translation initiation interaction map suggests that (1) the transcript is of low translation level and (2) some regulatory program between the RPI interpretable IRESs, the nearby uORFs and some ITAFs may exist. From experimental literature evidence, we know that the uORF represses FGF9 protein synthesis in normal conditions. Some ITAFs bind to the IRES to start cap-independent translation under hypoxia, verifying the testable hypotheses suggested by the constructed database.\\nComparison with related works\\nHuman IRES Atlas was created for two primary purposes: (i) to provide comprehensive and systematic IRES identifications in the human species and (ii) to integrate IRES elements and translation initiation–related information for biologists to understand the interconnections between them. By constructing a comprehensive IRES–translation initiation interaction map, biologists can get testable hypotheses on cellular IRES elements and devise subsequent verification experiments. Currently, four existing databases and datasets are devoted to similar tasks as Human IRES Atlas and thus are compared in the following discussion.\\nTwo IRES databases [IRESite (71) and IRESbase (21)] were designed to curate and deposit IRES verification experiments from the literature. Researchers have carried out different bicistronic experiments for studying IRES activity of certain genes. Nevertheless, most of the information is fragmentary in different literature papers. Therefore, these two databases tried to gather and store these valuable records. While these two works collected valuable IRES experimental data, till now there are still few cellular IRES elements that have been experimentally verified and deposited. Only 26 human mRNA transcripts were evaluated to contain IRES elements in IRESite and 112 human IRESs were collected in IRESbase. Potential IRES-driven translational regulation programs in the human transcriptome cannot be easily deduced through these two databases due to the lack of an integrative platform for understanding the interactions between IRES elements and the related translation initiation features. Since there are only few available verified IRES elements in the literature, there is still a need to carry out systematic in silico IRES identification and construct a comprehensive IRES–translation initiation interaction map to complement the sparsity of IRES verification experiments.\\nAnother related work called IRES Omnibus (77) studied human IRESs through systematic IRES prediction. In the preliminary work of IRES Omnibus, Chen and Tseng set up an in-house IRES prediction system based on the PatSearch tool to screen through human 5ʹ UTR sequences for potential IRES elements. In their work, IREZone, the special regions consisting of contiguous overlapping RNA structures, and IRES elements were predicted and collected. Using the IRES and IREZone repository, novel sequence patterns and IRES-related cellular functions were investigated. However, in their preliminary identification, merely one IRES prediction tool was used, and the complicated connections between IRES elements and translation initiation–related features were ignored. Hence, the constructed preliminary human IRES map may suffer from high false positives. Further, it would be beneficial for biologists to deduce the potential IRES-related regulatory molecular mechanisms if different high throughput translation initiation–related datasets can be integrated into a user-friendly platform.\\nThe study by Weingarten-Gabbay et al. (12) is another touchstone in the field. We then evaluated the IRES identification sensitivity and specificity for Human IRES Atlas and the work of Weingarten-Gabbay et al. on a ground truth IRES dataset consisting of verified IRES sequences as the positive set and random sequences as the negative set. The positive set of 190 known IRES elements is obtained from the IRESbase (21) and the review work of Baird et al. (22). Since it is known that house-keeping genes depend mainly on cap-dependent translation initiation (23), we randomly sampled 190 sequences from the 5ʹ UTRs of annotated house-keeping genes (24) to be the negative set. An identified IRES in a dataset is defined to be a true positive if >20% of the base pairs overlap with some sequence in the positive set. A sequence from the negative set is called a true negative for one identification method if no identified sequence has >20% overlapping base pairs with it. On this ground truth IRES set, the results of Weingarten-Gabbay et al. obtain a sensitivity of 0.19 and a false-positive rate (FPR) of 0.037. The putative IRESs deposited in Human IRES Atlas have a sensitivity of 0.32 and an FPR of 0.226 on the ground truth dataset. We can see from these results that Human IRES Atlas achieves a better sensitivity with a small sacrifice of the FPR. Furthermore, if we merge the potential IRESs in Human IRES Atlas and the results of Weingarten-Gabbay et al., a sensitivity of 0.43 and an FPR of 0.258 are observed. The near linearly independent sensitivity gain means that the two results are mostly complementary. By examining the data of Weingarten-Gabbay et al., 19% of the literature-curated IRESs were found. Thirty-two percent of the literature-curated IRESs could be determined in Human IRES Atlas. While oversampling 5ʹ UTRs can lead to higher chances of overlapping with the true positive set, it also increases the possibility of overlapping the putative IRESs with the true negative set, raising the FPR. There is a compromise between the sensitivity and the FPR when oversampling 5ʹ UTRs. The putative IRES elements deposited in Human IRES Atlas are estimated to have a higher sensitivity than the corresponding FPR. Hence, Human IRES Atlas provides extra insights for IRES-mediated translation regulation in humans.\\nHuman IRES Atlas is the first database explicitly designed to overcome current obstacles in studying human IRES elements. It also fulfills the necessity for deducing testable molecular mechanism hypotheses. In Human IRES Atlas, three different IRES prediction tools (IRESFinder, PatSearch and IRESpy) were used for systematic IRES screening. The results of different tools were collected by selecting the nested consensus minimal sequences among different prediction algorithms. Hence the prediction uncertainty and false discovery rate inherent in different tools were minimized during the identification process. To further help evaluate the functionality of the putative IRESs, three tests (conservation, structural RPI scores and conditional translation efficiency) were designed and performed. Besides performing the transcriptome-wide human IRES identification, we also integrated eight categories of translation initiation–related datasets. Biologists can further understand the IRES-driven translation regulation and develop testable molecular mechanism hypotheses for subsequent research through the translation initiation–related features. In summary, Human IRES Atlas integrates more data and provides more functionality than existing tools. It specifically aims to perform systematic IRES identification and help deduce the IRES-driven cellular translation regulation mechanisms.\\nIssues related to the constructed database\\nThe Human IRES Atlas database is constructed by depositing systematically identified in silico IRESs and various translation initiation–related data. Based on the eight gathered categories of translation initiation–related features, we can assess the significance of the putative IRESs and deduce potential IRES-driven translation regulation mechanism hypotheses. However, several issues may exist when interpreting the results generated from the systems-wide data integration process. Researchers usually perturbed cells and conducted distinct high-throughput experiments for gathering genome-wide information (78). However, different high-throughput experiments may be performed under different environmental conditions, cellular states or cell types. This experimental discrepancy causes inevitable systematic bias in the data integration process (79). Hence, some detailed subtleties should be considered when designing subsequent verification experiments based on the hypotheses formed from the systems view of data integration.\\nDiverse statistical analysis strategies were applied to the high-throughput experiment datasets. For example, we adopted the eTIS data by thresholding the positional read-count fold change and some other minimum detection requirement between samples treated with the ribosome E-site translation inhibitors CHX and LTM (64). The high-throughput screening of IRES activity adopted in this database was probed using fluorescence measurements (12). Different statistical analysis strategies provide different false discovery rates. In general, noises resulted from the data statistical analysis were assumed to be orthogonal in different high-throughput experimental results (35). Hence, by simultaneously combining experimental data of similar targeted systems, the false positive issue can be reduced. In Human IRES Atlas, we have gathered diverse translation initiation–related data and used different screening tools in the systematic putative IRES identification. This cross-referencing screening reduces the impact of false positives. For instance, TISs based on CHX/LTM experimental analysis and PreTIS prediction were both collected to lessen the false positive impact in TIS location information. We will keep Human IRES Atlas up-to-date with newly published datasets when more data or methods are released.\\nData integration in multicellular organisms often brings about the issue of cellular systems bias. In metazoa species, correct temporal and spatial gene expression of the same genome is the core control on cell differentiation (13). While different combinations of cell types in data integration may provide distinct false positives, the multicellular systems problem should be taken care of in using the constructed database. IRES activity data were probed in vitro based on experiments on sequence libraries. Non-canonical TIS and uORF information were identified in silico by investigating comparative genomics and sequence contents. These data reveal the normal status of the translation program. To assist in cell type-specific or condition-specific IRES verification experimental design, one can refer to ribosome profiling experiments conducted under different conditions/cell types or refer to special cellular conditions where the ITAF-binding events were observed. Diverse ribosome profiling sequencing results are fragmentary in the literature. Recently, a database called HPRDviewer (41) is constructed to deposit comprehensive human ribosome profiling datasets. In Human IRES Atlas, the conditional ribosome profiling results performed under the stressed and normal conditions are shown. More condition-specific human ribosome profiling information can be found in HPRDviewer. Through consulting more detailed ribosome profiling experiments or the ITAF-binding event conditions, one can further minimize the impact of cell type bias inherent in the data integration process.\\nGo to:\\nConclusions\\nIn this research, a novel database called Human IRES Atlas was constructed to deposit the systematic in silico identified IRESs from the human transcriptome. Besides the transcriptome-wide putative IRES identification, eight genres of translation initiation–related datasets were integrated and three novel IRES functionality tests were designed. To facilitate the understanding of the connections between IRES elements and these translation initiation–related features, an easy-to-use interface for the integrated results was implemented for biologists to infer possible molecular mechanisms of human IRES-driven translation initiation. Researchers can easily search the IRES of interest or browse the IRESs on genes involved in specific biological pathways with the database’s user-friendly web-interface. By simultaneously studying the human IRES and the translation initiation–related features, the database can provide testable molecular mechanism hypotheses on human gene translation regulation. We believe that Human IRES Atlas can help biologists better study mechanisms of cellular IRES-driven translation initiation, IRES-related cancer progression and other IRES-related genetic disorders.\\nGo to:\\nAcknowledgements\\nThe authors would like to thank Ting-Hsuan Chen and Yu-Cian Lin for helping with the preliminary data collection.\\nGo to:\\nContributor Information\\nTzu-Hsien Yang, Department of Information Management, National University of Kaohsiung, 700, Kaohsiung University Rd., Nanzih District, Kaohsiung 811, Taiwan.\\nChung-Yu Wang, Department of Information Management, National University of Kaohsiung, 700, Kaohsiung University Rd., Nanzih District, Kaohsiung 811, Taiwan.\\nHsiu-Chun Tsai, Department of Information Management, National University of Kaohsiung, 700, Kaohsiung University Rd., Nanzih District, Kaohsiung 811, Taiwan.\\nCheng-Tse Liu, Department of Information Management, National University of Kaohsiung, 700, Kaohsiung University Rd., Nanzih District, Kaohsiung 811, Taiwan.\\nGo to:\\nFunding\\nThis work is funded by National University of Kaohsiung and Ministry of Science and Technology of Taiwan (MOST 107-2218-E-390-009-MY3).\\nGo to:\\nConflict of interest.\\nThe authors declare that they have no competing interests.\\nGo to:\\nReferences\\n1. Gebauer F., Hentze M.W. (2004) Molecular mechanisms of translational control. Nat. Rev. Mol. Cell Biol., 5, nrm1488. [PMC free article] [PubMed] [Google Scholar]\\n2. Sonenberg N., Hinnebusch A.G. (2009) Regulation of translation initiation in eukaryotes: mechanisms and biological targets. Cell, 136, 731–745. [PMC free article] [PubMed] [Google Scholar]\\n3. Johnson A.G., Grosely R., Petrov A.N.. et al. (2017) Dynamics of IRES-mediated translation. Philos. Trans. R. Soc. B Biol. Sci., 372, 20160177. [PMC free article] [PubMed] [Google Scholar]\\n4. Pelletier J., Sonenberg N. (1988) Internal initiation of translation of eukaryotic mRNA directed by a sequence derived from poliovirus RNA. Nature, 334, 320. [PubMed] [Google Scholar]\\n5. Lukavsky P.J. (2009) Structure and function of HCV IRES domains. Virus Res., 139, 166–171. [PMC free article] [PubMed] [Google Scholar]\\n6. Ray P.S., Grover R., Das S. (2006) Two internal ribosome entry sites mediate the translation of p53 isoforms. EMBO Rep., 7, 404–410. [PMC free article] [PubMed] [Google Scholar]\\n7. Sharathchandra A., Katoch A., Das S. (2014) IRES mediated translational regulation of p53 isoforms. Wiley Interdiscip. Rev. RNA, 5, 131–139. [PubMed] [Google Scholar]\\n8. Sharma D.K., Bressler K., Patel H.. et al. (2016) Role of eukaryotic initiation factors during cellular stress and cancer progression. J. Nucleic Acids, 2016, 8235121. [PMC free article] [PubMed] [Google Scholar]\\n9. Holcik M., Sonenberg N. (2005) Translational control in stress and apoptosis. Nat. Rev. Mol. Cell Biol., 6, 318–327. [PubMed] [Google Scholar]\\n10. Spriggs K.A., Stoneley M., Bushell M.. et al. (2008) Re-programming of translation following cell stress allows IRES-mediated translation to predominate. Biol. Cell, 100, 27–38. [PubMed] [Google Scholar]\\n11. Leppek K., Das R., Barna M. (2018) Functional 5′UTR mRNA structures in eukaryotic translation regulation and how to find them. Nat. Rev. Mol. Cell Biol., 19, 158. [PMC free article] [PubMed] [Google Scholar]\\n12. Weingarten-Gabbay S., Elias-Kirma S., Nir R. et al. (2016) Systematic discovery of cap-independent translation sequences in human and viral genomes. Science, 351, aad4939. [PubMed] [Google Scholar]\\n13. Yang T.-H., Wang -C.-C., Hung P.-C.. et al. (2014) cis MEP: an integrated repository of genomic epigenetic profiles and cis-regulatory modules in Drosophila. BMC Syst. Biol., 8, S8. [PMC free article] [PubMed] [Google Scholar]\\n14. Yang T.-H., Wang -C.-C., Wang Y.-C.. et al. (2014) YTRP: a repository for yeast transcriptional regulatory pathways. Database, 2014, bau014. [PMC free article] [PubMed] [Google Scholar]\\n15. Haeussler M., Zweig A.S., Tyner C.. et al. (2019) The UCSC genome browser database: 2019 update. Nucleic Acids Res., 47, D853–D858. [PMC free article] [PubMed] [Google Scholar]\\n16. Zhao J., Wu J., Xu T.. et al. (2018) IRESfinder: identifying RNA internal ribosome entry site in eukaryotic cell using framed k-mer features. J. Genet. Genomics, 45, 403–406. [PubMed] [Google Scholar]\\n17. Grillo G., Licciulli F., Liuni S.. et al. (2003) PatSearch: a program for the detection of patterns and structural motifs in nucleotide sequences. Nucleic Acids Res., 31, 3608–3612. [PMC free article] [PubMed] [Google Scholar]\\n18. Washietl S., Hofacker I.L., Stadler P.F. (2005) Fast and reliable prediction of noncoding RNAs. Proc. Natl. Acad. Sci. USA, 102, 2454–2459. [PMC free article] [PubMed] [Google Scholar]\\n19. Kolekar P., Pataskar A., Kulkarni-Kale U.. et al. (2016) IRESPred: web server for prediction of cellular and viral internal ribosome entry site (IRES). Sci. Rep., 6, 27436. [PMC free article] [PubMed] [Google Scholar]\\n20. Wang J., Gribskov M. (2019) IRESpy: an XGBoost model for prediction of internal ribosome entry sites. BMC Bioinform., 20, 409. [PMC free article] [PubMed] [Google Scholar]\\n21. Zhao J., Li Y., Wang C.. et al. (2020) IRESbase: a comprehensive database of experimentally validated internal ribosome entry sites. Genomics Proteomics Bioinf, 18, 129–139. [PMC free article] [PubMed] [Google Scholar]\\n22. Baird S.D., Turcotte M., Korneluk R.G.. et al. (2006) Searching for IRES. RNA, 12, 1755–1785. [PMC free article] [PubMed] [Google Scholar]\\n23. Komar A.A., Hatzoglou M. (2011) Cellular IRES-mediated translation: the war of ITAFs in pathophysiological states. Cell Cycle, 10, 229–240. [PMC free article] [PubMed] [Google Scholar]\\n24. Eisenberg E., Levanon E.Y. (2013) Human housekeeping genes, revisited. Trends Genet., 29, 569–574. [PubMed] [Google Scholar]\\n25. Gritsenko A.A., Weingarten-Gabbay S., Elias-Kirma S.. et al. (2017) Sequence features of viral and human Internal Ribosome Entry Sites predictive of their activity. PLoS Comput. Biol., 13, e1005734. [PMC free article] [PubMed] [Google Scholar]\\n26. Du X., Wang J., Zhu H.. et al. (2013) Second cistron in CACNA1A gene encodes a transcription factor mediating cerebellar development and SCA6. Cell, 154, 118–133. [PMC free article] [PubMed] [Google Scholar]\\n27. Cornelis S., Bruynooghe Y., Denecker G.. et al. (2000) Identification and characterization of a novel cell cycle–regulated internal ribosome entry site. Mol. Cell, 5, 597–605. [PubMed] [Google Scholar]\\n28. Candeias M.M., Powell D.J., Roubalova E.. et al. (2006) Expression of p53 and p53/47 are controlled by alternative mechanisms of messenger RNA translation initiation. Oncogene, 25, 6936–6947. [PubMed] [Google Scholar]\\n29. Faye M.D., Graber T.E., Liu P.. et al. (2013) Nucleotide composition of cellular internal ribosome entry sites defines dependence on NF45 and predicts a posttranscriptional mitotic regulon. Mol. Cell. Biol., 33, 307–318. [PMC free article] [PubMed] [Google Scholar]\\n30. Stoneley M., Willis A.E. (2004) Cellular internal ribosome entry segments: structures, trans-acting factors and regulation of gene expression. Oncogene, 23, 3200. [PubMed] [Google Scholar]\\n31. Wan Y., Kertesz M., Spitale R.C.. et al. (2011) Understanding the transcriptome through RNA structure. Nat. Rev. Genet., 12, 641–655. [PMC free article] [PubMed] [Google Scholar]\\n32. Yang T.-H. (2019) Transcription factor regulatory modules provide the molecular mechanisms for functional redundancy observed among transcription factors in yeast. BMC Bioinform., 20, 1–16. [PMC free article] [PubMed] [Google Scholar]\\n33. Siepel A., Bejerano G., Pedersen J.S.. et al. (2005) Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. Genome Res., 15, 1034–1050. [PMC free article] [PubMed] [Google Scholar]\\n34. Margulies E.H., Blanchette M., Haussler D.et al. , N. C. S. Program . (2003) Identification and characterization of multi-species conserved sequences. Genome Res., 13, 2507–2518. [PMC free article] [PubMed] [Google Scholar]\\n35. Yang T.-H., Wu W.-S. (2013) Inferring functional transcription factor-gene binding pairs by integrating transcription factor binding data with transcription factor knockout data. BMC Syst. Biol., 7, S13. [PMC free article] [PubMed] [Google Scholar]\\n36. Lee K.-M., Chen C.-J., Shih S.-R. (2017) Regulation mechanisms of viral IRES-driven translation. Trends Microbiol., 25, 546–561. [PubMed] [Google Scholar]\\n37. Sidrauski C., McGeachy A.M., Ingolia N.T.. et al. (2015) The small molecule ISRIB reverses the effects of eIF2α phosphorylation on translation and stress granule assembly. Elife, 4, e05033. [PMC free article] [PubMed] [Google Scholar]\\n38. Mitchell S.A., Spriggs K.A., Coldwell M.J.. et al. (2003) The Apaf-1 internal ribosome entry segment attains the correct structural conformation for function via interactions with PTB and unr. Mol. Cell, 11, 757–771. [PubMed] [Google Scholar]\\n39. Zhang H., Wang Y., Lu J. (2019) Function and evolution of upstream ORFs in eukaryotes. Trends Biochem. Sci., 44, 782–794. [PubMed] [Google Scholar]\\n40. Chen T.-M., Shih Y.-H., Tseng J. T.. et al. (2014) Overexpression of FGF9 in colon cancer cells is mediated by hypoxia-induced translational activation. Nucleic Acids Res., 42, 2932–2944. [PMC free article] [PubMed] [Google Scholar]\\n41. Wu W.-S., Jiang Y.-X., Chang J.-W.. et al. (2018) HRPDviewer: human ribosome profiling data viewer. Database, 2018, bay074. [PMC free article] [PubMed] [Google Scholar]\\n42. Martin M. (2011) Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal, 17, 10. [Google Scholar]\\n43. Li B., Dewey C.N. (2011) RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome. BMC Bioinform., 12, 1–16. [PMC free article] [PubMed] [Google Scholar]\\n44. Le Quesne J.P.C., Stoneley M., Fraser G.A.. et al. (2001) Derivation of a structural model for the c-myc IRES. J. Mol. Biol., 310, 111–126. [PubMed] [Google Scholar]\\n45. Yaman I., Fernandez J., Liu H.. et al. (2003) The zipper model of translational control: a small upstream ORF is the switch that controls structural remodeling of an mRNA leader. Cell, 113, 519–531. [PubMed] [Google Scholar]\\n46. Lorenz R., Bernhart S.H., Zu Siederdissen C.H.. et al. (2011) ViennaRNA package 2.0. Algorithms Mol. Biol., 6, 26. [PMC free article] [PubMed] [Google Scholar]\\n47. Hofacker I.L., Priwitzer B. and Stadler P.F. (2004) Prediction of locally stable RNA secondary structures for genome-wide surveys. Bioinformatics, 20, 186–190. [PubMed] [Google Scholar]\\n48. Deng F., Ledda M., Vaziri S.. et al. (2016) Data-directed RNA secondary structure prediction using probabilistic modeling. RNA, 22, 1109–1109. [PMC free article] [PubMed] [Google Scholar]\\n49. Lu Z.J., Gloor J.W., Mathews D.H. (2009) Improved RNA secondary structure prediction by maximizing expected pair accuracy. RNA, 15, 1805–1813. [PMC free article] [PubMed] [Google Scholar]\\n50. Bernhart S.H., Hofacker I.L., Will S.. et al. (2008) RNAalifold: improved consensus structure prediction for RNA alignments. BMC Bioinform., 9, 474. [PMC free article] [PubMed] [Google Scholar]\\n51. Reuter J.S., Mathews D.H. (2010) RNAstructure: software for RNA secondary structure prediction and analysis. BMC Bioinform., 11, 129. [PMC free article] [PubMed] [Google Scholar]\\n52. Will S., Otto C., Miladi M.. et al. (2015) SPARSE: quadratic time simultaneous alignment and folding of RNAs without sequence-based heuristics. Bioinformatics, 31, 2489–2496. [PMC free article] [PubMed] [Google Scholar]\\n53. Tabei Y., Kiryu H., Kin T.. et al. (2008) A fast structural multiple alignment method for long RNA sequences. BMC Bioinform., 9, 33. [PMC free article] [PubMed] [Google Scholar]\\n54. Glouzon J.-P.S. and Ouangraoua A. (2018) aliFreeFold: an alignment-free approach to predict secondary structure from homologous RNA sequences. Bioinformatics, 34, i70–i78. [PMC free article] [PubMed] [Google Scholar]\\n55. Will S., Joshi T., Hofacker I.L.. et al. (2012) LocARNA-P: accurate boundary prediction and improved detection of structural RNAs. RNA, 18, 900–914. [PMC free article] [PubMed] [Google Scholar]\\n56. Ji Y., Xu X., Stormo G.D. (2004) A graph theoretical approach for predicting common RNA secondary structure motifs including pseudoknots in unaligned sequences. Bioinformatics, 20, 1591–1602. [PubMed] [Google Scholar]\\n57. Wu Y., Shi B., Ding X.. et al. (2015) Improved prediction of RNA secondary structure by integrating the free energy model with restraints derived from experimental probing data. Nucleic Acids Res., 43, 7247–7259. [PMC free article] [PubMed] [Google Scholar]\\n58. Lin Y., Liu T., Cui T.. et al. (2020) RNAInter in 2020: RNA interactome repository with increased coverage and annotation. Nucleic Acids Res., 48, D189–D197. [PMC free article] [PubMed] [Google Scholar]\\n59. Gardner P.P., Wilm A., Washietl S. (2005) A benchmark of multiple sequence alignment programs upon structural RNAs. Nucleic Acids Res., 33, 2433–2439. [PMC free article] [PubMed] [Google Scholar]\\n60. Jiang T., Lin G., Ma B.. et al. (2002) A general edit distance between RNA structures. J. Comput. Biol., 9, 371–388. [PubMed] [Google Scholar]\\n61. Yang T.H., Wu W.S. (2012) Identifying biologically interpretable transcription factor knockout targets by jointly analyzing the transcription factor knockout microarray and the ChIP-chip data. BMC Syst. Biol., 6, 102. [PMC free article] [PubMed] [Google Scholar]\\n62. Blin K., Dieterich C., Wurmus R.. et al. (2015) DoRiNA 2.0—upgrading the doRiNA database of RNA interactions in post-transcriptional regulation. Nucleic Acids Res., 43, D160–D167. [PMC free article] [PubMed] [Google Scholar]\\n63. Touriol C., Bornes S., Bonnal S.. et al. (2003) Generation of protein isoform diversity by alternative initiation of translation at non-AUG codons. Biol. Cell, 95, 169–178. [PubMed] [Google Scholar]\\n64. Lee S., Liu B., Lee S.. et al. (2012) Global mapping of translation initiation sites in mammalian cells at single-nucleotide resolution. Proc. Natl. Acad. Sci. USA, 109, E2424–E2432. [PMC free article] [PubMed] [Google Scholar]\\n65. Wan J. and Qian S.-B. (2013) TISdb: a database for alternative translation initiation in mammalian cells. Nucleic Acids Res., 42, D845–D850. [PMC free article] [PubMed] [Google Scholar]\\n66. Langmead B., Trapnell C., Pop M.. et al. (2009) Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biol., 10, R25. [PMC free article] [PubMed] [Google Scholar]\\n67. Reuter K., Biehl A., Koch L.. et al. (2016) PreTIS: a tool to predict non-canonical 5’UTR translational initiation sites in human and mouse. PLoS Comput. Biol., 12, e1005170. [PMC free article] [PubMed] [Google Scholar]\\n68. Somers J., Pöyry T. and Willis A.E. (2013) A perspective on mammalian upstream open reading frame function. Int. J. Biochem. Cell Biol., 45, 1690–1700. [PMC free article] [PubMed] [Google Scholar]\\n69. Fernandez J., Yaman I., Huang C.. et al. (2005) Ribosome stalling regulates IRES-mediated translation in eukaryotes, a parallel to prokaryotic attenuation. Mol. Cell, 17, 405–416. [PubMed] [Google Scholar]\\n70. Ye Y., Liang Y., Yu Q.. et al. (2015) Analysis of human upstream open reading frames and impact on gene expression. Hum. Genet., 134, 605–612. [PubMed] [Google Scholar]\\n71. Mokrejš M., Mašek T., Vopálenský V.. et al. (2009) IRESite—a tool for the examination of viral and cellular internal ribosome entry sites. Nucleic Acids Res., 38, D131–D136. [PMC free article] [PubMed] [Google Scholar]\\n72. Colvin J.S., Green R.P., Schmahl J.. et al. (2001) Male-to-female sex reversal in mice lacking fibroblast growth factor 9. Cell, 104, 875–889. [PubMed] [Google Scholar]\\n73. Robinson D., Hasharoni A., Evron Z.. et al. (2000) Synovial chondromatosis: the possible role of FGF 9 and FGF receptor 3 in its pathology. Int. J. Exp. Pathol., 81, 183–189. [PMC free article] [PubMed] [Google Scholar]\\n74. Arai D., Hegab A.E., Soejima K.. et al. (2015) Characterization of the cell of origin and propagation potential of the fibroblast growth factor 9-induced mouse model of lung adenocarcinoma. J. Pathol., 235, 593–605. [PMC free article] [PubMed] [Google Scholar]\\n75. Colvin J.S., Feldman B., Nadeau J.H.. et al. (1999) Genomic organization and embryonic expression of the mouse fibroblast growth factor 9 gene. Dev. Dyn. An Off. Publ. Am. Assoc. Anat., 216, 72–88. [PubMed] [Google Scholar]\\n76. Chen T.-M., Hsu C.-H., Tsai S.-J.. et al. (2010) AUF1 p42 isoform selectively controls both steady-state and PGE2-induced FGF9 mRNA decay. Nucleic Acids Res., 38, 8061–8071. [PMC free article] [PubMed] [Google Scholar]\\n77. Chen W.-M. (2009) IRES Omnibus: An Integrated Web Server for Predicting and Searching the Human Genes Bearing the IRES Elements. National Cheng Kung University, Tainan. [Google Scholar]\\n78. Ideker T., Galitski T., Hood L. (2001) A new approach to decoding life: systems biology. Annu. Rev. Genomics Hum. Genet., 2, 343–372. [PubMed] [Google Scholar]\\n79. Hwang D., Rust A.G., Ramsey S.. et al. (2005) A data integration methodology for systems biology. Proc. Natl. Acad. Sci. USA, 102, 17296–17301. [PMC free article] [PubMed] [Google Scholar]\\nArticles from Database: The Journal of Biological Databases and Curation are provided here courtesy of Oxford University Press'"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "content_list[0].text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "6271b0fb",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Database (Oxford). 2021; 2021: baab025.\\nPublished online 2021 May 18. doi: 10.1093/database/baab025\\nPMCID: PMC8094437\\nPMID: 33942874\\nHuman IRES Atlas: an integrative platform for studying IRES-driven translational regulation in humans\\nTzu-Hsien Yang, Chung-Yu Wang, Hsiu-Chun Tsai, and Cheng-Tse Liu\\nAuthor information Article notes Copyright and License information Disclaimer\\nGo to:\\nAbstract\\nIt is now known that cap-independent translation initiation facilitated by internal ribosome entry sites (IRESs) is vital in selective cellular protein synthesis under stress and different physiological conditions. However, three problems make it hard to understand transcriptome-wide cellular IRES-mediated translation initiation mechanisms: (i) complex interplay between IRESs and other translation initiation–related information, (ii) reliability issue of in silico cellular IRES investigation and (iii) labor-intensive in vivo IRES identification. In this research, we constructed the Human IRES Atlas database for a comprehensive understanding of cellular IRESs in humans. First, currently available and suitable IRES prediction tools (IRESfinder, PatSearch and IRESpy) were used to obtain transcriptome-wide human IRESs. Then, we collected eight genres of translation initiation–related features to help study the potential molecular mechanisms of each of the putative IRESs. Three functional tests (conservation, structural RNA–protein scores and conditional translation efficiency) were devised to evaluate the functionality of the identified putative IRESs. Moreover, an easy-to-use interface and an IRES–translation initiation interaction map for each gene transcript were implemented to help understand the interactions between IRESs and translation initiation–related features. Researchers can easily search/browse an IRES of interest using the web interface and deduce testable mechanism hypotheses of human IRES-driven translation initiation based on the integrated results. In summary, Human IRES Atlas integrates putative IRES elements and translation initiation–related experiments for better usage of these data and deduction of mechanism hypotheses.\\nDatabase URL: http://cobishss0.im.nuk.edu.tw/Human_IRES_Atlas/\\nGo to:\\nBackground\\nCells in proliferation adapt their metabolism to environmental stimuli through the regulation of protein synthesis, and translational regulation can provide instant protein level adjustment for handling sudden environmental changes and stress responses (1). Importantly, translation initiation is widely considered to be a vital step in translational regulation (2). For eukaryotes, the complex phase of cellular translation initiation can be categorized into two different types: cap-dependent translation initiation and cap-independent translation initiation (3). Under normal situations, eukaryotic canonical cap-dependent translation initiation is started by the pre-initiation complex formed with initiators tRNA, eIF2, eIF3 and eIF4 to scan for the start codon. However, when facing stressed environments or other physiological challenges, cellular translation can be initiated in a cap-independent manner via the help of the internal ribosome entry site (IRES) elements. IRES-mediated translational initiation was first discovered in poliovirus (4). Since then, scientists have found different IRES elements in other viruses (5) and in cellular genes (6). Cellular IRES elements are now known to be closely related to selective cellular protein synthesis under stress and physiopathological conditions (7) and can have profound effects on human tumor cells (8). Therefore, it is crucial to investigate cellular IRES elements, thus unraveling the nature of cellular stress responses and tumor therapeutic targets.\\nCurrently known cellular IRES elements are mostly found in genes involved in stress, mitosis or apoptosis (9). They usually bare sequence or structural motifs that can recruit the translation machinery to initiate the translation initiation stage (10). Although cellular IRESs generally show lower structural similarities to one another and are less structured than viral IRESs, they can be roughly categorized into three different functional genres with some identifiable patterns (11). First of all, IRES structural forms within 5ʹ untranslated regions (UTRs) can interact with RNA-binding proteins known as IRES trans-acting factors (ITAFs) to remodel their structures, thus opening the hindrance to ribosome recruitment. Second, upstream open reading frames (uORFs) in the neighborhood of an IRES can retain ribosomes in the uORF regions to transform the IRES structure into an active form using amino acid stress. The resulting active conformed IRES then helps presume the cap-independent translation initiation. Third, IRES can modulate different regulation signals around 5ʹ UTR and recruit internal ribosomes accordingly. Some known translational regulation signals such as cap inhibition, uORF translation, ITAFs or RNA G-quadruplexes can trigger the bypassing of canonical cap-dependent translation initiation. These basic known cellular IRES functions provide clues to select the transcriptome-wide IRES sequences in humans.\\nVarious translation initiation–related cellular features are associated with the aforementioned cellular IRES functions. They should be simultaneously considered to elucidate IRES-mediated translational regulation mechanisms. According to the known cellular IRES functions, IRESs reveal complicated interplay with the data of uORFs around IRES sequences, ITAF-binding targets, the IRES structural forms, conservation of the functional sequences and the TIS triplets around IRESs. Therefore, to understand IRES-mediated gene regulation and deduce testable molecular mechanisms, there is an urgent need to articulate the interconnections between IRES elements and these translation initiation–related features. While many of these translation regulation features have been measured and studied, most of the resources are still fragmentary over diverse literature or supplementary repositories. Some of the valuable experimental results require tedious data pre-processing to be used in a comparative analysis. Furthermore, current bicistronic reporter assays for discovering novel IRES elements require series of experiments to rule out the noises from cryptic promoters or spicing activities (12), leading to the time-consuming, small-scale and labor-intensive nature of these experimental approaches. Systematic identification of IRES elements currently resorts mainly to in silico methods to first narrow down the search space. As a consequence, studying IRES in humans requires the following two advances: (i) evaluate the functionality of the in silico predicted putative IRESs (13, 14) and (ii) integrate IRES elements and translation initiation–related information to elucidate and propose testable regulatory mechanisms for each identified IRES.\\nThe complicated connections between IRES elements and translation initiation–related features bring about an immediate demand to construct an integrative IRES database. This database should perform a systematic transcriptome-wide IRES identification and integrate the comprehensive translational regulatory features to help biologists convey researches on IRES-driven translational regulation. In this research, we constructed a database called Human IRES Atlas to accomplish these goals. Human IRES Atlas is designed for easier understanding and evaluating the functionality of IRES candidates. It aims to reduce the long computing time of large-scale IRES screening on the human transcriptome and integrate the translation-related information for better hypothesis generation and evaluation. We first performed comprehensive putative IRES searching on the human transcriptome using currently available IRES prediction tools. To assess the functionality of the putative IRES, we considered their sequence conservation, structural RNA–protein interaction (RPI) scores and conditional translation efficiency. For elucidating the complex connections between IRES elements and translation initiation–related features, we integrated eight types of datasets that probe vital translation initiation information to help biologists infer possible IRES-mediated translational regulatory mechanisms. For analyzing the interconnections between IRES and translation initiation–related features for individual gene transcripts, an easy-to-use interface and interaction maps were constructed. Human IRES Atlas provides the functionality to search for an IRES of interest or browse the human transcriptome for putative IRES lists satisfying the user-customized filtering criteria. In summary, this database is designed in an integrative manner for simultaneous consideration of putative IRESs and translation initiation features. Many critical observations and testable mechanism hypotheses can be retrieved from this integration. Human IRES Atlas is available online at http://cobishss0.im.nuk.edu.tw/Human_IRES_Atlas/.\\nGo to:\\nConstruction and contents\\nConstruction of Human IRES Atlas\\nHuman IRES Atlas deposits systematically identified putative IRES elements and integrates comprehensive translation initiation–related features to assist the study of IRES-driven translational regulation in humans. The overall database contents and construction can be roughly divided into four parts (see Figure 1). First, the transcriptome-wide putative human IRES screening was performed (Figure 1a). Currently available and suitable IRES prediction tools were used to identify the putative IRES elements in the human transcriptome. To evaluate their potential as functional IRESs, three functionality tests were conducted for each of the putative cellular IRES elements: sequence conservation (among different mammalian species), structural RPI scores and conditional translation efficiency (Figure 1b). Users can assess the functionality of the putative IRES for the queried gene based on these computed False Discovery Rate (FDR)-corrected test P-values. Third, eight genres of translation initiation–related datasets were collected and integrated into the database to facilitate biologists to form testable mechanism hypotheses for IRES-driven translational regulation (Figure 1c). In the final stage of database construction, an easy-to-use interface to query and browse the potential IRESs with the associated translation initiation features was constructed (Figure 1d). A transcript detail page that presents the complicated connections between IRES elements and translation initiation features can be found for the selected transcript. Testable functional hypotheses of the putative IRESs identified within the transcript can be inferred from the IRES structure conformation and other probed translation initiation–related features in the detail page. Overall, Human IRES Atlas aims to deposit transcriptome-wide putative human IRESs and articulate the complicated interplay between IRESs and translation initiation features. This integration helps suggest testable molecular mechanism hypotheses for subsequent IRES-mediated translation regulation experiments.\\nFigure 1.\\nOverview of the Human IRES Atlas database.\\nThe construction of Human IRES Atlas can be divided into four different parts. First, the systematic transcriptome-wide IRES identification on all human transcripts was performed based on the evaluated screening tools (part a). The IRES functionality from sequence conservation, structural RPI score and conditional translation efficiency was then estimated for each identified putative IRES (part b). To further provide potential translational regulation information, eight types of datasets were collected and integrated into the repository to assist the deduction of IRES-mediated translational regulation mechanism hypotheses (part c). Finally, an easy-to-use query interface and a transcript detail page with translation initiation interaction maps were implemented to help biologists infer testable hypotheses for subsequent IRES-driven translation initiation research experiments (part d).\\nTranscriptome-wide human IRES screening\\nWe first performed transcriptome-wide IRES element identification on all human 5ʹ UTRs. The human 5ʹ UTR sequences were downloaded from the UCSC Genome Browser database (15) (Dec. 2013 assembly of the human genome, hg38). 53\\u2009971 messenger RNA (mRNA) transcripts for 19\\u2009367 genes were available in total. The sequences before the coding sequence (CDS) regions of each of these transcripts were defined as the 5ʹ UTR sequences. Based on the defined 5ʹ UTR sequences, currently available computational IRES scanning algorithms were evaluated to be used in this research. Five IRES prediction tools [IRESfinder (16), PatSearch (17), RNAz (18), IRESPred (19) and IRESpy (20)] are still currently available. To evaluate these tools, we collected a ground truth dataset with verified IRES sequences as the positive set and random sequences as the negative set. The positive set of 190 known IRES elements is obtained from IRESbase (21) and the review work of Baird et al. (22). Since it is known that house-keeping genes depend mainly on cap-dependent translation initiation (23), we randomly sampled 190 sequences from the 5ʹ UTRs of the house-keeping genes annotated by Eisenberg and Levanon (24) to be the negative set. We forced the negative set to be of the same sequence length distribution as the positive set. Using the receiver operating characteristic curve technique (see Figure 2), we can see that RNAz and IRESPred have performance similar to random guessing, and IRESfinder, IRESpy and PatSearch have area under curve larger than 0.5. Based on this analysis, only IRESpy, PatSearch and IRESfinder are suitable for obtaining potential IRES elements in 5ʹ UTRs. Therefore, IRESpy, PatSearch and IRESfinder were applied in the screening of possible IRESs in 5ʹ UTRs. The thresholds used in the database construction are 0.7 and 0.9 for IRESpy and IRESfinder, respectively.\\nFigure 2.\\nROC curve analysis for currently available IRES prediction tools on 5ʹ UTRs.\\nWe used the ROC curve analysis technique to evaluate the results of different IRES prediction tools. From the ROC curve, RNAz and IRESPred have performance similar to random guessing. IRESfinder, IRESpy and PatSearch have AUC larger than 0.5. Based on this analysis, only IRESpy, PatSearch and IRESfinder are suitable for obtaining potential IRES elements in 5ʹ UTRs.\\nCurrently, most computational IRES identification tools assumed input sequences to be of medium lengths (<174 bps) in their model training process (25). Hence, in this research, we applied the sliding window approach proposed by Zhao et al. (16) to systematically identify transcriptome-wide IRES elements. We first divide the 5ʹ UTR sequence into segments of 174 nts long with a step size of 10 nts between consecutive overlapping segments. Each of these fragments was then fed separately into different IRES identification tools to calculate the probability of the sequence to have a potential functional IRES element. In the last stage of the IRES scanning process, consecutive potential IRES elements identified by the same IRES prediction tool were concatenated to form the final functional IRES elements. 17\\u2009689 IRES sequence segments were identified in human 5ʹ UTRs using these three IRES screening tools.\\nIt is known that the false discovery rate of the prediction process can be reduced if a potential IRES sequence is identified by more than one IRES screening tool (13). Hence to decrease the false positives in IRES screening, we adopted the nested minimal sequence definition to cross-reference the identified IRESs from different screening tools. For a group of nested potential IRES elements, we classified these IRESs into nested sets and took the minimal sequences in the nested sets as the representative IRES sequence segments. Among the 17\\u2009689 IRES elements identified by the three IRES screening tools, 17\\u2009013 minimal IRES elements in 5ʹ UTRs were cross-referenced and deposited in Human IRES Atlas.\\nAlthough most well-known cellular IRESs were found in the 5ʹ UTRs, some IRESs in CDSs and 3ʹ UTRs were also reported to drive the translation of novel proteins or protein isoforms (26–28). IRES screening experiments revealed that 3ʹ UTRs also contained many IRESs (12). Therefore, we have also extended the Human IRES Atlas database to include the potential IRES elements in CDSs (including the main ORF start codon) and 3ʹ UTRs of mRNAs. In the IRES identification beyond 5ʹ UTRs, only IRESpy was used. For PatSearch and IRESfinder, the designed features and the training process were performed on only 5ʹ UTR sequences. Thus, these two tools are not suitable for the IRES identification process on regions other than 5ʹ UTRs. Since no existing tool was trained on Long non-coding RNA (lncRNAs), it is currently unavailable to predict IRESs on lncRNAs and will be carried out when further advances are made in model construction. Extra 1868 IRESs on the CDSs/3ʹ UTRs were included. In total, there are 18\\u2009881 IRESs deposited in Human IRES Atlas.\\nData collection and pre-processing of translation initiation feature data\\nEight genres of translation initiation feature data were gathered and integrated into the database to assist biologists in understanding the interplay between IRES elements and translation initiation. The integration also helps users propose testable molecular mechanism hypotheses for IRES-driven translational regulation. The following contents were collected and integrated in this repository: translation initiation site (TIS) probing experiments, uORF sequences, sequence conservation scores (among mammalian species), ribosome profiling data, structure information of the IRES elements, experimentally identified ITAF target sequences, IRES activity bicistronic measurements and literature-curated IRES verification experiment evidence. Details of the deposited datasets are described in the following subsections.\\nIn Human IRES Atlas, the putative human IRES elements were systematically identified by computational methods. To form reliable IRES-driven translation initiation mechanism hypotheses, we are required to understand the functionality of these putative IRES identifications. Recent studies on cellular IRES have revealed that known human IRES elements are rich in their GC contents to form some special ITAF interacting sequences or structures (11, 22). However, there are still some IRESs that are characterized by their high AU-rich contents (29). These AU-rich IRES elements interact with specific ITAF chaperones that shape the RNA structure into forms that facilitate translation initiation (30). Functional sequences in the genome are usually believed to be conserved among different species (31, 32). Based on the understanding of cellular IRES elements, three functionality tests are also performed with the collected data in this database: conservation, structural RPI scores and conditional translation efficiency between stressed and normal states. Note that the thresholds of the functional tests should be treated as statistical type I error control for the functional aspects. It should not be misinterpreted as the prediction confidence since it is reported that some human IRESs are not conserved in mammals or lack complete structural forms (30).\\nData of sequence conservation between mammalian species\\nFunctional elements are often evolutionary conserved among different closely related species (31, 32). Hence when the specified sequences are to carry out vital cellular functions, they tend to be significantly more conserved among different related species (33). The base-by-base sequence conservation level can be captured by the phastCons conservation scores (34). The phastCons conservation scores were based on multiple alignments of human and other 19 mammalian genomes (baboon, bushbaby, bonobo, chimp, crab-eating macaque, gibbon, golden snub-nosed monkey, gorilla, green monkey, marmoset, mouse lemur, orangutan, proboscis monkey, rhesus, squirrel monkey, tarsier, tree shrew, mouse and dog). We mapped the conservation scores in the genome to transcripts using the transcript mapping annotated by the UCSC Genome Annotation database (15). To test the conservation significance of a putative IRES element, we used the rank-sum test (35) to calculate the P-value against the hypothesis that an IRES element of interest is more conserved than its corresponding transcript in phastCons conservation scores. Multiple hypotheses testing bias was corrected by the FDR control approach for the calculated P-values among all IRES elements.\\nRibosome profiling data and conditional translation efficiency\\nTranslation initiation in eukaryotes proceeds in a series of complicated steps that lead to the final ribosome complex formation. Hence, the ribosome-binding sites located within the 5ʹ UTR sequences under different cellular conditions can reveal important translational regulation hints for active translation programs (36). The experimental procedure called ribosome profiling (ribo-seq) can help identify the transcriptome-wide in vivo ribosome existence and can be used to indicate whether the given sequences show higher translation efficiency in the stress condition. To help users obtain the conditional translation efficiency for a given transcript, we collected the ribosome profiling data of tunicamycin-treated (SRX870808 and SRX870809) and untreated (SRX870804 and SRX870805) HEK293T cells (37). Tunicamycin is a toxin that blocks N-linked glycosylation of Endoplasmic Reticulum (ER)-residing proteins and induces cell depth, leading to ER stress. ER stress induces translational reprogramming that favors alternative translation such as IRES mechanisms. From these ribo-seq experiment results, users can recognize if the transcript is differentially expressed between normal and stress conditions. By simultaneously considering the ITAF data and the ribo-seq results, users can postulate if the stress response of the transcript is mainly regulated by IRES-mediated (38), uORF-mediated (39) or IRES–uORF-interplay (40) translation initiation. To integrate the ribosome profiling results, we first used the NCBI SRA Toolkit to download the sequencing results and then transformed the results into the fastq format. We followed the peak calling procedure used in the HRPDviewer (41) to obtain the transcript nucleotide-level reads for both conditions. First, the adaptor linker sequences or poly-(A) tails of the short reads are trimmed by Cutadapt (42). These trimmed reads are then mapped to the hg38 reference transcriptome using RSEM (43) with the bowtie engine. Finally, the reads per million mapped reads (RPM) values are calculated on all transcript nucleotide positions. By performing the rank-sum test on the RPM values between the stressed and normal conditions, users can get the idea if the identified IRES is stress-induced. Multiple hypotheses testing bias was corrected by the FDR control approach for the calculated P-values among all IRES elements.\\nRPI scores and structural information within IRESs\\nStructural features have been found in several well-studied cellular IRESs (44, 45). Although the structure–function relationship seems to be less rigid in cellular IRESs than in the viral ones, current investigation of known cellular IRESs has indicated that at least some structural modules play important roles with other ITAFs in cellular IRES-mediated translation initiation (11, 30). Hence, studying the potential regulatory IRES structures can help infer the potential functional molecular mechanisms. 12 currently available RNA secondary structure prediction tools [RNAFold (46), RNALfold (47), RNAprob (48), MaxExpect (49), RNAalifold (50), TurboFold (51), SPARSE (52), MXSCARNA (53), aliFreeFold (54), LocaRNA (55), comRNA (56) and RME (57)] were used to comprehensively identify the candidate secondary structures of the putative IRES elements. We further implemented an RPI score to estimate the given IRES’s tendency to interact with proteins. This score is derived from a comparison between the queried sequence and verified RPIs curated in the RNAInter database (58).\\nStructurally similar non-coding RNA (ncRNA) usually carry out similar roles in RNA–protein complexes. Thus, proteins showing interactions with them tend to work in similar biological processes. For estimating the tendency of a given RNA candidate structure to interact with a protein, we resorted to the human ncRNAs having verified RPIs curated in the RNAInter database (58). Specifically, by BLASTing these ncRNAs to the BRAlibaseII structure benchmark dataset (59), we collected a reference set with known structures and their corresponding interacting protein set. To obtain the potential IRES structural forms, the abovementioned 12 different RNA structure prediction tools were integrated in this research to produce candidate structures for a given IRES sequence. For each candidate structure, an ncRNA set consisting of the top 10 structurally similar ncRNAs from the constructed reference set was extracted. The similarity between a candidate structure and a reference ncRNA structure was measured by the general RNA structure distance defined by Jiang et al. (60). The defined general RNA structure distance considers both sequence mismatches and structure pairing mismatches. From the extracted structurally similar ncRNA set of a candidate structure, proteins interacting with these structurally similar ncRNAs were listed based on the data from the RNAInter database. On the listed protein set, we performed the protein interaction prevalence test and the ontology enrichment test as defined in previous researches (61). In brief, the protein interaction prevalence test evaluates if the protein set forms an enriched protein complex, and the ontology enrichment test checks if the protein set has statistically significant annotated functional ontology terms. The enrichment P-values are defined as the RPI scores. A given IRES’s candidate structure with the lowest RPI score among its structure predictions was selected as its representative IRES structure. Lower RPI scores of the predicted structure indicate that the queried sequence is structurally similar to ncRNAs selected from the reference RNA collection and is more likely to interact with a protein complex that regulates a particular cellular function(s). The RPI scores among all putative IRESs in Human IRES Atlas were corrected by the FDR multiple hypothesis control approach.\\nData of ITAF-binding events\\nCellular IRESs in eukaryotes are reported to cooperate with specialized RNA-binding protein called ITAFs in the regulation of the ribosome translation machinery (9, 23). By acting as molecular chaperones or RNA structure remodelers, ITAFs identify particular sequences or structures in IRES elements to help facilitate the translation initiation factor recruitment and initiation complex formation (38). Hence, investigating the connections between ITAF-binding sites and IRES locations can help understand the potential cellular translational regulation mechanisms. Various cellular ITAFs have been reported in humans. Some of their binding targets have been experimentally probed through the transcriptome-wide cross-linking and immunoprecipitation (CLIP) or the photoactivatable-ribonucleoside enhanced cross-linking and immunoprecipitation (PAR-CLIP) techniques at a nucleotide-level resolution. We collected the CLIP and PAR-CLIP-binding events for the following ITAFs from the doRiNA database (62) and remapped them into the hg38 transcriptome: 4\\u2009001\\u2009271 FUS-binding sites, 4899 hnRNPC-binding sites, 30\\u2009593 hnRNPL-binding sites, 621\\u2009116 HuR/ELAVL1-binding sites, 806 p54nrb-binding sites and 1\\u2009367\\u2009825 eIF4A-3-binding sites. Simultaneously considering the predicted IRES elements, the identified ITAF-binding events and the potential IRES structural forms can help users infer possible IRES-mediated translational regulation molecular mechanisms.\\nData of translation initiation site probing data\\nThe translation machinery is completely formed to start polypeptide chain synthesis on special triplets called TISs. Canonical translation initiation uses the AUG triplet as the normal start codon. However, many non-AUG triplets have been shown to provide alternative starting sites for initiating uORF translation or the IRES-driven initiation (63). Hence, the TIS signals around or within IRES elements can provide translation initiation evidence that supports the regulatory effects of these IRES elements. Two different types of TIS data were collected and mapped to human transcript 5ʹ UTRs. The first type that indicates translation initiation is the TIS peaks probed by sequencing techniques. The high-throughput transcriptome-wide TIS positioning for humans is collected from the global translation initiation sequencing (GTI-seq) performed in the work by Lee et al. (64). In their work, two different ribosome E-site translation inhibitors, namely the cycloheximide (CHX) and lactimidomycin (LTM), were used in HEK293 cells. CHX reports elongating ribosomes by blocking translation elongation and LTM probes initiating ribosomes since it preferentially binds to the TIS, while the elongating ribosomes run off the mRNAs. We followed the same analysis procedure used by TISDB (65) to map their sequencing results to the human hg38 transcriptome. In brief, the GTI-seq data were first mapped to the hg38 genome and transcriptome by Tophat (66). The 13th nucleotide of a uniquely mapped read was identified as the start codon recognized in translation initiation. The initiation sites with statistically significant read counts were then determined using the zero-truncated binomial negative model. In total, 11\\u2009395 TIS peaks were identified in the 5ʹ UTRs of the transcripts. These experimentally probed TISs are called experimentally probed translation initiation site (eTIS) in this database. The eTIS experimental contexts suggest either uORF start codons resulting from cap-dependent scanning or cap-independent IRES activity under growth conditions in these particular cells. The second group of translation initiation data is the non-canonical start codon predicted by the PreTIS software (67). PreTIS constructed from a linear regression approach was trained on different ribosome profiling datasets, significant Position Weight Matrix (PWM) and biologically motivated features. Using the PreTIS tool, the potential non-canonical TISs were generated. We adopted a confidence level of 0.9 in this research to obtain 13\\u2009900 predicted non-canonical TIS within the human 5ʹ UTR sequences. The collected non-canonical TIS is called nTIS in this database.\\nData of uORF sequences\\nResearchers have reported that the stalling of ribosomes in the uORF regions can regulate IRES-driven translation initiation (68). uORF translations can mediate 5ʹ UTR structural remodeling to conform the regulated IRES element into the active form (69) or unfold the IRES structure to repress IRES activity (40). Hence, considering the connections between IRES elements and uORFs within 5ʹ UTRs could help indicate the potential regulatory mechanisms of IRESs. We adopted the analysis procedure proposed by Ye et al. (70) to obtain the uORFs in each of the transcripts. uORFs were identified by scanning the 5ʹ UTRs of the transcripts to obtain the sequences that start with the start codon AUG and continue with a three-nucleotide reading frame until the stop codon is encountered. In total, 131\\u2009612 uORFs starting from human 5ʹ UTRs were collected in this database.\\nEvidence of IRES activity\\nRecently, Weingarten-Gabbay et al. (12) devised a systematic bicistronic assay coupled with fluorescence-activated cell sorting to uncover the human transcript sequences with cap-independent translation activity. In their work, they measured the enhanced green fluorescent protein (eGFP) expression scores as the IRES-mediated translation activity in the prepared oligo pEF1 vector library consisting of the virus genome, human genome and reported IRES mutations. They performed the bicistronic experiments for 6946 5ʹ UTRs of 5058 genes in humans, about one-fourth of all human genes. These IRES activity values can indicate the existence of IRES elements for the given sequence segments. Following the experimental design settings of Weingarten-Gabbay et al., the IRES activity was designated as background expression when the oligos were not detected in adjacent eGFP bins during GFP sorting but had >100 reads in the full mRFP-oligos-eGFP library, indicating these background oligos did not convey cap-independent translation signals in the experiments. The oligos were assigned ‘NaN’ if they got <100 reads in the full monomeric red fluorescent protein (mRFP)-oligos-eGFP library, representing a lack of identification confidence. Other normal real numbers represent the extent of IRES activity of the regulatory sequences. When values larger than the background signal are reported for a given identified IRES, subsequent verification experiments to rule out cryptic promoters and splicing activity can be designed to confirm the existence of IRES functionality. In total, we collected and mapped the probed human oligo sequences with IRES activity measurement to 39\\u2009700 transcript segments (E-value\\u2009<1E-6, identity percentage\\u2009>85%, query cover\\u2009>85%, promoter activity\\u2009<0.2 and splicing activity >−2.5) in this database.\\nCollection of literature-curated IRESs\\nSince the first discovery of the viral IRES sequence in poliovirus, various researchers have studied and described diverse cellular IRES elements in the literature. These results have been collected in various review papers. Some databases were also constructed to deposit IRES verification experiments. However, all of these databases either stopped updating the contents or were no longer available to the public. In this research, we collected those literature-verified human IRESs from the IRESite database (71), the IRESbase database (21) and the review works of Baird et al. (22), Leppek et al. (11) and Stoneley et al. (30). These verified IRES sequences were mapped back to the transcripts of the reported IRES-regulating genes by BLASTN (E-value\\u2009<1E-6, identity percentage\\u2009>85% and query cover\\u2009>85%). In total, 659 IRESs with experimental evidence were retrieved from IRESite, IRESbase and the literature.\\nImplementation of the Human IRES Atlas web interface\\nIRES identification and the translation initiation features data pre-processing were carried out using Python. The web interface of the database is constructed using the PHP CodeIgniter MVC framework. IRES data and the related datasets were deposited through MySQL.\\nGo to:\\nUtility and discussion\\nDatabase interface\\nIn Human IRES Atlas, four essential database functions are implemented to help select IRES elements of interest: (i) search function: users can search the putative IRESs for different transcripts of a given gene. (ii) Filter and browse function: users are allowed to browse the deposited transcriptome-wide human IRESs categorized by chromosomes or KEGG pathway annotations of genes regulated by them. The KEGG pathway annotation helps users identify the IRES elements that regulate genes involved in the same biological pathway. This criterion helps biologists check if a particular biological pathway consists of different potential IRES regulation mechanisms. To enable users to select only the putative IRESs with significant q-values for the three functional tests, we have further implemented the q-value filters in the browse page. Users can set the intended q-value threshold(s) for the functional test(s), and the database will show only the potential IRES elements with significant q-values. The resulting IRES list can be downloaded in the browse page. (iii) Transcript detail page: after selecting the transcript of interest in the query results, a detailed translation mechanism deduction page is provided. In this page, the detailed IRES summary and translation initiation–related features are integrated to help understand and infer the testable mechanism hypotheses for each IRES. (iv) Download function: the integrative IRES–translation initiation interaction map and the supplemented experimental datasets can be downloaded as plain text files for subsequent analyses. An easy-to-use web interface was built to facilitate these functions.\\nUsers can type in a particular gene name or its aliases in the search page (Figure 3a). Users can also browse the database to find IRESs categorized by chromosomes and/or KEGG pathway annotations of genes regulated by them (Figure 3b). The putative IRESs and the summary of the corresponding translation initiation–related information are then presented in a tabular form (Figure 3c). For each transcript of the input gene, Human IRES Atlas summaries the brief transcript information, the putative IRES elements found on the transcripts, the number of prediction tools that specify this sequence to be a potential IRES, the functionality test results for each identified IRES element and finally the summaries of other translation initiation–related data observed within IRES regions. Three tests (conservation significance, RPI score significance and conditional translation efficiency significance) are provided to assess the functionality of each identified IRES. The summary of the translation initiation–related data matched to an IRES region includes the following information: (i) the number of eTIS (experimentally probed TIS) found in the upstream to the downstream 100 bps of the IRES element, (ii) the number of nTIS (non-canonical TIS) predicted in the upstream to the downstream 100 bps of the IRES element, (iii) the number of uORF observed in the upstream and downstream 100 bps of the IRES element, (iv) the number of ITAF-binding events that overlap and interact with the IRES element, (v) IRES activity measured for the IRES element and (vi) the number of literature IRES evidence that overlaps with the IRES prediction. Some IRES elements have been shown to use the cis uORFs and TISs to mediate their activity (45, 69). The amino acid stress or the stalling ribosome near an IRES can trigger a structural switch of the IRES to an active conformation, leading to IRES-mediated translation initiation. Therefore, we included extra 100 bps upstream and downstream of an IRES when associating TISs and uORFs to it. For ITAFs, they usually bind and remodel IRES structures to set up the ribosome-binding platform (38).\\nFigure 3.\\nThe search result page for the identified IRESs of a given gene.\\n(a) The search mode: users can type in the gene of interest to search for the identified IRESs on the given gene transcripts. (b) The browse mode: users can filter out the IRESs categorized by their residing chromosomes, annotated KEGG pathways of the regulated genes and the thresholds of IRES functionality q-values. (c) The result table for the identified IRES elements in all transcript isoforms of the queried gene. In this table, three IRES functionality test results and other translational regulatory features for each identified IRES element are summarized. (d) The link to the detail page of the selected transcript.\\nBelow the summary table of the translation initiation–related features, an interaction map between translation initiation–related features for the transcript can be viewed in this table for better data visualization (Figure 3c). For each transcript, one can click the ‘detail’ hyperlink (Figure 3d) to obtain the detail page that lists the complete information of the putative IRES elements, the visualized IRES–translation initiation interaction maps and the translational regulation information at a single nucleotide resolution. Both the genomic chromosomal coordinates and the transcript locations for different translation initiation–related information are listed in the detail page. Moreover, we have also provided the .bed files for the putative IRES track and different transcription-related information tracks. These files can facilitate users to integrate the application with the genome browser. In the last part, the implemented transcript detail page assists in forming molecular mechanism hypotheses for IRES-driven translation initiation. The walkthrough of the detail page is provided along with the case study.\\nCase study\\nHuman IRES Atlas aims to integrate the transcriptome-wide human IRESs and translational regulatory features. The integration provides ways to infer testable mechanism hypotheses for IRES-mediated translation regulation. One can use the IRES–translation initiation interaction map and the queried transcript’s detailed information to deduce such hypotheses. We discuss the IRES-mediated regulatory program of human FGF9 in detail to demonstrate the biological applicability of the constructed database. Human Fibroblast Growth Factor 9 (FGF9) is the gene that transcribes and translates to a protein that acts as an autocrine/paracrine growth factor in lung branching morphogenesis (72), bone development (73) and even cancer progression (74). It has been reported that the FGF9 mRNA is expressed in embryos while the FGF9 protein is identified to be of low level in most cells and is restrictedly expressed in specific cell organs (75). Hence, it is of interest to biologists that how the FGF9 transcripts are translationally regulated in human cells. In Human IRES Atlas, the FGF9 transcript map of IRES–translation initiation interactions provides clues for deducing the testable mechanism hypotheses. There is an identified overlapping IRES region (IRES2: 746.840) that was found by at least two IRES screening tools and is calculated to be conserved and significant in functional RPI scores (q\\u2009=\\u20092.83E-13*) (Figure 4a and b). In the FGF9 IRES–translation initiation interaction map, very low levels of TIS and ribosome occupancy signals were found (Figure 4c), suggesting the low translation level in normal conditions. Further, uORFs were discovered in proximity to these structurally significant IRES elements. Finally, HuR-binding signals were observed to be overlapped with the putative IRES. Details of the translation initiation–related data found around each IRES element are listed in the fourth part of the page (Figure 4d). A link to download the text files in the .csv format that contain the full information of the detail page is provided for users (Figure 4e).\\nFigure 4.\\nThe transcript detail page showing the potential IRES-driven translation regulatory program of the given transcript.\\n(a) The brief information table of the queried gene transcript. (b) The translation initiation–related feature summary table for the identified IRES elements within the queried gene transcript. (c) The IRES–translation initiation interaction map for the queried gene transcript. (d) The detail of the translation initiation–related feature data found for each IRES. (e) The download function to obtain the .csv or .bed formats of all information in the detail page.\\nBy careful consideration of the integrated information, we can deduce that FGF9 transcripts could be translationally regulated through the interplay between the structural IRES, uORF segments and ITAFs (see Figure 5). By the effort of molecular biologists, it is now verified that there is indeed a functional IRES structure near the end of the 5ʹ UTR of the FGF9 transcript. The uORF segments found around the IRES elements were shown to maintain the low protein expression level (40). The IRES structure induced under hypoxia recruits the ribosome to the CDS and activates the FGF9 protein synthesis with IRES–ITAF interactions. Although it is later found that the ITAF HuR may bind to the AU-rich elements in FGF9 transcripts (76), further studies and experiments should be carried out to figure out whether other ITAFs interact with these IRES elements. These findings conform to the hypotheses suggested by the constructed database. In conclusion, translation regulatory programs can be inferred with the facilitation of Human IRES Atlas.\\nFigure 5.\\nHuman IRES Atlas helps suggest testable IRES-mediated translational regulation mechanism hypotheses.\\nFor the FGF9 5ʹ UTR, the IRES–translation initiation interaction map suggests that (1) the transcript is of low translation level and (2) some regulatory program between the RPI interpretable IRESs, the nearby uORFs and some ITAFs may exist. From experimental literature evidence, we know that the uORF represses FGF9 protein synthesis in normal conditions. Some ITAFs bind to the IRES to start cap-independent translation under hypoxia, verifying the testable hypotheses suggested by the constructed database.\\nComparison with related works\\nHuman IRES Atlas was created for two primary purposes: (i) to provide comprehensive and systematic IRES identifications in the human species and (ii) to integrate IRES elements and translation initiation–related information for biologists to understand the interconnections between them. By constructing a comprehensive IRES–translation initiation interaction map, biologists can get testable hypotheses on cellular IRES elements and devise subsequent verification experiments. Currently, four existing databases and datasets are devoted to similar tasks as Human IRES Atlas and thus are compared in the following discussion.\\nTwo IRES databases [IRESite (71) and IRESbase (21)] were designed to curate and deposit IRES verification experiments from the literature. Researchers have carried out different bicistronic experiments for studying IRES activity of certain genes. Nevertheless, most of the information is fragmentary in different literature papers. Therefore, these two databases tried to gather and store these valuable records. While these two works collected valuable IRES experimental data, till now there are still few cellular IRES elements that have been experimentally verified and deposited. Only 26 human mRNA transcripts were evaluated to contain IRES elements in IRESite and 112 human IRESs were collected in IRESbase. Potential IRES-driven translational regulation programs in the human transcriptome cannot be easily deduced through these two databases due to the lack of an integrative platform for understanding the interactions between IRES elements and the related translation initiation features. Since there are only few available verified IRES elements in the literature, there is still a need to carry out systematic in silico IRES identification and construct a comprehensive IRES–translation initiation interaction map to complement the sparsity of IRES verification experiments.\\nAnother related work called IRES Omnibus (77) studied human IRESs through systematic IRES prediction. In the preliminary work of IRES Omnibus, Chen and Tseng set up an in-house IRES prediction system based on the PatSearch tool to screen through human 5ʹ UTR sequences for potential IRES elements. In their work, IREZone, the special regions consisting of contiguous overlapping RNA structures, and IRES elements were predicted and collected. Using the IRES and IREZone repository, novel sequence patterns and IRES-related cellular functions were investigated. However, in their preliminary identification, merely one IRES prediction tool was used, and the complicated connections between IRES elements and translation initiation–related features were ignored. Hence, the constructed preliminary human IRES map may suffer from high false positives. Further, it would be beneficial for biologists to deduce the potential IRES-related regulatory molecular mechanisms if different high throughput translation initiation–related datasets can be integrated into a user-friendly platform.\\nThe study by Weingarten-Gabbay et al. (12) is another touchstone in the field. We then evaluated the IRES identification sensitivity and specificity for Human IRES Atlas and the work of Weingarten-Gabbay et al. on a ground truth IRES dataset consisting of verified IRES sequences as the positive set and random sequences as the negative set. The positive set of 190 known IRES elements is obtained from the IRESbase (21) and the review work of Baird et al. (22). Since it is known that house-keeping genes depend mainly on cap-dependent translation initiation (23), we randomly sampled 190 sequences from the 5ʹ UTRs of annotated house-keeping genes (24) to be the negative set. An identified IRES in a dataset is defined to be a true positive if >20% of the base pairs overlap with some sequence in the positive set. A sequence from the negative set is called a true negative for one identification method if no identified sequence has >20% overlapping base pairs with it. On this ground truth IRES set, the results of Weingarten-Gabbay et al. obtain a sensitivity of 0.19 and a false-positive rate (FPR) of 0.037. The putative IRESs deposited in Human IRES Atlas have a sensitivity of 0.32 and an FPR of 0.226 on the ground truth dataset. We can see from these results that Human IRES Atlas achieves a better sensitivity with a small sacrifice of the FPR. Furthermore, if we merge the potential IRESs in Human IRES Atlas and the results of Weingarten-Gabbay et al., a sensitivity of 0.43 and an FPR of 0.258 are observed. The near linearly independent sensitivity gain means that the two results are mostly complementary. By examining the data of Weingarten-Gabbay et al., 19% of the literature-curated IRESs were found. Thirty-two percent of the literature-curated IRESs could be determined in Human IRES Atlas. While oversampling 5ʹ UTRs can lead to higher chances of overlapping with the true positive set, it also increases the possibility of overlapping the putative IRESs with the true negative set, raising the FPR. There is a compromise between the sensitivity and the FPR when oversampling 5ʹ UTRs. The putative IRES elements deposited in Human IRES Atlas are estimated to have a higher sensitivity than the corresponding FPR. Hence, Human IRES Atlas provides extra insights for IRES-mediated translation regulation in humans.\\nHuman IRES Atlas is the first database explicitly designed to overcome current obstacles in studying human IRES elements. It also fulfills the necessity for deducing testable molecular mechanism hypotheses. In Human IRES Atlas, three different IRES prediction tools (IRESFinder, PatSearch and IRESpy) were used for systematic IRES screening. The results of different tools were collected by selecting the nested consensus minimal sequences among different prediction algorithms. Hence the prediction uncertainty and false discovery rate inherent in different tools were minimized during the identification process. To further help evaluate the functionality of the putative IRESs, three tests (conservation, structural RPI scores and conditional translation efficiency) were designed and performed. Besides performing the transcriptome-wide human IRES identification, we also integrated eight categories of translation initiation–related datasets. Biologists can further understand the IRES-driven translation regulation and develop testable molecular mechanism hypotheses for subsequent research through the translation initiation–related features. In summary, Human IRES Atlas integrates more data and provides more functionality than existing tools. It specifically aims to perform systematic IRES identification and help deduce the IRES-driven cellular translation regulation mechanisms.\\nIssues related to the constructed database\\nThe Human IRES Atlas database is constructed by depositing systematically identified in silico IRESs and various translation initiation–related data. Based on the eight gathered categories of translation initiation–related features, we can assess the significance of the putative IRESs and deduce potential IRES-driven translation regulation mechanism hypotheses. However, several issues may exist when interpreting the results generated from the systems-wide data integration process. Researchers usually perturbed cells and conducted distinct high-throughput experiments for gathering genome-wide information (78). However, different high-throughput experiments may be performed under different environmental conditions, cellular states or cell types. This experimental discrepancy causes inevitable systematic bias in the data integration process (79). Hence, some detailed subtleties should be considered when designing subsequent verification experiments based on the hypotheses formed from the systems view of data integration.\\nDiverse statistical analysis strategies were applied to the high-throughput experiment datasets. For example, we adopted the eTIS data by thresholding the positional read-count fold change and some other minimum detection requirement between samples treated with the ribosome E-site translation inhibitors CHX and LTM (64). The high-throughput screening of IRES activity adopted in this database was probed using fluorescence measurements (12). Different statistical analysis strategies provide different false discovery rates. In general, noises resulted from the data statistical analysis were assumed to be orthogonal in different high-throughput experimental results (35). Hence, by simultaneously combining experimental data of similar targeted systems, the false positive issue can be reduced. In Human IRES Atlas, we have gathered diverse translation initiation–related data and used different screening tools in the systematic putative IRES identification. This cross-referencing screening reduces the impact of false positives. For instance, TISs based on CHX/LTM experimental analysis and PreTIS prediction were both collected to lessen the false positive impact in TIS location information. We will keep Human IRES Atlas up-to-date with newly published datasets when more data or methods are released.\\nData integration in multicellular organisms often brings about the issue of cellular systems bias. In metazoa species, correct temporal and spatial gene expression of the same genome is the core control on cell differentiation (13). While different combinations of cell types in data integration may provide distinct false positives, the multicellular systems problem should be taken care of in using the constructed database. IRES activity data were probed in vitro based on experiments on sequence libraries. Non-canonical TIS and uORF information were identified in silico by investigating comparative genomics and sequence contents. These data reveal the normal status of the translation program. To assist in cell type-specific or condition-specific IRES verification experimental design, one can refer to ribosome profiling experiments conducted under different conditions/cell types or refer to special cellular conditions where the ITAF-binding events were observed. Diverse ribosome profiling sequencing results are fragmentary in the literature. Recently, a database called HPRDviewer (41) is constructed to deposit comprehensive human ribosome profiling datasets. In Human IRES Atlas, the conditional ribosome profiling results performed under the stressed and normal conditions are shown. More condition-specific human ribosome profiling information can be found in HPRDviewer. Through consulting more detailed ribosome profiling experiments or the ITAF-binding event conditions, one can further minimize the impact of cell type bias inherent in the data integration process.\\nGo to:\\nConclusions\\nIn this research, a novel database called Human IRES Atlas was constructed to deposit the systematic in silico identified IRESs from the human transcriptome. Besides the transcriptome-wide putative IRES identification, eight genres of translation initiation–related datasets were integrated and three novel IRES functionality tests were designed. To facilitate the understanding of the connections between IRES elements and these translation initiation–related features, an easy-to-use interface for the integrated results was implemented for biologists to infer possible molecular mechanisms of human IRES-driven translation initiation. Researchers can easily search the IRES of interest or browse the IRESs on genes involved in specific biological pathways with the database’s user-friendly web-interface. By simultaneously studying the human IRES and the translation initiation–related features, the database can provide testable molecular mechanism hypotheses on human gene translation regulation. We believe that Human IRES Atlas can help biologists better study mechanisms of cellular IRES-driven translation initiation, IRES-related cancer progression and other IRES-related genetic disorders.\\nGo to:\\nAcknowledgements\\nThe authors would like to thank Ting-Hsuan Chen and Yu-Cian Lin for helping with the preliminary data collection.\\nGo to:\\nContributor Information\\nTzu-Hsien Yang, Department of Information Management, National University of Kaohsiung, 700, Kaohsiung University Rd., Nanzih District, Kaohsiung 811, Taiwan.\\nChung-Yu Wang, Department of Information Management, National University of Kaohsiung, 700, Kaohsiung University Rd., Nanzih District, Kaohsiung 811, Taiwan.\\nHsiu-Chun Tsai, Department of Information Management, National University of Kaohsiung, 700, Kaohsiung University Rd., Nanzih District, Kaohsiung 811, Taiwan.\\nCheng-Tse Liu, Department of Information Management, National University of Kaohsiung, 700, Kaohsiung University Rd., Nanzih District, Kaohsiung 811, Taiwan.\\nGo to:\\nFunding\\nThis work is funded by National University of Kaohsiung and Ministry of Science and Technology of Taiwan (MOST 107-2218-E-390-009-MY3).\\nGo to:\\nConflict of interest.\\nThe authors declare that they have no competing interests.\\nGo to:\\nReferences\\n1. Gebauer F., Hentze M.W. (2004) Molecular mechanisms of translational control. Nat. Rev. Mol. Cell Biol., 5, nrm1488. [PMC free article] [PubMed] [Google Scholar]\\n2. Sonenberg N., Hinnebusch A.G. (2009) Regulation of translation initiation in eukaryotes: mechanisms and biological targets. Cell, 136, 731–745. [PMC free article] [PubMed] [Google Scholar]\\n3. Johnson A.G., Grosely R., Petrov A.N.. et al. (2017) Dynamics of IRES-mediated translation. Philos. Trans. R. Soc. B Biol. Sci., 372, 20160177. [PMC free article] [PubMed] [Google Scholar]\\n4. Pelletier J., Sonenberg N. (1988) Internal initiation of translation of eukaryotic mRNA directed by a sequence derived from poliovirus RNA. Nature, 334, 320. [PubMed] [Google Scholar]\\n5. Lukavsky P.J. (2009) Structure and function of HCV IRES domains. Virus Res., 139, 166–171. [PMC free article] [PubMed] [Google Scholar]\\n6. Ray P.S., Grover R., Das S. (2006) Two internal ribosome entry sites mediate the translation of p53 isoforms. EMBO Rep., 7, 404–410. [PMC free article] [PubMed] [Google Scholar]\\n7. Sharathchandra A., Katoch A., Das S. (2014) IRES mediated translational regulation of p53 isoforms. Wiley Interdiscip. Rev. RNA, 5, 131–139. [PubMed] [Google Scholar]\\n8. Sharma D.K., Bressler K., Patel H.. et al. (2016) Role of eukaryotic initiation factors during cellular stress and cancer progression. J. Nucleic Acids, 2016, 8235121. [PMC free article] [PubMed] [Google Scholar]\\n9. Holcik M., Sonenberg N. (2005) Translational control in stress and apoptosis. Nat. Rev. Mol. Cell Biol., 6, 318–327. [PubMed] [Google Scholar]\\n10. Spriggs K.A., Stoneley M., Bushell M.. et al. (2008) Re-programming of translation following cell stress allows IRES-mediated translation to predominate. Biol. Cell, 100, 27–38. [PubMed] [Google Scholar]\\n11. Leppek K., Das R., Barna M. (2018) Functional 5′UTR mRNA structures in eukaryotic translation regulation and how to find them. Nat. Rev. Mol. Cell Biol., 19, 158. [PMC free article] [PubMed] [Google Scholar]\\n12. Weingarten-Gabbay S., Elias-Kirma S., Nir R. et al. (2016) Systematic discovery of cap-independent translation sequences in human and viral genomes. Science, 351, aad4939. [PubMed] [Google Scholar]\\n13. Yang T.-H., Wang -C.-C., Hung P.-C.. et al. (2014) cis MEP: an integrated repository of genomic epigenetic profiles and cis-regulatory modules in Drosophila. BMC Syst. Biol., 8, S8. [PMC free article] [PubMed] [Google Scholar]\\n14. Yang T.-H., Wang -C.-C., Wang Y.-C.. et al. (2014) YTRP: a repository for yeast transcriptional regulatory pathways. Database, 2014, bau014. [PMC free article] [PubMed] [Google Scholar]\\n15. Haeussler M., Zweig A.S., Tyner C.. et al. (2019) The UCSC genome browser database: 2019 update. Nucleic Acids Res., 47, D853–D858. [PMC free article] [PubMed] [Google Scholar]\\n16. Zhao J., Wu J., Xu T.. et al. (2018) IRESfinder: identifying RNA internal ribosome entry site in eukaryotic cell using framed k-mer features. J. Genet. Genomics, 45, 403–406. [PubMed] [Google Scholar]\\n17. Grillo G., Licciulli F., Liuni S.. et al. (2003) PatSearch: a program for the detection of patterns and structural motifs in nucleotide sequences. Nucleic Acids Res., 31, 3608–3612. [PMC free article] [PubMed] [Google Scholar]\\n18. Washietl S., Hofacker I.L., Stadler P.F. (2005) Fast and reliable prediction of noncoding RNAs. Proc. Natl. Acad. Sci. USA, 102, 2454–2459. [PMC free article] [PubMed] [Google Scholar]\\n19. Kolekar P., Pataskar A., Kulkarni-Kale U.. et al. (2016) IRESPred: web server for prediction of cellular and viral internal ribosome entry site (IRES). Sci. Rep., 6, 27436. [PMC free article] [PubMed] [Google Scholar]\\n20. Wang J., Gribskov M. (2019) IRESpy: an XGBoost model for prediction of internal ribosome entry sites. BMC Bioinform., 20, 409. [PMC free article] [PubMed] [Google Scholar]\\n21. Zhao J., Li Y., Wang C.. et al. (2020) IRESbase: a comprehensive database of experimentally validated internal ribosome entry sites. Genomics Proteomics Bioinf, 18, 129–139. [PMC free article] [PubMed] [Google Scholar]\\n22. Baird S.D., Turcotte M., Korneluk R.G.. et al. (2006) Searching for IRES. RNA, 12, 1755–1785. [PMC free article] [PubMed] [Google Scholar]\\n23. Komar A.A., Hatzoglou M. (2011) Cellular IRES-mediated translation: the war of ITAFs in pathophysiological states. Cell Cycle, 10, 229–240. [PMC free article] [PubMed] [Google Scholar]\\n24. Eisenberg E., Levanon E.Y. (2013) Human housekeeping genes, revisited. Trends Genet., 29, 569–574. [PubMed] [Google Scholar]\\n25. Gritsenko A.A., Weingarten-Gabbay S., Elias-Kirma S.. et al. (2017) Sequence features of viral and human Internal Ribosome Entry Sites predictive of their activity. PLoS Comput. Biol., 13, e1005734. [PMC free article] [PubMed] [Google Scholar]\\n26. Du X., Wang J., Zhu H.. et al. (2013) Second cistron in CACNA1A gene encodes a transcription factor mediating cerebellar development and SCA6. Cell, 154, 118–133. [PMC free article] [PubMed] [Google Scholar]\\n27. Cornelis S., Bruynooghe Y., Denecker G.. et al. (2000) Identification and characterization of a novel cell cycle–regulated internal ribosome entry site. Mol. Cell, 5, 597–605. [PubMed] [Google Scholar]\\n28. Candeias M.M., Powell D.J., Roubalova E.. et al. (2006) Expression of p53 and p53/47 are controlled by alternative mechanisms of messenger RNA translation initiation. Oncogene, 25, 6936–6947. [PubMed] [Google Scholar]\\n29. Faye M.D., Graber T.E., Liu P.. et al. (2013) Nucleotide composition of cellular internal ribosome entry sites defines dependence on NF45 and predicts a posttranscriptional mitotic regulon. Mol. Cell. Biol., 33, 307–318. [PMC free article] [PubMed] [Google Scholar]\\n30. Stoneley M., Willis A.E. (2004) Cellular internal ribosome entry segments: structures, trans-acting factors and regulation of gene expression. Oncogene, 23, 3200. [PubMed] [Google Scholar]\\n31. Wan Y., Kertesz M., Spitale R.C.. et al. (2011) Understanding the transcriptome through RNA structure. Nat. Rev. Genet., 12, 641–655. [PMC free article] [PubMed] [Google Scholar]\\n32. Yang T.-H. (2019) Transcription factor regulatory modules provide the molecular mechanisms for functional redundancy observed among transcription factors in yeast. BMC Bioinform., 20, 1–16. [PMC free article] [PubMed] [Google Scholar]\\n33. Siepel A., Bejerano G., Pedersen J.S.. et al. (2005) Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. Genome Res., 15, 1034–1050. [PMC free article] [PubMed] [Google Scholar]\\n34. Margulies E.H., Blanchette M., Haussler D.et al. , N. C. S. Program . (2003) Identification and characterization of multi-species conserved sequences. Genome Res., 13, 2507–2518. [PMC free article] [PubMed] [Google Scholar]\\n35. Yang T.-H., Wu W.-S. (2013) Inferring functional transcription factor-gene binding pairs by integrating transcription factor binding data with transcription factor knockout data. BMC Syst. Biol., 7, S13. [PMC free article] [PubMed] [Google Scholar]\\n36. Lee K.-M., Chen C.-J., Shih S.-R. (2017) Regulation mechanisms of viral IRES-driven translation. Trends Microbiol., 25, 546–561. [PubMed] [Google Scholar]\\n37. Sidrauski C., McGeachy A.M., Ingolia N.T.. et al. (2015) The small molecule ISRIB reverses the effects of eIF2α phosphorylation on translation and stress granule assembly. Elife, 4, e05033. [PMC free article] [PubMed] [Google Scholar]\\n38. Mitchell S.A., Spriggs K.A., Coldwell M.J.. et al. (2003) The Apaf-1 internal ribosome entry segment attains the correct structural conformation for function via interactions with PTB and unr. Mol. Cell, 11, 757–771. [PubMed] [Google Scholar]\\n39. Zhang H., Wang Y., Lu J. (2019) Function and evolution of upstream ORFs in eukaryotes. Trends Biochem. Sci., 44, 782–794. [PubMed] [Google Scholar]\\n40. Chen T.-M., Shih Y.-H., Tseng J. T.. et al. (2014) Overexpression of FGF9 in colon cancer cells is mediated by hypoxia-induced translational activation. Nucleic Acids Res., 42, 2932–2944. [PMC free article] [PubMed] [Google Scholar]\\n41. Wu W.-S., Jiang Y.-X., Chang J.-W.. et al. (2018) HRPDviewer: human ribosome profiling data viewer. Database, 2018, bay074. [PMC free article] [PubMed] [Google Scholar]\\n42. Martin M. (2011) Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal, 17, 10. [Google Scholar]\\n43. Li B., Dewey C.N. (2011) RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome. BMC Bioinform., 12, 1–16. [PMC free article] [PubMed] [Google Scholar]\\n44. Le Quesne J.P.C., Stoneley M., Fraser G.A.. et al. (2001) Derivation of a structural model for the c-myc IRES. J. Mol. Biol., 310, 111–126. [PubMed] [Google Scholar]\\n45. Yaman I., Fernandez J., Liu H.. et al. (2003) The zipper model of translational control: a small upstream ORF is the switch that controls structural remodeling of an mRNA leader. Cell, 113, 519–531. [PubMed] [Google Scholar]\\n46. Lorenz R., Bernhart S.H., Zu Siederdissen C.H.. et al. (2011) ViennaRNA package 2.0. Algorithms Mol. Biol., 6, 26. [PMC free article] [PubMed] [Google Scholar]\\n47. Hofacker I.L., Priwitzer B. and Stadler P.F. (2004) Prediction of locally stable RNA secondary structures for genome-wide surveys. Bioinformatics, 20, 186–190. [PubMed] [Google Scholar]\\n48. Deng F., Ledda M., Vaziri S.. et al. (2016) Data-directed RNA secondary structure prediction using probabilistic modeling. RNA, 22, 1109–1109. [PMC free article] [PubMed] [Google Scholar]\\n49. Lu Z.J., Gloor J.W., Mathews D.H. (2009) Improved RNA secondary structure prediction by maximizing expected pair accuracy. RNA, 15, 1805–1813. [PMC free article] [PubMed] [Google Scholar]\\n50. Bernhart S.H., Hofacker I.L., Will S.. et al. (2008) RNAalifold: improved consensus structure prediction for RNA alignments. BMC Bioinform., 9, 474. [PMC free article] [PubMed] [Google Scholar]\\n51. Reuter J.S., Mathews D.H. (2010) RNAstructure: software for RNA secondary structure prediction and analysis. BMC Bioinform., 11, 129. [PMC free article] [PubMed] [Google Scholar]\\n52. Will S., Otto C., Miladi M.. et al. (2015) SPARSE: quadratic time simultaneous alignment and folding of RNAs without sequence-based heuristics. Bioinformatics, 31, 2489–2496. [PMC free article] [PubMed] [Google Scholar]\\n53. Tabei Y., Kiryu H., Kin T.. et al. (2008) A fast structural multiple alignment method for long RNA sequences. BMC Bioinform., 9, 33. [PMC free article] [PubMed] [Google Scholar]\\n54. Glouzon J.-P.S. and Ouangraoua A. (2018) aliFreeFold: an alignment-free approach to predict secondary structure from homologous RNA sequences. Bioinformatics, 34, i70–i78. [PMC free article] [PubMed] [Google Scholar]\\n55. Will S., Joshi T., Hofacker I.L.. et al. (2012) LocARNA-P: accurate boundary prediction and improved detection of structural RNAs. RNA, 18, 900–914. [PMC free article] [PubMed] [Google Scholar]\\n56. Ji Y., Xu X., Stormo G.D. (2004) A graph theoretical approach for predicting common RNA secondary structure motifs including pseudoknots in unaligned sequences. Bioinformatics, 20, 1591–1602. [PubMed] [Google Scholar]\\n57. Wu Y., Shi B., Ding X.. et al. (2015) Improved prediction of RNA secondary structure by integrating the free energy model with restraints derived from experimental probing data. Nucleic Acids Res., 43, 7247–7259. [PMC free article] [PubMed] [Google Scholar]\\n58. Lin Y., Liu T., Cui T.. et al. (2020) RNAInter in 2020: RNA interactome repository with increased coverage and annotation. Nucleic Acids Res., 48, D189–D197. [PMC free article] [PubMed] [Google Scholar]\\n59. Gardner P.P., Wilm A., Washietl S. (2005) A benchmark of multiple sequence alignment programs upon structural RNAs. Nucleic Acids Res., 33, 2433–2439. [PMC free article] [PubMed] [Google Scholar]\\n60. Jiang T., Lin G., Ma B.. et al. (2002) A general edit distance between RNA structures. J. Comput. Biol., 9, 371–388. [PubMed] [Google Scholar]\\n61. Yang T.H., Wu W.S. (2012) Identifying biologically interpretable transcription factor knockout targets by jointly analyzing the transcription factor knockout microarray and the ChIP-chip data. BMC Syst. Biol., 6, 102. [PMC free article] [PubMed] [Google Scholar]\\n62. Blin K., Dieterich C., Wurmus R.. et al. (2015) DoRiNA 2.0—upgrading the doRiNA database of RNA interactions in post-transcriptional regulation. Nucleic Acids Res., 43, D160–D167. [PMC free article] [PubMed] [Google Scholar]\\n63. Touriol C., Bornes S., Bonnal S.. et al. (2003) Generation of protein isoform diversity by alternative initiation of translation at non-AUG codons. Biol. Cell, 95, 169–178. [PubMed] [Google Scholar]\\n64. Lee S., Liu B., Lee S.. et al. (2012) Global mapping of translation initiation sites in mammalian cells at single-nucleotide resolution. Proc. Natl. Acad. Sci. USA, 109, E2424–E2432. [PMC free article] [PubMed] [Google Scholar]\\n65. Wan J. and Qian S.-B. (2013) TISdb: a database for alternative translation initiation in mammalian cells. Nucleic Acids Res., 42, D845–D850. [PMC free article] [PubMed] [Google Scholar]\\n66. Langmead B., Trapnell C., Pop M.. et al. (2009) Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biol., 10, R25. [PMC free article] [PubMed] [Google Scholar]\\n67. Reuter K., Biehl A., Koch L.. et al. (2016) PreTIS: a tool to predict non-canonical 5’UTR translational initiation sites in human and mouse. PLoS Comput. Biol., 12, e1005170. [PMC free article] [PubMed] [Google Scholar]\\n68. Somers J., Pöyry T. and Willis A.E. (2013) A perspective on mammalian upstream open reading frame function. Int. J. Biochem. Cell Biol., 45, 1690–1700. [PMC free article] [PubMed] [Google Scholar]\\n69. Fernandez J., Yaman I., Huang C.. et al. (2005) Ribosome stalling regulates IRES-mediated translation in eukaryotes, a parallel to prokaryotic attenuation. Mol. Cell, 17, 405–416. [PubMed] [Google Scholar]\\n70. Ye Y., Liang Y., Yu Q.. et al. (2015) Analysis of human upstream open reading frames and impact on gene expression. Hum. Genet., 134, 605–612. [PubMed] [Google Scholar]\\n71. Mokrejš M., Mašek T., Vopálenský V.. et al. (2009) IRESite—a tool for the examination of viral and cellular internal ribosome entry sites. Nucleic Acids Res., 38, D131–D136. [PMC free article] [PubMed] [Google Scholar]\\n72. Colvin J.S., Green R.P., Schmahl J.. et al. (2001) Male-to-female sex reversal in mice lacking fibroblast growth factor 9. Cell, 104, 875–889. [PubMed] [Google Scholar]\\n73. Robinson D., Hasharoni A., Evron Z.. et al. (2000) Synovial chondromatosis: the possible role of FGF 9 and FGF receptor 3 in its pathology. Int. J. Exp. Pathol., 81, 183–189. [PMC free article] [PubMed] [Google Scholar]\\n74. Arai D., Hegab A.E., Soejima K.. et al. (2015) Characterization of the cell of origin and propagation potential of the fibroblast growth factor 9-induced mouse model of lung adenocarcinoma. J. Pathol., 235, 593–605. [PMC free article] [PubMed] [Google Scholar]\\n75. Colvin J.S., Feldman B., Nadeau J.H.. et al. (1999) Genomic organization and embryonic expression of the mouse fibroblast growth factor 9 gene. Dev. Dyn. An Off. Publ. Am. Assoc. Anat., 216, 72–88. [PubMed] [Google Scholar]\\n76. Chen T.-M., Hsu C.-H., Tsai S.-J.. et al. (2010) AUF1 p42 isoform selectively controls both steady-state and PGE2-induced FGF9 mRNA decay. Nucleic Acids Res., 38, 8061–8071. [PMC free article] [PubMed] [Google Scholar]\\n77. Chen W.-M. (2009) IRES Omnibus: An Integrated Web Server for Predicting and Searching the Human Genes Bearing the IRES Elements. National Cheng Kung University, Tainan. [Google Scholar]\\n78. Ideker T., Galitski T., Hood L. (2001) A new approach to decoding life: systems biology. Annu. Rev. Genomics Hum. Genet., 2, 343–372. [PubMed] [Google Scholar]\\n79. Hwang D., Rust A.G., Ramsey S.. et al. (2005) A data integration methodology for systems biology. Proc. Natl. Acad. Sci. USA, 102, 17296–17301. [PMC free article] [PubMed] [Google Scholar]\\nArticles from Database: The Journal of Biological Databases and Curation are provided here courtesy of Oxford University Press'"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "content.text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "f1ef543d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'<div id=\"mc\" class=\" article lit-style content pmc-wm slang-all page-box\"><!--main-content--><div class=\"jig-ncbiinpagenav\" data-jigconfig=\"smoothScroll: false, allHeadingLevels: [\\'h2\\'], headingExclude: \\':hidden,.nomenu\\'\" id=\"ui-ncbiinpagenav-1\"><div class=\"fm-sec half_rhythm no_top_margin\"><div class=\"fm-flexbox\"><div class=\"fm-citation\"><div class=\"citation-default\"><div class=\"part1\"><span role=\"menubar\"><a href=\"#\" role=\"menuitem\" aria-expanded=\"false\" aria-haspopup=\"true\">Database (Oxford).</a></span> 2021; 2021: baab025. </div><div class=\"part2\"><span class=\"fm-vol-iss-date\">Published online 2021 May 18. </span>  <span class=\"doi\"><span>doi:&nbsp;</span><a href=\"//dx.doi.org/10.1093%2Fdatabase%2Fbaab025\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">10.1093/database/baab025</a></span></div></div></div><div class=\"fm-ids\"><div class=\"fm-citation-pmcid\"><span class=\"fm-citation-ids-label\">PMCID: </span><span>PMC8094437</span></div><div class=\"fm-citation-pmid\">PMID: <a href=\"/pubmed/33942874\">33942874</a></div></div></div><h1 class=\"content-title\">Human IRES Atlas: an integrative platform for studying IRES-driven translational regulation in humans</h1><div class=\"half_rhythm\"><div class=\"contrib-group fm-author\"><a href=\"/pubmed/?term=Yang%20TH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33942874\" class=\"affpopup\" co-rid=\"_co_idm139637300985152\" co-class=\"co-affbox\">Tzu-Hsien Yang</a>,<sup></sup> <a href=\"/pubmed/?term=Wang%20CY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33942874\" class=\"affpopup\" co-rid=\"_co_idm139637300981824\" co-class=\"co-affbox\">Chung-Yu Wang</a>,<sup></sup> <a href=\"/pubmed/?term=Tsai%20HC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33942874\" class=\"affpopup\" co-rid=\"_co_idm139637300978880\" co-class=\"co-affbox\">Hsiu-Chun Tsai</a>,<sup></sup> and  <a href=\"/pubmed/?term=Liu%20CT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33942874\" class=\"affpopup\" co-rid=\"_co_idm139637300975488\" co-class=\"co-affbox\">Cheng-Tse Liu</a><sup></sup></div><div style=\"display:none\" class=\"contrib-group aff-tip\"><div id=\"_co_idm139637300985152\"><h3 class=\"no_margin\">Tzu-Hsien Yang</h3><p>\\nDepartment of Information Management, National University of Kaohsiung, 700, Kaohsiung University Rd., Nanzih District, Kaohsiung 811, Taiwan</p><div>Find articles by <a href=\"/pubmed/?term=Yang%20TH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33942874\">Tzu-Hsien Yang</a></div></div><div id=\"_co_idm139637300981824\"><h3 class=\"no_margin\">Chung-Yu Wang</h3><p>\\nDepartment of Information Management, National University of Kaohsiung, 700, Kaohsiung University Rd., Nanzih District, Kaohsiung 811, Taiwan</p><div>Find articles by <a href=\"/pubmed/?term=Wang%20CY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33942874\">Chung-Yu Wang</a></div></div><div id=\"_co_idm139637300978880\"><h3 class=\"no_margin\">Hsiu-Chun Tsai</h3><p>\\nDepartment of Information Management, National University of Kaohsiung, 700, Kaohsiung University Rd., Nanzih District, Kaohsiung 811, Taiwan</p><div>Find articles by <a href=\"/pubmed/?term=Tsai%20HC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33942874\">Hsiu-Chun Tsai</a></div></div><div id=\"_co_idm139637300975488\"><h3 class=\"no_margin\">Cheng-Tse Liu</h3><p>\\nDepartment of Information Management, National University of Kaohsiung, 700, Kaohsiung University Rd., Nanzih District, Kaohsiung 811, Taiwan</p><div>Find articles by <a href=\"/pubmed/?term=Liu%20CT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33942874\">Cheng-Tse Liu</a></div></div></div></div><div class=\"fm-panel half_rhythm\"><div class=\"togglers fm-copyright-license\"><a href=\"#\" class=\"pmctoggle\" rid=\"idm139637292208448_ai\">Author information</a> <a href=\"#\" class=\"pmctoggle\" rid=\"idm139637292208448_an\">Article notes</a> <a href=\"#\" class=\"pmctoggle\" rid=\"idm139637292208448_cpl\">Copyright and License information</a> <a href=\"/pmc/about/disclaimer/\" style=\"margin-left: 1em\">Disclaimer</a></div><div class=\"fm-authors-info fm-panel hide half_rhythm\" id=\"idm139637292208448_ai\" style=\"display:none\"><div><span class=\"fm-affl\">Tzu-Hsien Yang, </span><span class=\"fm-affl\" lang=\"en\" id=\"idm139637289670160\"> \\nDepartment of Information Management, National University of Kaohsiung, 700, Kaohsiung University Rd., Nanzih District, Kaohsiung 811, Taiwan;</span></div><div><a href=\"#idm139637292208448aff-info\">Contributor Information</a>.</div><div id=\"COR0001\">*Corresponding author: Tel: 886-7-5916772; Fax: 886-7-5919328; Email: <a href=\"mailto:dev@null\" data-email=\"wt.ude.kun@5201azgnayht\" class=\"oemail\">wt.ude.kun@5201azgnayht</a></div><div id=\"FN001\"><sup>\\n†\\n</sup>Chung-Yu Wang, Hsiu-Chun Tsai and Cheng-Tse Liu contributed equally to this work.</div></div><div class=\"fm-article-notes fm-panel hide half_rhythm\" id=\"idm139637292208448_an\" style=\"display:none\"><div class=\"fm-pubdate half_rhythm\">Received 2020 Nov 18; Revised 2021 Apr 16; Accepted 2021 Apr 23.</div></div><div class=\"permissions fm-panel half_rhythm hide\" id=\"idm139637292208448_cpl\" style=\"display:none\"><div class=\"fm-copyright half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright</a> © The Author(s) 2021. Published by Oxford University Press.</div><div class=\"license half_rhythm\">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href=\"https://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div></div><div id=\"pmclinksbox\" class=\"links-box whole_rhythm hidden\"></div></div><div class=\"sec\"></div><div id=\"idm139637299406304\" lang=\"en\" class=\"tsec sec\"><div class=\"goto jig-ncbiinpagenav-goto-container\"><span role=\"menubar\"><a class=\"tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading\" href=\"#\" title=\"Go to other sections in this page\" role=\"menuitem\" aria-expanded=\"false\" aria-haspopup=\"true\">Go to:</a></span></div><h2 class=\"head no_bottom_margin ui-helper-clearfix\" id=\"idm139637299406304title\">Abstract</h2><!--article-meta--><div><p id=\"idm139637299405920\" class=\"p p-first\">It is now known that cap-independent translation initiation facilitated by internal ribosome entry sites (IRESs) is vital in selective cellular protein synthesis under stress and different physiological conditions. However, three problems make it hard to understand transcriptome-wide cellular IRES-mediated translation initiation mechanisms: (i) complex interplay between IRESs and other translation initiation–related information, (ii) reliability issue of <em>in silico</em> cellular IRES investigation and (iii) labor-intensive <em>in vivo</em> IRES identification. In this research, we constructed the Human IRES Atlas database for a comprehensive understanding of cellular IRESs in humans. First, currently available and suitable IRES prediction tools (IRESfinder, PatSearch and IRESpy) were used to obtain transcriptome-wide human IRESs. Then, we collected eight genres of translation initiation–related features to help study the potential molecular mechanisms of each of the putative IRESs. Three functional tests (conservation, structural RNA–protein scores and conditional translation efficiency) were devised to evaluate the functionality of the identified putative IRESs. Moreover, an easy-to-use interface and an IRES–translation initiation interaction map for each gene transcript were implemented to help understand the interactions between IRESs and translation initiation–related features. Researchers can easily search/browse an IRES of interest using the web interface and deduce testable mechanism hypotheses of human IRES-driven translation initiation based on the integrated results. In summary, Human IRES Atlas integrates putative IRES elements and translation initiation–related experiments for better usage of these data and deduction of mechanism hypotheses.</p><p id=\"idm139637299402640\" class=\"p p-last\">\\n<strong>Database URL</strong>: <a href=\"http://cobishss0.im.nuk.edu.tw/Human_IRES_Atlas/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://cobishss0.im.nuk.edu.tw/Human_IRES_Atlas/</a></p></div></div><div id=\"s1\" class=\"tsec sec\"><div class=\"goto jig-ncbiinpagenav-goto-container\"><a class=\"tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading\" href=\"#\" title=\"Go to other sections in this page\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Go to:</a></div><h2 class=\"head no_bottom_margin ui-helper-clearfix\" id=\"s1title\">Background</h2><p id=\"idm139637334523936\" class=\"p p-first\">Cells in proliferation adapt their metabolism to environmental stimuli through the regulation of protein synthesis, and translational regulation can provide instant protein level adjustment for handling sudden environmental changes and stress responses (<a href=\"#R1\" rid=\"R1\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">1</a>). Importantly, translation initiation is widely considered to be a vital step in translational regulation (<a href=\"#R2\" rid=\"R2\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">2</a>). For eukaryotes, the complex phase of cellular translation initiation can be categorized into two different types: cap-dependent translation initiation and cap-independent translation initiation (<a href=\"#R3\" rid=\"R3\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">3</a>). Under normal situations, eukaryotic canonical cap-dependent translation initiation is started by the pre-initiation complex formed with initiators tRNA, eIF2, eIF3 and eIF4 to scan for the start codon. However, when facing stressed environments or other physiological challenges, cellular translation can be initiated in a cap-independent manner via the help of the internal ribosome entry site (IRES) elements. IRES-mediated translational initiation was first discovered in poliovirus (<a href=\"#R4\" rid=\"R4\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">4</a>). Since then, scientists have found different IRES elements in other viruses (<a href=\"#R5\" rid=\"R5\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">5</a>) and in cellular genes (<a href=\"#R6\" rid=\"R6\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">6</a>). Cellular IRES elements are now known to be closely related to selective cellular protein synthesis under stress and physiopathological conditions (<a href=\"#R7\" rid=\"R7\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">7</a>) and can have profound effects on human tumor cells (<a href=\"#R8\" rid=\"R8\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">8</a>). Therefore, it is crucial to investigate cellular IRES elements, thus unraveling the nature of cellular stress responses and tumor therapeutic targets.</p><p id=\"idm139637289516400\">Currently known cellular IRES elements are mostly found in genes involved in stress, mitosis or apoptosis (<a href=\"#R9\" rid=\"R9\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">9</a>). They usually bare sequence or structural motifs that can recruit the translation machinery to initiate the translation initiation stage (<a href=\"#R10\" rid=\"R10\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">10</a>). Although cellular IRESs generally show lower structural similarities to one another and are less structured than viral IRESs, they can be roughly categorized into three different functional genres with some identifiable patterns (<a href=\"#R11\" rid=\"R11\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">11</a>). First of all, IRES structural forms within 5ʹ untranslated regions (UTRs) can interact with RNA-binding proteins known as IRES <em>trans</em>-acting factors (ITAFs) to remodel their structures, thus opening the hindrance to ribosome recruitment. Second, upstream open reading frames (uORFs) in the neighborhood of an IRES can retain ribosomes in the uORF regions to transform the IRES structure into an active form using amino acid stress. The resulting active conformed IRES then helps presume the cap-independent translation initiation. Third, IRES can modulate different regulation signals around 5ʹ UTR and recruit internal ribosomes accordingly. Some known translational regulation signals such as cap inhibition, uORF translation, ITAFs or RNA G-quadruplexes can trigger the bypassing of canonical cap-dependent translation initiation. These basic known cellular IRES functions provide clues to select the transcriptome-wide IRES sequences in humans.</p><p id=\"idm139637289511392\">Various translation initiation–related cellular features are associated with the aforementioned cellular IRES functions. They should be simultaneously considered to elucidate IRES-mediated translational regulation mechanisms. According to the known cellular IRES functions, IRESs reveal complicated interplay with the data of uORFs around IRES sequences, ITAF-binding targets, the IRES structural forms, conservation of the functional sequences and the TIS triplets around IRESs. Therefore, to understand IRES-mediated gene regulation and deduce testable molecular mechanisms, there is an urgent need to articulate the interconnections between IRES elements and these translation initiation–related features. While many of these translation regulation features have been measured and studied, most of the resources are still fragmentary over diverse literature or supplementary repositories. Some of the valuable experimental results require tedious data pre-processing to be used in a comparative analysis. Furthermore, current bicistronic reporter assays for discovering novel IRES elements require series of experiments to rule out the noises from cryptic promoters or spicing activities (<a href=\"#R12\" rid=\"R12\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">12</a>), leading to the time-consuming, small-scale and labor-intensive nature of these experimental approaches. Systematic identification of IRES elements currently resorts mainly to <em>in silico</em> methods to first narrow down the search space. As a consequence, studying IRES in humans requires the following two advances: (i) evaluate the functionality of the <em>in silico</em> predicted putative IRESs (<a href=\"#R13\" rid=\"R13\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">13</a>, <a href=\"#R14\" rid=\"R14\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">14</a>) and (ii) integrate IRES elements and translation initiation–related information to elucidate and propose testable regulatory mechanisms for each identified IRES.</p><p id=\"idm139637302710560\" class=\"p p-last\">The complicated connections between IRES elements and translation initiation–related features bring about an immediate demand to construct an integrative IRES database. This database should perform a systematic transcriptome-wide IRES identification and integrate the comprehensive translational regulatory features to help biologists convey researches on IRES-driven translational regulation. In this research, we constructed a database called Human IRES Atlas to accomplish these goals. Human IRES Atlas is designed for easier understanding and evaluating the functionality of IRES candidates. It aims to reduce the long computing time of large-scale IRES screening on the human transcriptome and integrate the translation-related information for better hypothesis generation and evaluation. We first performed comprehensive putative IRES searching on the human transcriptome using currently available IRES prediction tools. To assess the functionality of the putative IRES, we considered their sequence conservation, structural RNA–protein interaction (RPI) scores and conditional translation efficiency. For elucidating the complex connections between IRES elements and translation initiation–related features, we integrated eight types of datasets that probe vital translation initiation information to help biologists infer possible IRES-mediated translational regulatory mechanisms. For analyzing the interconnections between IRES and translation initiation–related features for individual gene transcripts, an easy-to-use interface and interaction maps were constructed. Human IRES Atlas provides the functionality to search for an IRES of interest or browse the human transcriptome for putative IRES lists satisfying the user-customized filtering criteria. In summary, this database is designed in an integrative manner for simultaneous consideration of putative IRESs and translation initiation features. Many critical observations and testable mechanism hypotheses can be retrieved from this integration. Human IRES Atlas is available online at <a href=\"http://cobishss0.im.nuk.edu.tw/Human_IRES_Atlas/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://cobishss0.im.nuk.edu.tw/Human_IRES_Atlas/</a>.</p></div><div id=\"s2\" class=\"tsec sec\"><div class=\"goto jig-ncbiinpagenav-goto-container\"><a class=\"tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading\" href=\"#\" title=\"Go to other sections in this page\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Go to:</a></div><h2 class=\"head no_bottom_margin ui-helper-clearfix\" id=\"s2title\">Construction and contents</h2><div id=\"s2-s1\" class=\"sec sec-first\"><h3 id=\"s2-s1title\">Construction of Human IRES Atlas</h3><p id=\"idm139637302706080\" class=\"p p-first\">Human IRES Atlas deposits systematically identified putative IRES elements and integrates comprehensive translation initiation–related features to assist the study of IRES-driven translational regulation in humans. The overall database contents and construction can be roughly divided into four parts (see <a href=\"/pmc/articles/PMC8094437/figure/F1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"F1\" rid-ob=\"ob-F1\" co-legend-rid=\"lgnd_F1\"><span>Figure 1</span></a>). First, the transcriptome-wide putative human IRES screening was performed (<a href=\"/pmc/articles/PMC8094437/figure/F1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"F1\" rid-ob=\"ob-F1\" co-legend-rid=\"lgnd_F1\"><span>Figure 1a</span></a>). Currently available and suitable IRES prediction tools were used to identify the putative IRES elements in the human transcriptome. To evaluate their potential as functional IRESs, three functionality tests were conducted for each of the putative cellular IRES elements: sequence conservation (among different mammalian species), structural RPI scores and conditional translation efficiency (<a href=\"/pmc/articles/PMC8094437/figure/F1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"F1\" rid-ob=\"ob-F1\" co-legend-rid=\"lgnd_F1\"><span>Figure 1b</span></a>). Users can assess the functionality of the putative IRES for the queried gene based on these computed False Discovery Rate (FDR)-corrected test <em>P</em>-values. Third, eight genres of translation initiation–related datasets were collected and integrated into the database to facilitate biologists to form testable mechanism hypotheses for IRES-driven translational regulation (<a href=\"/pmc/articles/PMC8094437/figure/F1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"F1\" rid-ob=\"ob-F1\" co-legend-rid=\"lgnd_F1\"><span>Figure 1c</span></a>). In the final stage of database construction, an easy-to-use interface to query and browse the potential IRESs with the associated translation initiation features was constructed (<a href=\"/pmc/articles/PMC8094437/figure/F1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"F1\" rid-ob=\"ob-F1\" co-legend-rid=\"lgnd_F1\"><span>Figure 1d</span></a>). A transcript detail page that presents the complicated connections between IRES elements and translation initiation features can be found for the selected transcript. Testable functional hypotheses of the putative IRESs identified within the transcript can be inferred from the IRES structure conformation and other probed translation initiation–related features in the detail page. Overall, Human IRES Atlas aims to deposit transcriptome-wide putative human IRESs and articulate the complicated interplay between IRESs and translation initiation features. This integration helps suggest testable molecular mechanism hypotheses for subsequent IRES-mediated translation regulation experiments.</p><!--fig ft0--><!--fig mode=article f1--><div class=\"fig iconblock whole_rhythm\" id=\"F1\" co-legend-rid=\"lgnd_F1\"><a href=\"/pmc/articles/PMC8094437/figure/F1/\" target=\"figure\" rid-figpopup=\"F1\" rid-ob=\"ob-F1\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><div data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139637299818528\" class=\"figure\"><img loading=\"lazy\" class=\"fig-image\" alt=\"An external file that holds a picture, illustration, etc.\\nObject name is baab025f1.jpg\" title=\"An external file that holds a picture, illustration, etc.\\nObject name is baab025f1.jpg\" src=\"/pmc/articles/PMC8094437/bin/baab025f1.jpg\"></div></a><div id=\"largeobj_idm139637299818528\" class=\"largeobj-link align_right\" style=\"display: none\"><a href=\"/pmc/articles/PMC8094437/figure/F1/\" target=\"figure\" rid-figpopup=\"F1\" rid-ob=\"ob-F1\"></a><a target=\"object\" rel=\"noopener\" href=\"/pmc/articles/PMC8094437/figure/F1/?report=objectonly\">Open in a separate window</a></div><div class=\"icnblk_cntnt\" id=\"lgnd_F1\"><div><a class=\"figpopup\" href=\"/pmc/articles/PMC8094437/figure/F1/\" target=\"figure\" rid-figpopup=\"F1\" rid-ob=\"ob-F1\">Figure 1.</a></div><!--caption a7--><div class=\"caption\"><p id=\"idm139637299818880\">Overview of the Human IRES Atlas database.</p></div><div class=\"half_rhythm align_right attrib\">The construction of Human IRES Atlas can be divided into four different parts. First, the systematic transcriptome-wide IRES identification on all human transcripts was performed based on the evaluated screening tools (part a). The IRES functionality from sequence conservation, structural RPI score and conditional translation efficiency was then estimated for each identified putative IRES (part b). To further provide potential translational regulation information, eight types of datasets were collected and integrated into the repository to assist the deduction of IRES-mediated translational regulation mechanism hypotheses (part c). Finally, an easy-to-use query interface and a transcript detail page with translation initiation interaction maps were implemented to help biologists infer testable hypotheses for subsequent IRES-driven translation initiation research experiments (part d).</div></div></div></div><div id=\"s2-s2\" class=\"sec\"><h3 id=\"s2-s2title\">Transcriptome-wide human IRES screening</h3><p id=\"idm139637299815888\" class=\"p p-first\">We first performed transcriptome-wide IRES element identification on all human 5ʹ UTRs. The human 5ʹ UTR sequences were downloaded from the UCSC Genome Browser database (<a href=\"#R15\" rid=\"R15\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">15</a>) (Dec. 2013 assembly of the human genome, hg38). 53\\u2009971 messenger RNA (mRNA) transcripts for 19\\u2009367 genes were available in total. The sequences before the coding sequence (CDS) regions of each of these transcripts were defined as the 5ʹ UTR sequences. Based on the defined 5ʹ UTR sequences, currently available computational IRES scanning algorithms were evaluated to be used in this research. Five IRES prediction tools [IRESfinder (<a href=\"#R16\" rid=\"R16\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">16</a>), PatSearch (<a href=\"#R17\" rid=\"R17\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">17</a>), RNAz (<a href=\"#R18\" rid=\"R18\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">18</a>), IRESPred (<a href=\"#R19\" rid=\"R19\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">19</a>) and IRESpy (<a href=\"#R20\" rid=\"R20\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">20</a>)] are still currently available. To evaluate these tools, we collected a ground truth dataset with verified IRES sequences as the positive set and random sequences as the negative set. The positive set of 190 known IRES elements is obtained from IRESbase (<a href=\"#R21\" rid=\"R21\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">21</a>) and the review work of Baird <em>et al.</em> (<a href=\"#R22\" rid=\"R22\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">22</a>). Since it is known that house-keeping genes depend mainly on cap-dependent translation initiation (<a href=\"#R23\" rid=\"R23\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">23</a>), we randomly sampled 190 sequences from the 5ʹ UTRs of the house-keeping genes annotated by Eisenberg and Levanon (<a href=\"#R24\" rid=\"R24\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">24</a>) to be the negative set. We forced the negative set to be of the same sequence length distribution as the positive set. Using the receiver operating characteristic curve technique (see <a href=\"/pmc/articles/PMC8094437/figure/F2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"F2\" rid-ob=\"ob-F2\" co-legend-rid=\"lgnd_F2\"><span>Figure 2</span></a>), we can see that RNAz and IRESPred have performance similar to random guessing, and IRESfinder, IRESpy and PatSearch have area under curve larger than 0.5. Based on this analysis, only IRESpy, PatSearch and IRESfinder are suitable for obtaining potential IRES elements in 5ʹ UTRs. Therefore, IRESpy, PatSearch and IRESfinder were applied in the screening of possible IRESs in 5ʹ UTRs. The thresholds used in the database construction are 0.7 and 0.9 for IRESpy and IRESfinder, respectively.</p><!--fig ft0--><!--fig mode=article f1--><div class=\"fig iconblock whole_rhythm\" id=\"F2\" co-legend-rid=\"lgnd_F2\"><a href=\"/pmc/articles/PMC8094437/figure/F2/\" target=\"figure\" rid-figpopup=\"F2\" rid-ob=\"ob-F2\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><div data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139637335825392\" class=\"figure\"><img loading=\"lazy\" class=\"fig-image\" alt=\"An external file that holds a picture, illustration, etc.\\nObject name is baab025f2.jpg\" title=\"An external file that holds a picture, illustration, etc.\\nObject name is baab025f2.jpg\" src=\"/pmc/articles/PMC8094437/bin/baab025f2.jpg\"></div></a><div id=\"largeobj_idm139637335825392\" class=\"largeobj-link align_right\" style=\"display: none\"><a href=\"/pmc/articles/PMC8094437/figure/F2/\" target=\"figure\" rid-figpopup=\"F2\" rid-ob=\"ob-F2\"></a><a target=\"object\" rel=\"noopener\" href=\"/pmc/articles/PMC8094437/figure/F2/?report=objectonly\">Open in a separate window</a></div><div class=\"icnblk_cntnt\" id=\"lgnd_F2\"><div><a class=\"figpopup\" href=\"/pmc/articles/PMC8094437/figure/F2/\" target=\"figure\" rid-figpopup=\"F2\" rid-ob=\"ob-F2\">Figure 2.</a></div><!--caption a7--><div class=\"caption\"><p id=\"idm139637292687488\">ROC curve analysis for currently available IRES prediction tools on 5ʹ UTRs.</p></div><div class=\"half_rhythm align_right attrib\">We used the ROC curve analysis technique to evaluate the results of different IRES prediction tools. From the ROC curve, RNAz and IRESPred have performance similar to random guessing. IRESfinder, IRESpy and PatSearch have AUC larger than 0.5. Based on this analysis, only IRESpy, PatSearch and IRESfinder are suitable for obtaining potential IRES elements in 5ʹ UTRs.</div></div></div><p id=\"idm139637335823904\">Currently, most computational IRES identification tools assumed input sequences to be of medium lengths (&lt;174 bps) in their model training process (<a href=\"#R25\" rid=\"R25\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">25</a>). Hence, in this research, we applied the sliding window approach proposed by Zhao <em>et al.</em> (<a href=\"#R16\" rid=\"R16\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">16</a>) to systematically identify transcriptome-wide IRES elements. We first divide the 5ʹ UTR sequence into segments of 174 nts long with a step size of 10 nts between consecutive overlapping segments. Each of these fragments was then fed separately into different IRES identification tools to calculate the probability of the sequence to have a potential functional IRES element. In the last stage of the IRES scanning process, consecutive potential IRES elements identified by the same IRES prediction tool were concatenated to form the final functional IRES elements. 17\\u2009689 IRES sequence segments were identified in human 5ʹ UTRs using these three IRES screening tools.</p><p id=\"idm139637335820416\">It is known that the false discovery rate of the prediction process can be reduced if a potential IRES sequence is identified by more than one IRES screening tool (<a href=\"#R13\" rid=\"R13\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">13</a>). Hence to decrease the false positives in IRES screening, we adopted the nested minimal sequence definition to cross-reference the identified IRESs from different screening tools. For a group of nested potential IRES elements, we classified these IRESs into nested sets and took the minimal sequences in the nested sets as the representative IRES sequence segments. Among the 17\\u2009689 IRES elements identified by the three IRES screening tools, 17\\u2009013 minimal IRES elements in 5ʹ UTRs were cross-referenced and deposited in Human IRES Atlas.</p><p id=\"idm139637335818544\" class=\"p p-last\">Although most well-known cellular IRESs were found in the 5ʹ UTRs, some IRESs in CDSs and 3ʹ UTRs were also reported to drive the translation of novel proteins or protein isoforms (<a href=\"#R26\" rid=\"R26\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">26–28</a>). IRES screening experiments revealed that 3ʹ UTRs also contained many IRESs (<a href=\"#R12\" rid=\"R12\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">12</a>). Therefore, we have also extended the Human IRES Atlas database to include the potential IRES elements in CDSs (including the main ORF start codon) and 3ʹ UTRs of mRNAs. In the IRES identification beyond 5ʹ UTRs, only IRESpy was used. For PatSearch and IRESfinder, the designed features and the training process were performed on only 5ʹ UTR sequences. Thus, these two tools are not suitable for the IRES identification process on regions other than 5ʹ UTRs. Since no existing tool was trained on Long non-coding RNA (lncRNAs), it is currently unavailable to predict IRESs on lncRNAs and will be carried out when further advances are made in model construction. Extra 1868 IRESs on the CDSs/3ʹ UTRs were included. In total, there are 18\\u2009881 IRESs deposited in Human IRES Atlas.</p></div><div id=\"s2-s3\" class=\"sec\"><h3 id=\"s2-s3title\">Data collection and pre-processing of translation initiation feature data</h3><p id=\"idm139637335814688\" class=\"p p-first\">Eight genres of translation initiation feature data were gathered and integrated into the database to assist biologists in understanding the interplay between IRES elements and translation initiation. The integration also helps users propose testable molecular mechanism hypotheses for IRES-driven translational regulation. The following contents were collected and integrated in this repository: translation initiation site (TIS) probing experiments, uORF sequences, sequence conservation scores (among mammalian species), ribosome profiling data, structure information of the IRES elements, experimentally identified ITAF target sequences, IRES activity bicistronic measurements and literature-curated IRES verification experiment evidence. Details of the deposited datasets are described in the following subsections.</p><p id=\"idm139637290219904\" class=\"p\">In Human IRES Atlas, the putative human IRES elements were systematically identified by computational methods. To form reliable IRES-driven translation initiation mechanism hypotheses, we are required to understand the functionality of these putative IRES identifications. Recent studies on cellular IRES have revealed that known human IRES elements are rich in their GC contents to form some special ITAF interacting sequences or structures (<a href=\"#R11\" rid=\"R11\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">11</a>, <a href=\"#R22\" rid=\"R22\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">22</a>). However, there are still some IRESs that are characterized by their high AU-rich contents (<a href=\"#R29\" rid=\"R29\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">29</a>). These AU-rich IRES elements interact with specific ITAF chaperones that shape the RNA structure into forms that facilitate translation initiation (<a href=\"#R30\" rid=\"R30\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">30</a>). Functional sequences in the genome are usually believed to be conserved among different species (<a href=\"#R31\" rid=\"R31\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">31</a>, <a href=\"#R32\" rid=\"R32\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">32</a>). Based on the understanding of cellular IRES elements, three functionality tests are also performed with the collected data in this database: conservation, structural RPI scores and conditional translation efficiency between stressed and normal states. Note that the thresholds of the functional tests should be treated as statistical type I error control for the functional aspects. It should not be misinterpreted as the prediction confidence since it is reported that some human IRESs are not conserved in mammals or lack complete structural forms (<a href=\"#R30\" rid=\"R30\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">30</a>).</p><div id=\"s2-s3-s1\" class=\"sec\"><p></p><h4 id=\"s2-s3-s1title\" class=\"inline\">Data of sequence conservation between mammalian species </h4><p id=\"idm139637290211552\" class=\"p p-first-last\">Functional elements are often evolutionary conserved among different closely related species (<a href=\"#R31\" rid=\"R31\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">31</a>, <a href=\"#R32\" rid=\"R32\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">32</a>). Hence when the specified sequences are to carry out vital cellular functions, they tend to be significantly more conserved among different related species (<a href=\"#R33\" rid=\"R33\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">33</a>). The base-by-base sequence conservation level can be captured by the phastCons conservation scores (<a href=\"#R34\" rid=\"R34\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">34</a>). The phastCons conservation scores were based on multiple alignments of human and other 19 mammalian genomes (baboon, bushbaby, bonobo, chimp, crab-eating macaque, gibbon, golden snub-nosed monkey, gorilla, green monkey, marmoset, mouse lemur, orangutan, proboscis monkey, rhesus, squirrel monkey, tarsier, tree shrew, mouse and dog). We mapped the conservation scores in the genome to transcripts using the transcript mapping annotated by the UCSC Genome Annotation database (<a href=\"#R15\" rid=\"R15\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">15</a>). To test the conservation significance of a putative IRES element, we used the rank-sum test (<a href=\"#R35\" rid=\"R35\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">35</a>) to calculate the <em>P</em>-value against the hypothesis that an IRES element of interest is more conserved than its corresponding transcript in phastCons conservation scores. Multiple hypotheses testing bias was corrected by the FDR control approach for the calculated <em>P</em>-values among all IRES elements.</p></div><div id=\"s2-s3-s2\" class=\"sec\"><p></p><h4 id=\"s2-s3-s2title\" class=\"inline\">Ribosome profiling data and conditional translation efficiency </h4><p id=\"idm139637335163744\" class=\"p p-first-last\">Translation initiation in eukaryotes proceeds in a series of complicated steps that lead to the final ribosome complex formation. Hence, the ribosome-binding sites located within the 5ʹ UTR sequences under different cellular conditions can reveal important translational regulation hints for active translation programs (<a href=\"#R36\" rid=\"R36\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">36</a>). The experimental procedure called ribosome profiling (ribo-seq) can help identify the transcriptome-wide in vivo ribosome existence and can be used to indicate whether the given sequences show higher translation efficiency in the stress condition. To help users obtain the conditional translation efficiency for a given transcript, we collected the ribosome profiling data of tunicamycin-treated (SRX870808 and SRX870809) and untreated (SRX870804 and SRX870805) HEK293T cells (<a href=\"#R37\" rid=\"R37\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">37</a>). Tunicamycin is a toxin that blocks N-linked glycosylation of Endoplasmic Reticulum (ER)-residing proteins and induces cell depth, leading to ER stress. ER stress induces translational reprogramming that favors alternative translation such as IRES mechanisms. From these ribo-seq experiment results, users can recognize if the transcript is differentially expressed between normal and stress conditions. By simultaneously considering the ITAF data and the ribo-seq results, users can postulate if the stress response of the transcript is mainly regulated by IRES-mediated (<a href=\"#R38\" rid=\"R38\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">38</a>), uORF-mediated (<a href=\"#R39\" rid=\"R39\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">39</a>) or IRES–uORF-interplay (<a href=\"#R40\" rid=\"R40\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">40</a>) translation initiation. To integrate the ribosome profiling results, we first used the NCBI SRA Toolkit to download the sequencing results and then transformed the results into the fastq format. We followed the peak calling procedure used in the HRPDviewer (<a href=\"#R41\" rid=\"R41\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">41</a>) to obtain the transcript nucleotide-level reads for both conditions. First, the adaptor linker sequences or poly-(A) tails of the short reads are trimmed by Cutadapt (<a href=\"#R42\" rid=\"R42\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">42</a>). These trimmed reads are then mapped to the hg38 reference transcriptome using RSEM (<a href=\"#R43\" rid=\"R43\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">43</a>) with the bowtie engine. Finally, the reads per million mapped reads (RPM) values are calculated on all transcript nucleotide positions. By performing the rank-sum test on the RPM values between the stressed and normal conditions, users can get the idea if the identified IRES is stress-induced. Multiple hypotheses testing bias was corrected by the FDR control approach for the calculated <em>P</em>-values among all IRES elements.</p></div><div id=\"s2-s3-s3\" class=\"sec\"><p></p><h4 id=\"s2-s3-s3title\" class=\"inline\">RPI scores and structural information within IRESs </h4><p id=\"idm139637294316912\" class=\"p p-first\">Structural features have been found in several well-studied cellular IRESs (<a href=\"#R44\" rid=\"R44\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">44</a>, <a href=\"#R45\" rid=\"R45\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">45</a>). Although the structure–function relationship seems to be less rigid in cellular IRESs than in the viral ones, current investigation of known cellular IRESs has indicated that at least some structural modules play important roles with other ITAFs in cellular IRES-mediated translation initiation (<a href=\"#R11\" rid=\"R11\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">11</a>, <a href=\"#R30\" rid=\"R30\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">30</a>). Hence, studying the potential regulatory IRES structures can help infer the potential functional molecular mechanisms. 12 currently available RNA secondary structure prediction tools [RNAFold (<a href=\"#R46\" rid=\"R46\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">46</a>), RNALfold (<a href=\"#R47\" rid=\"R47\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">47</a>), RNAprob (<a href=\"#R48\" rid=\"R48\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">48</a>), MaxExpect (<a href=\"#R49\" rid=\"R49\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">49</a>), RNAalifold (<a href=\"#R50\" rid=\"R50\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">50</a>), TurboFold (<a href=\"#R51\" rid=\"R51\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">51</a>), SPARSE (<a href=\"#R52\" rid=\"R52\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">52</a>), MXSCARNA (<a href=\"#R53\" rid=\"R53\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">53</a>), aliFreeFold (<a href=\"#R54\" rid=\"R54\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">54</a>), LocaRNA (<a href=\"#R55\" rid=\"R55\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">55</a>), comRNA (<a href=\"#R56\" rid=\"R56\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">56</a>) and RME (<a href=\"#R57\" rid=\"R57\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">57</a>)] were used to comprehensively identify the candidate secondary structures of the putative IRES elements. We further implemented an RPI score to estimate the given IRES’s tendency to interact with proteins. This score is derived from a comparison between the queried sequence and verified RPIs curated in the RNAInter database (<a href=\"#R58\" rid=\"R58\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">58</a>).</p><p id=\"idm139637291893728\" class=\"p p-last\">Structurally similar non-coding RNA (ncRNA) usually carry out similar roles in RNA–protein complexes. Thus, proteins showing interactions with them tend to work in similar biological processes. For estimating the tendency of a given RNA candidate structure to interact with a protein, we resorted to the human ncRNAs having verified RPIs curated in the RNAInter database (<a href=\"#R58\" rid=\"R58\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">58</a>). Specifically, by BLASTing these ncRNAs to the BRAlibaseII structure benchmark dataset (<a href=\"#R59\" rid=\"R59\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">59</a>), we collected a reference set with known structures and their corresponding interacting protein set. To obtain the potential IRES structural forms, the abovementioned 12 different RNA structure prediction tools were integrated in this research to produce candidate structures for a given IRES sequence. For each candidate structure, an ncRNA set consisting of the top 10 structurally similar ncRNAs from the constructed reference set was extracted. The similarity between a candidate structure and a reference ncRNA structure was measured by the general RNA structure distance defined by Jiang <em>et al.</em> (<a href=\"#R60\" rid=\"R60\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">60</a>). The defined general RNA structure distance considers both sequence mismatches and structure pairing mismatches. From the extracted structurally similar ncRNA set of a candidate structure, proteins interacting with these structurally similar ncRNAs were listed based on the data from the RNAInter database. On the listed protein set, we performed the protein interaction prevalence test and the ontology enrichment test as defined in previous researches (<a href=\"#R61\" rid=\"R61\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">61</a>). In brief, the protein interaction prevalence test evaluates if the protein set forms an enriched protein complex, and the ontology enrichment test checks if the protein set has statistically significant annotated functional ontology terms. The enrichment <em>P</em>-values are defined as the RPI scores. A given IRES’s candidate structure with the lowest RPI score among its structure predictions was selected as its representative IRES structure. Lower RPI scores of the predicted structure indicate that the queried sequence is structurally similar to ncRNAs selected from the reference RNA collection and is more likely to interact with a protein complex that regulates a particular cellular function(s). The RPI scores among all putative IRESs in Human IRES Atlas were corrected by the FDR multiple hypothesis control approach.</p></div><div id=\"s2-s3-s4\" class=\"sec\"><p></p><h4 id=\"s2-s3-s4title\" class=\"inline\">Data of ITAF-binding events </h4><p id=\"idm139637291885680\" class=\"p p-first-last\">Cellular IRESs in eukaryotes are reported to cooperate with specialized RNA-binding protein called ITAFs in the regulation of the ribosome translation machinery (<a href=\"#R9\" rid=\"R9\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">9</a>, <a href=\"#R23\" rid=\"R23\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">23</a>). By acting as molecular chaperones or RNA structure remodelers, ITAFs identify particular sequences or structures in IRES elements to help facilitate the translation initiation factor recruitment and initiation complex formation (<a href=\"#R38\" rid=\"R38\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">38</a>). Hence, investigating the connections between ITAF-binding sites and IRES locations can help understand the potential cellular translational regulation mechanisms. Various cellular ITAFs have been reported in humans. Some of their binding targets have been experimentally probed through the transcriptome-wide cross-linking and immunoprecipitation (CLIP) or the photoactivatable-ribonucleoside enhanced cross-linking and immunoprecipitation (PAR-CLIP) techniques at a nucleotide-level resolution. We collected the CLIP and PAR-CLIP-binding events for the following ITAFs from the doRiNA database (<a href=\"#R62\" rid=\"R62\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">62</a>) and remapped them into the hg38 transcriptome: 4\\u2009001\\u2009271 FUS-binding sites, 4899 hnRNPC-binding sites, 30\\u2009593 hnRNPL-binding sites, 621\\u2009116 HuR/ELAVL1-binding sites, 806 p54nrb-binding sites and 1\\u2009367\\u2009825 eIF4A-3-binding sites. Simultaneously considering the predicted IRES elements, the identified ITAF-binding events and the potential IRES structural forms can help users infer possible IRES-mediated translational regulation molecular mechanisms.</p></div><div id=\"s2-s3-s5\" class=\"sec\"><p></p><h4 id=\"s2-s3-s5title\" class=\"inline\">Data of translation initiation site probing data </h4><p id=\"idm139637371894240\" class=\"p p-first-last\">The translation machinery is completely formed to start polypeptide chain synthesis on special triplets called TISs. Canonical translation initiation uses the AUG triplet as the normal start codon. However, many non-AUG triplets have been shown to provide alternative starting sites for initiating uORF translation or the IRES-driven initiation (<a href=\"#R63\" rid=\"R63\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">63</a>). Hence, the TIS signals around or within IRES elements can provide translation initiation evidence that supports the regulatory effects of these IRES elements. Two different types of TIS data were collected and mapped to human transcript 5ʹ UTRs. The first type that indicates translation initiation is the TIS peaks probed by sequencing techniques. The high-throughput transcriptome-wide TIS positioning for humans is collected from the global translation initiation sequencing (GTI-seq) performed in the work by Lee <em>et al</em>. (<a href=\"#R64\" rid=\"R64\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">64</a>). In their work, two different ribosome E-site translation inhibitors, namely the cycloheximide (CHX) and lactimidomycin (LTM), were used in HEK293 cells. CHX reports elongating ribosomes by blocking translation elongation and LTM probes initiating ribosomes since it preferentially binds to the TIS, while the elongating ribosomes run off the mRNAs. We followed the same analysis procedure used by TISDB (<a href=\"#R65\" rid=\"R65\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">65</a>) to map their sequencing results to the human hg38 transcriptome. In brief, the GTI-seq data were first mapped to the hg38 genome and transcriptome by Tophat (<a href=\"#R66\" rid=\"R66\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">66</a>). The 13th nucleotide of a uniquely mapped read was identified as the start codon recognized in translation initiation. The initiation sites with statistically significant read counts were then determined using the zero-truncated binomial negative model. In total, 11\\u2009395 TIS peaks were identified in the 5ʹ UTRs of the transcripts. These experimentally probed TISs are called experimentally probed translation initiation site (eTIS) in this database. The eTIS experimental contexts suggest either uORF start codons resulting from cap-dependent scanning or cap-independent IRES activity under growth conditions in these particular cells. The second group of translation initiation data is the non-canonical start codon predicted by the PreTIS software (<a href=\"#R67\" rid=\"R67\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">67</a>). PreTIS constructed from a linear regression approach was trained on different ribosome profiling datasets, significant Position Weight Matrix (PWM) and biologically motivated features. Using the PreTIS tool, the potential non-canonical TISs were generated. We adopted a confidence level of 0.9 in this research to obtain 13\\u2009900 predicted non-canonical TIS within the human 5ʹ UTR sequences. The collected non-canonical TIS is called nTIS in this database.</p></div><div id=\"s2-s3-s6\" class=\"sec\"><p></p><h4 id=\"s2-s3-s6title\" class=\"inline\">Data of uORF sequences </h4><p id=\"idm139637301550112\" class=\"p p-first-last\">Researchers have reported that the stalling of ribosomes in the uORF regions can regulate IRES-driven translation initiation (<a href=\"#R68\" rid=\"R68\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">68</a>). uORF translations can mediate 5ʹ UTR structural remodeling to conform the regulated IRES element into the active form (<a href=\"#R69\" rid=\"R69\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">69</a>) or unfold the IRES structure to repress IRES activity (<a href=\"#R40\" rid=\"R40\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">40</a>). Hence, considering the connections between IRES elements and uORFs within 5ʹ UTRs could help indicate the potential regulatory mechanisms of IRESs. We adopted the analysis procedure proposed by Ye <em>et al</em>. (<a href=\"#R70\" rid=\"R70\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">70</a>) to obtain the uORFs in each of the transcripts. uORFs were identified by scanning the 5ʹ UTRs of the transcripts to obtain the sequences that start with the start codon AUG and continue with a three-nucleotide reading frame until the stop codon is encountered. In total, 131\\u2009612 uORFs starting from human 5ʹ UTRs were collected in this database.</p></div><div id=\"s2-s3-s7\" class=\"sec\"><p></p><h4 id=\"s2-s3-s7title\" class=\"inline\">Evidence of IRES activity </h4><p id=\"idm139637301544256\" class=\"p p-first-last\">Recently, Weingarten-Gabbay <em>et al</em>. (<a href=\"#R12\" rid=\"R12\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">12</a>) devised a systematic bicistronic assay coupled with fluorescence-activated cell sorting to uncover the human transcript sequences with cap-independent translation activity. In their work, they measured the enhanced green fluorescent protein (eGFP) expression scores as the IRES-mediated translation activity in the prepared oligo pEF1 vector library consisting of the virus genome, human genome and reported IRES mutations. They performed the bicistronic experiments for 6946 5ʹ UTRs of 5058 genes in humans, about one-fourth of all human genes. These IRES activity values can indicate the existence of IRES elements for the given sequence segments. Following the experimental design settings of Weingarten-Gabbay <em>et al</em>., the IRES activity was designated as background expression when the oligos were not detected in adjacent eGFP bins during GFP sorting but had &gt;100 reads in the full mRFP-oligos-eGFP library, indicating these background oligos did not convey cap-independent translation signals in the experiments. The oligos were assigned ‘NaN’ if they got &lt;100 reads in the full monomeric red fluorescent protein (mRFP)-oligos-eGFP library, representing a lack of identification confidence. Other normal real numbers represent the extent of IRES activity of the regulatory sequences. When values larger than the background signal are reported for a given identified IRES, subsequent verification experiments to rule out cryptic promoters and splicing activity can be designed to confirm the existence of IRES functionality. In total, we collected and mapped the probed human oligo sequences with IRES activity measurement to 39\\u2009700 transcript segments (E-value\\u2009&lt;1E-6, identity percentage\\u2009&gt;85%, query cover\\u2009&gt;85%, promoter activity\\u2009&lt;0.2 and splicing activity &gt;−2.5) in this database.</p></div><div id=\"s2-s3-s8\" class=\"sec sec-last\"><p></p><h4 id=\"s2-s3-s8title\" class=\"inline\">Collection of literature-curated IRESs </h4><p id=\"idm139637289292128\" class=\"p p-first-last\">Since the first discovery of the viral IRES sequence in poliovirus, various researchers have studied and described diverse cellular IRES elements in the literature. These results have been collected in various review papers. Some databases were also constructed to deposit IRES verification experiments. However, all of these databases either stopped updating the contents or were no longer available to the public. In this research, we collected those literature-verified human IRESs from the IRESite database (<a href=\"#R71\" rid=\"R71\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">71</a>), the IRESbase database (<a href=\"#R21\" rid=\"R21\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">21</a>) and the review works of Baird <em>et al</em>. (<a href=\"#R22\" rid=\"R22\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">22</a>), Leppek <em>et al</em>. (<a href=\"#R11\" rid=\"R11\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">11</a>) and Stoneley <em>et al</em>. (<a href=\"#R30\" rid=\"R30\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">30</a>). These verified IRES sequences were mapped back to the transcripts of the reported IRES-regulating genes by BLASTN (E-value\\u2009&lt;1E-6, identity percentage\\u2009&gt;85% and query cover\\u2009&gt;85%). In total, 659 IRESs with experimental evidence were retrieved from IRESite, IRESbase and the literature.</p></div></div><div id=\"s2-s4\" class=\"sec sec-last\"><h3 id=\"s2-s4title\">Implementation of the Human IRES Atlas web interface</h3><p id=\"idm139637289284176\" class=\"p p-first-last\">IRES identification and the translation initiation features data pre-processing were carried out using Python. The web interface of the database is constructed using the PHP CodeIgniter MVC framework. IRES data and the related datasets were deposited through MySQL.</p></div></div><div id=\"s3\" class=\"tsec sec\"><div class=\"goto jig-ncbiinpagenav-goto-container\"><a class=\"tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading\" href=\"#\" title=\"Go to other sections in this page\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Go to:</a></div><h2 class=\"head no_bottom_margin ui-helper-clearfix\" id=\"s3title\">Utility and discussion</h2><div id=\"s3-s1\" class=\"sec sec-first\"><h3 id=\"s3-s1title\">Database interface</h3><p id=\"idm139637303424224\" class=\"p p-first\">In Human IRES Atlas, four essential database functions are implemented to help select IRES elements of interest: (i) search function: users can search the putative IRESs for different transcripts of a given gene. (ii) Filter and browse function: users are allowed to browse the deposited transcriptome-wide human IRESs categorized by chromosomes or KEGG pathway annotations of genes regulated by them. The KEGG pathway annotation helps users identify the IRES elements that regulate genes involved in the same biological pathway. This criterion helps biologists check if a particular biological pathway consists of different potential IRES regulation mechanisms. To enable users to select only the putative IRESs with significant <em>q</em>-values for the three functional tests, we have further implemented the <em>q</em>-value filters in the browse page. Users can set the intended <em>q</em>-value threshold(s) for the functional test(s), and the database will show only the potential IRES elements with significant <em>q</em>-values. The resulting IRES list can be downloaded in the browse page. (iii) Transcript detail page: after selecting the transcript of interest in the query results, a detailed translation mechanism deduction page is provided. In this page, the detailed IRES summary and translation initiation–related features are integrated to help understand and infer the testable mechanism hypotheses for each IRES. (iv) Download function: the integrative IRES–translation initiation interaction map and the supplemented experimental datasets can be downloaded as plain text files for subsequent analyses. An easy-to-use web interface was built to facilitate these functions.</p><p id=\"idm139637303419888\">Users can type in a particular gene name or its aliases in the search page (<a href=\"/pmc/articles/PMC8094437/figure/F3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"F3\" rid-ob=\"ob-F3\" co-legend-rid=\"lgnd_F3\"><span>Figure 3a</span></a>). Users can also browse the database to find IRESs categorized by chromosomes and/or KEGG pathway annotations of genes regulated by them (<a href=\"/pmc/articles/PMC8094437/figure/F3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"F3\" rid-ob=\"ob-F3\" co-legend-rid=\"lgnd_F3\"><span>Figure 3b</span></a>). The putative IRESs and the summary of the corresponding translation initiation–related information are then presented in a tabular form (<a href=\"/pmc/articles/PMC8094437/figure/F3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"F3\" rid-ob=\"ob-F3\" co-legend-rid=\"lgnd_F3\"><span>Figure 3c</span></a>). For each transcript of the input gene, Human IRES Atlas summaries the brief transcript information, the putative IRES elements found on the transcripts, the number of prediction tools that specify this sequence to be a potential IRES, the functionality test results for each identified IRES element and finally the summaries of other translation initiation–related data observed within IRES regions. Three tests (conservation significance, RPI score significance and conditional translation efficiency significance) are provided to assess the functionality of each identified IRES. The summary of the translation initiation–related data matched to an IRES region includes the following information: (i) the number of eTIS (experimentally probed TIS) found in the upstream to the downstream 100 bps of the IRES element, (ii) the number of nTIS (non-canonical TIS) predicted in the upstream to the downstream 100 bps of the IRES element, (iii) the number of uORF observed in the upstream and downstream 100 bps of the IRES element, (iv) the number of ITAF-binding events that overlap and interact with the IRES element, (v) IRES activity measured for the IRES element and (vi) the number of literature IRES evidence that overlaps with the IRES prediction. Some IRES elements have been shown to use the <em>cis</em> uORFs and TISs to mediate their activity (<a href=\"#R45\" rid=\"R45\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">45</a>, <a href=\"#R69\" rid=\"R69\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">69</a>). The amino acid stress or the stalling ribosome near an IRES can trigger a structural switch of the IRES to an active conformation, leading to IRES-mediated translation initiation. Therefore, we included extra 100 bps upstream and downstream of an IRES when associating TISs and uORFs to it. For ITAFs, they usually bind and remodel IRES structures to set up the ribosome-binding platform (<a href=\"#R38\" rid=\"R38\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">38</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class=\"fig iconblock whole_rhythm\" id=\"F3\" co-legend-rid=\"lgnd_F3\"><a href=\"/pmc/articles/PMC8094437/figure/F3/\" target=\"figure\" rid-figpopup=\"F3\" rid-ob=\"ob-F3\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><div data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139637289424368\" class=\"figure\"><img loading=\"lazy\" class=\"fig-image\" alt=\"An external file that holds a picture, illustration, etc.\\nObject name is baab025f3.jpg\" title=\"An external file that holds a picture, illustration, etc.\\nObject name is baab025f3.jpg\" src=\"/pmc/articles/PMC8094437/bin/baab025f3.jpg\"></div></a><div id=\"largeobj_idm139637289424368\" class=\"largeobj-link align_right\" style=\"display: none\"><a href=\"/pmc/articles/PMC8094437/figure/F3/\" target=\"figure\" rid-figpopup=\"F3\" rid-ob=\"ob-F3\"></a><a target=\"object\" rel=\"noopener\" href=\"/pmc/articles/PMC8094437/figure/F3/?report=objectonly\">Open in a separate window</a></div><div class=\"icnblk_cntnt\" id=\"lgnd_F3\"><div><a class=\"figpopup\" href=\"/pmc/articles/PMC8094437/figure/F3/\" target=\"figure\" rid-figpopup=\"F3\" rid-ob=\"ob-F3\">Figure 3.</a></div><!--caption a7--><div class=\"caption\"><p id=\"idm139637289424800\">The search result page for the identified IRESs of a given gene.</p></div><div class=\"half_rhythm align_right attrib\">(a) The search mode: users can type in the gene of interest to search for the identified IRESs on the given gene transcripts. (b) The browse mode: users can filter out the IRESs categorized by their residing chromosomes, annotated KEGG pathways of the regulated genes and the thresholds of IRES functionality <em>q</em>-values. (c) The result table for the identified IRES elements in all transcript isoforms of the queried gene. In this table, three IRES functionality test results and other translational regulatory features for each identified IRES element are summarized. (d) The link to the detail page of the selected transcript.</div></div></div><p id=\"idm139637289421984\" class=\"p p-last\">Below the summary table of the translation initiation–related features, an interaction map between translation initiation–related features for the transcript can be viewed in this table for better data visualization (<a href=\"/pmc/articles/PMC8094437/figure/F3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"F3\" rid-ob=\"ob-F3\" co-legend-rid=\"lgnd_F3\"><span>Figure 3c</span></a>). For each transcript, one can click the ‘detail’ hyperlink (<a href=\"/pmc/articles/PMC8094437/figure/F3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"F3\" rid-ob=\"ob-F3\" co-legend-rid=\"lgnd_F3\"><span>Figure 3d</span></a>) to obtain the detail page that lists the complete information of the putative IRES elements, the visualized IRES–translation initiation interaction maps and the translational regulation information at a single nucleotide resolution. Both the genomic chromosomal coordinates and the transcript locations for different translation initiation–related information are listed in the detail page. Moreover, we have also provided the <em>.bed</em> files for the putative IRES track and different transcription-related information tracks. These files can facilitate users to integrate the application with the genome browser. In the last part, the implemented transcript detail page assists in forming molecular mechanism hypotheses for IRES-driven translation initiation. The walkthrough of the detail page is provided along with the case study.</p></div><div id=\"s3-s2\" class=\"sec\"><h3 id=\"s3-s2title\">Case study</h3><p id=\"idm139637289417616\" class=\"p p-first\">Human IRES Atlas aims to integrate the transcriptome-wide human IRESs and translational regulatory features. The integration provides ways to infer testable mechanism hypotheses for IRES-mediated translation regulation. One can use the IRES–translation initiation interaction map and the queried transcript’s detailed information to deduce such hypotheses. We discuss the IRES-mediated regulatory program of human FGF9 in detail to demonstrate the biological applicability of the constructed database. Human Fibroblast Growth Factor 9 (FGF9) is the gene that transcribes and translates to a protein that acts as an autocrine/paracrine growth factor in lung branching morphogenesis (<a href=\"#R72\" rid=\"R72\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">72</a>), bone development (<a href=\"#R73\" rid=\"R73\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">73</a>) and even cancer progression (<a href=\"#R74\" rid=\"R74\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">74</a>). It has been reported that the FGF9 mRNA is expressed in embryos while the FGF9 protein is identified to be of low level in most cells and is restrictedly expressed in specific cell organs (<a href=\"#R75\" rid=\"R75\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">75</a>). Hence, it is of interest to biologists that how the FGF9 transcripts are translationally regulated in human cells. In Human IRES Atlas, the FGF9 transcript map of IRES–translation initiation interactions provides clues for deducing the testable mechanism hypotheses. There is an identified overlapping IRES region (IRES2: 746.840) that was found by at least two IRES screening tools and is calculated to be conserved and significant in functional RPI scores (<em>q</em>\\u2009=\\u20092.83E-13*) (<a href=\"/pmc/articles/PMC8094437/figure/F4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"F4\" rid-ob=\"ob-F4\" co-legend-rid=\"lgnd_F4\"><span>Figure 4a and b</span></a>). In the FGF9 IRES–translation initiation interaction map, very low levels of TIS and ribosome occupancy signals were found (<a href=\"/pmc/articles/PMC8094437/figure/F4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"F4\" rid-ob=\"ob-F4\" co-legend-rid=\"lgnd_F4\"><span>Figure 4c</span></a>), suggesting the low translation level in normal conditions. Further, uORFs were discovered in proximity to these structurally significant IRES elements. Finally, HuR-binding signals were observed to be overlapped with the putative IRES. Details of the translation initiation–related data found around each IRES element are listed in the fourth part of the page (<a href=\"/pmc/articles/PMC8094437/figure/F4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"F4\" rid-ob=\"ob-F4\" co-legend-rid=\"lgnd_F4\"><span>Figure 4d</span></a>). A link to download the text files in the <em>.csv</em> format that contain the full information of the detail page is provided for users (<a href=\"/pmc/articles/PMC8094437/figure/F4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"F4\" rid-ob=\"ob-F4\" co-legend-rid=\"lgnd_F4\"><span>Figure 4e</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class=\"fig iconblock whole_rhythm\" id=\"F4\" co-legend-rid=\"lgnd_F4\"><a href=\"/pmc/articles/PMC8094437/figure/F4/\" target=\"figure\" rid-figpopup=\"F4\" rid-ob=\"ob-F4\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><div data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139637302421760\" class=\"figure\"><img loading=\"lazy\" class=\"fig-image\" alt=\"An external file that holds a picture, illustration, etc.\\nObject name is baab025f4.jpg\" title=\"An external file that holds a picture, illustration, etc.\\nObject name is baab025f4.jpg\" src=\"/pmc/articles/PMC8094437/bin/baab025f4.jpg\"></div></a><div id=\"largeobj_idm139637302421760\" class=\"largeobj-link align_right\" style=\"display: none\"><a href=\"/pmc/articles/PMC8094437/figure/F4/\" target=\"figure\" rid-figpopup=\"F4\" rid-ob=\"ob-F4\"></a><a target=\"object\" rel=\"noopener\" href=\"/pmc/articles/PMC8094437/figure/F4/?report=objectonly\">Open in a separate window</a></div><div class=\"icnblk_cntnt\" id=\"lgnd_F4\"><div><a class=\"figpopup\" href=\"/pmc/articles/PMC8094437/figure/F4/\" target=\"figure\" rid-figpopup=\"F4\" rid-ob=\"ob-F4\">Figure 4.</a></div><!--caption a7--><div class=\"caption\"><p id=\"idm139637302422240\">The transcript detail page showing the potential IRES-driven translation regulatory program of the given transcript.</p></div><div class=\"half_rhythm align_right attrib\">(a) The brief information table of the queried gene transcript. (b) The translation initiation–related feature summary table for the identified IRES elements within the queried gene transcript. (c) The IRES–translation initiation interaction map for the queried gene transcript. (d) The detail of the translation initiation–related feature data found for each IRES. (e) The download function to obtain the <em>.csv</em> or <em>.bed</em> formats of all information in the detail page.</div></div></div><p id=\"idm139637333813472\" class=\"p\">By careful consideration of the integrated information, we can deduce that FGF9 transcripts could be translationally regulated through the interplay between the structural IRES, uORF segments and ITAFs (see <a href=\"/pmc/articles/PMC8094437/figure/F5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"F5\" rid-ob=\"ob-F5\" co-legend-rid=\"lgnd_F5\"><span>Figure 5</span></a>). By the effort of molecular biologists, it is now verified that there is indeed a functional IRES structure near the end of the 5ʹ UTR of the FGF9 transcript. The uORF segments found around the IRES elements were shown to maintain the low protein expression level (<a href=\"#R40\" rid=\"R40\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">40</a>). The IRES structure induced under hypoxia recruits the ribosome to the CDS and activates the FGF9 protein synthesis with IRES–ITAF interactions. Although it is later found that the ITAF HuR may bind to the AU-rich elements in FGF9 transcripts (<a href=\"#R76\" rid=\"R76\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">76</a>), further studies and experiments should be carried out to figure out whether other ITAFs interact with these IRES elements. These findings conform to the hypotheses suggested by the constructed database. In conclusion, translation regulatory programs can be inferred with the facilitation of Human IRES Atlas.</p><!--fig ft0--><!--fig mode=article f1--><div class=\"fig iconblock whole_rhythm\" id=\"F5\" co-legend-rid=\"lgnd_F5\"><a href=\"/pmc/articles/PMC8094437/figure/F5/\" target=\"figure\" rid-figpopup=\"F5\" rid-ob=\"ob-F5\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><div data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139637333807680\" class=\"figure\"><img loading=\"lazy\" class=\"fig-image\" alt=\"An external file that holds a picture, illustration, etc.\\nObject name is baab025f5.jpg\" title=\"An external file that holds a picture, illustration, etc.\\nObject name is baab025f5.jpg\" src=\"/pmc/articles/PMC8094437/bin/baab025f5.jpg\"></div></a><div id=\"largeobj_idm139637333807680\" class=\"largeobj-link align_right\" style=\"display: none\"><a href=\"/pmc/articles/PMC8094437/figure/F5/\" target=\"figure\" rid-figpopup=\"F5\" rid-ob=\"ob-F5\"></a><a target=\"object\" rel=\"noopener\" href=\"/pmc/articles/PMC8094437/figure/F5/?report=objectonly\">Open in a separate window</a></div><div class=\"icnblk_cntnt\" id=\"lgnd_F5\"><div><a class=\"figpopup\" href=\"/pmc/articles/PMC8094437/figure/F5/\" target=\"figure\" rid-figpopup=\"F5\" rid-ob=\"ob-F5\">Figure 5.</a></div><!--caption a7--><div class=\"caption\"><p id=\"idm139637333808048\">Human IRES Atlas helps suggest testable IRES-mediated translational regulation mechanism hypotheses.</p></div><div class=\"half_rhythm align_right attrib\">For the FGF9 5ʹ UTR, the IRES–translation initiation interaction map suggests that (1) the transcript is of low translation level and (2) some regulatory program between the RPI interpretable IRESs, the nearby uORFs and some ITAFs may exist. From experimental literature evidence, we know that the uORF represses FGF9 protein synthesis in normal conditions. Some ITAFs bind to the IRES to start cap-independent translation under hypoxia, verifying the testable hypotheses suggested by the constructed database.</div></div></div></div><div id=\"s3-s3\" class=\"sec\"><h3 id=\"s3-s3title\">Comparison with related works</h3><p id=\"idm139637333805408\" class=\"p p-first\">Human IRES Atlas was created for two primary purposes: (i) to provide comprehensive and systematic IRES identifications in the human species and (ii) to integrate IRES elements and translation initiation–related information for biologists to understand the interconnections between them. By constructing a comprehensive IRES–translation initiation interaction map, biologists can get testable hypotheses on cellular IRES elements and devise subsequent verification experiments. Currently, four existing databases and datasets are devoted to similar tasks as Human IRES Atlas and thus are compared in the following discussion.</p><p id=\"idm139637289227072\">Two IRES databases [IRESite (<a href=\"#R71\" rid=\"R71\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">71</a>) and IRESbase (<a href=\"#R21\" rid=\"R21\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">21</a>)] were designed to curate and deposit IRES verification experiments from the literature. Researchers have carried out different bicistronic experiments for studying IRES activity of certain genes. Nevertheless, most of the information is fragmentary in different literature papers. Therefore, these two databases tried to gather and store these valuable records. While these two works collected valuable IRES experimental data, till now there are still few cellular IRES elements that have been experimentally verified and deposited. Only 26 human mRNA transcripts were evaluated to contain IRES elements in IRESite and 112 human IRESs were collected in IRESbase. Potential IRES-driven translational regulation programs in the human transcriptome cannot be easily deduced through these two databases due to the lack of an integrative platform for understanding the interactions between IRES elements and the related translation initiation features. Since there are only few available verified IRES elements in the literature, there is still a need to carry out systematic <em>in silico</em> IRES identification and construct a comprehensive IRES–translation initiation interaction map to complement the sparsity of IRES verification experiments.</p><p id=\"idm139637289223440\">Another related work called IRES Omnibus (<a href=\"#R77\" rid=\"R77\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">77</a>) studied human IRESs through systematic IRES prediction. In the preliminary work of IRES Omnibus, Chen and Tseng set up an in-house IRES prediction system based on the PatSearch tool to screen through human 5ʹ UTR sequences for potential IRES elements. In their work, IREZone, the special regions consisting of contiguous overlapping RNA structures, and IRES elements were predicted and collected. Using the IRES and IREZone repository, novel sequence patterns and IRES-related cellular functions were investigated. However, in their preliminary identification, merely one IRES prediction tool was used, and the complicated connections between IRES elements and translation initiation–related features were ignored. Hence, the constructed preliminary human IRES map may suffer from high false positives. Further, it would be beneficial for biologists to deduce the potential IRES-related regulatory molecular mechanisms if different high throughput translation initiation–related datasets can be integrated into a user-friendly platform.</p><p id=\"idm139637289221248\">The study by Weingarten-Gabbay <em>et al.</em> (<a href=\"#R12\" rid=\"R12\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">12</a>) is another touchstone in the field. We then evaluated the IRES identification sensitivity and specificity for Human IRES Atlas and the work of Weingarten-Gabbay <em>et al.</em> on a ground truth IRES dataset consisting of verified IRES sequences as the positive set and random sequences as the negative set. The positive set of 190 known IRES elements is obtained from the IRESbase (<a href=\"#R21\" rid=\"R21\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">21</a>) and the review work of Baird <em>et al.</em> (<a href=\"#R22\" rid=\"R22\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">22</a>). Since it is known that house-keeping genes depend mainly on cap-dependent translation initiation (<a href=\"#R23\" rid=\"R23\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">23</a>), we randomly sampled 190 sequences from the 5ʹ UTRs of annotated house-keeping genes (<a href=\"#R24\" rid=\"R24\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">24</a>) to be the negative set. An identified IRES in a dataset is defined to be a true positive if <em>&gt;</em>20% of the base pairs overlap with some sequence in the positive set. A sequence from the negative set is called a true negative for one identification method if no identified sequence has <em>&gt;</em>20% overlapping base pairs with it. On this ground truth IRES set, the results of Weingarten-Gabbay <em>et al.</em> obtain a sensitivity of 0.19 and a false-positive rate (FPR) of 0.037. The putative IRESs deposited in Human IRES Atlas have a sensitivity of 0.32 and an FPR of 0.226 on the ground truth dataset. We can see from these results that Human IRES Atlas achieves a better sensitivity with a small sacrifice of the FPR. Furthermore, if we merge the potential IRESs in Human IRES Atlas and the results of Weingarten-Gabbay <em>et al.</em>, a sensitivity of 0.43 and an FPR of 0.258 are observed. The near linearly independent sensitivity gain means that the two results are mostly complementary. By examining the data of Weingarten-Gabbay <em>et al.</em>, 19% of the literature-curated IRESs were found. Thirty-two percent of the literature-curated IRESs could be determined in Human IRES Atlas. While oversampling 5ʹ UTRs can lead to higher chances of overlapping with the true positive set, it also increases the possibility of overlapping the putative IRESs with the true negative set, raising the FPR. There is a compromise between the sensitivity and the FPR when oversampling 5ʹ UTRs. The putative IRES elements deposited in Human IRES Atlas are estimated to have a higher sensitivity than the corresponding FPR. Hence, Human IRES Atlas provides extra insights for IRES-mediated translation regulation in humans.</p><p id=\"idm139637335184400\" class=\"p p-last\">Human IRES Atlas is the first database explicitly designed to overcome current obstacles in studying human IRES elements. It also fulfills the necessity for deducing testable molecular mechanism hypotheses. In Human IRES Atlas, three different IRES prediction tools (IRESFinder, PatSearch and IRESpy) were used for systematic IRES screening. The results of different tools were collected by selecting the nested consensus minimal sequences among different prediction algorithms. Hence the prediction uncertainty and false discovery rate inherent in different tools were minimized during the identification process. To further help evaluate the functionality of the putative IRESs, three tests (conservation, structural RPI scores and conditional translation efficiency) were designed and performed. Besides performing the transcriptome-wide human IRES identification, we also integrated eight categories of translation initiation–related datasets. Biologists can further understand the IRES-driven translation regulation and develop testable molecular mechanism hypotheses for subsequent research through the translation initiation–related features. In summary, Human IRES Atlas integrates more data and provides more functionality than existing tools. It specifically aims to perform systematic IRES identification and help deduce the IRES-driven cellular translation regulation mechanisms.</p></div><div id=\"s3-s4\" class=\"sec sec-last\"><h3 id=\"s3-s4title\">Issues related to the constructed database</h3><p id=\"idm139637335182096\" class=\"p p-first\">The Human IRES Atlas database is constructed by depositing systematically identified <em>in silico</em> IRESs and various translation initiation–related data. Based on the eight gathered categories of translation initiation–related features, we can assess the significance of the putative IRESs and deduce potential IRES-driven translation regulation mechanism hypotheses. However, several issues may exist when interpreting the results generated from the systems-wide data integration process. Researchers usually perturbed cells and conducted distinct high-throughput experiments for gathering genome-wide information (<a href=\"#R78\" rid=\"R78\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">78</a>). However, different high-throughput experiments may be performed under different environmental conditions, cellular states or cell types. This experimental discrepancy causes inevitable systematic bias in the data integration process (<a href=\"#R79\" rid=\"R79\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">79</a>). Hence, some detailed subtleties should be considered when designing subsequent verification experiments based on the hypotheses formed from the systems view of data integration.</p><p id=\"idm139637370998112\">Diverse statistical analysis strategies were applied to the high-throughput experiment datasets. For example, we adopted the eTIS data by thresholding the positional read-count fold change and some other minimum detection requirement between samples treated with the ribosome E-site translation inhibitors CHX and LTM (<a href=\"#R64\" rid=\"R64\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">64</a>). The high-throughput screening of IRES activity adopted in this database was probed using fluorescence measurements (<a href=\"#R12\" rid=\"R12\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">12</a>). Different statistical analysis strategies provide different false discovery rates. In general, noises resulted from the data statistical analysis were assumed to be orthogonal in different high-throughput experimental results (<a href=\"#R35\" rid=\"R35\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">35</a>). Hence, by simultaneously combining experimental data of similar targeted systems, the false positive issue can be reduced. In Human IRES Atlas, we have gathered diverse translation initiation–related data and used different screening tools in the systematic putative IRES identification. This cross-referencing screening reduces the impact of false positives. For instance, TISs based on CHX/LTM experimental analysis and PreTIS prediction were both collected to lessen the false positive impact in TIS location information. We will keep Human IRES Atlas up-to-date with newly published datasets when more data or methods are released.</p><p id=\"idm139637370993904\" class=\"p p-last\">Data integration in multicellular organisms often brings about the issue of cellular systems bias. In metazoa species, correct temporal and spatial gene expression of the same genome is the core control on cell differentiation (<a href=\"#R13\" rid=\"R13\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">13</a>). While different combinations of cell types in data integration may provide distinct false positives, the multicellular systems problem should be taken care of in using the constructed database. IRES activity data were probed <em>in vitro</em> based on experiments on sequence libraries. Non-canonical TIS and uORF information were identified <em>in silico</em> by investigating comparative genomics and sequence contents. These data reveal the normal status of the translation program. To assist in cell type-specific or condition-specific IRES verification experimental design, one can refer to ribosome profiling experiments conducted under different conditions/cell types or refer to special cellular conditions where the ITAF-binding events were observed. Diverse ribosome profiling sequencing results are fragmentary in the literature. Recently, a database called HPRDviewer (<a href=\"#R41\" rid=\"R41\" class=\" bibr popnode\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">41</a>) is constructed to deposit comprehensive human ribosome profiling datasets. In Human IRES Atlas, the conditional ribosome profiling results performed under the stressed and normal conditions are shown. More condition-specific human ribosome profiling information can be found in HPRDviewer. Through consulting more detailed ribosome profiling experiments or the ITAF-binding event conditions, one can further minimize the impact of cell type bias inherent in the data integration process.</p></div></div><div id=\"s4\" class=\"tsec sec\"><div class=\"goto jig-ncbiinpagenav-goto-container\"><a class=\"tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading\" href=\"#\" title=\"Go to other sections in this page\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Go to:</a></div><h2 class=\"head no_bottom_margin ui-helper-clearfix\" id=\"s4title\">Conclusions</h2><p id=\"idm139637295659648\" class=\"p p-first-last\">In this research, a novel database called Human IRES Atlas was constructed to deposit the systematic <em>in silico</em> identified IRESs from the human transcriptome. Besides the transcriptome-wide putative IRES identification, eight genres of translation initiation–related datasets were integrated and three novel IRES functionality tests were designed. To facilitate the understanding of the connections between IRES elements and these translation initiation–related features, an easy-to-use interface for the integrated results was implemented for biologists to infer possible molecular mechanisms of human IRES-driven translation initiation. Researchers can easily search the IRES of interest or browse the IRESs on genes involved in specific biological pathways with the database’s user-friendly web-interface. By simultaneously studying the human IRES and the translation initiation–related features, the database can provide testable molecular mechanism hypotheses on human gene translation regulation. We believe that Human IRES Atlas can help biologists better study mechanisms of cellular IRES-driven translation initiation, IRES-related cancer progression and other IRES-related genetic disorders.</p></div><div id=\"ack1\" class=\"tsec sec\"><div class=\"goto jig-ncbiinpagenav-goto-container\"><a class=\"tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading\" href=\"#\" title=\"Go to other sections in this page\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Go to:</a></div><h2 class=\"head no_bottom_margin ui-helper-clearfix\" id=\"ack1title\">Acknowledgements</h2><div class=\"sec\"><p id=\"idm139637295656768\">The authors would like to thank Ting-Hsuan Chen and Yu-Cian Lin for helping with the preliminary data collection.</p></div></div><div id=\"idm139637292208448aff-info\" class=\"tsec sec\"><div class=\"goto jig-ncbiinpagenav-goto-container\"><a class=\"tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading\" href=\"#\" title=\"Go to other sections in this page\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Go to:</a></div><h2 class=\"head no_bottom_margin ui-helper-clearfix\" id=\"idm139637292208448aff-infotitle\">Contributor Information</h2><p><span class=\"fm-affl\">Tzu-Hsien Yang, </span><span class=\"fm-affl\" lang=\"en\" id=\"idm139637289670160\"> \\nDepartment of Information Management, National University of Kaohsiung, 700, Kaohsiung University Rd., Nanzih District, Kaohsiung 811, Taiwan.</span></p><p><span class=\"fm-affl\">Chung-Yu Wang, </span><span class=\"fm-affl\" lang=\"en\" id=\"idm139637300980416\"> \\nDepartment of Information Management, National University of Kaohsiung, 700, Kaohsiung University Rd., Nanzih District, Kaohsiung 811, Taiwan.</span></p><p><span class=\"fm-affl\">Hsiu-Chun Tsai, </span><span class=\"fm-affl\" lang=\"en\" id=\"idm139637300977024\"> \\nDepartment of Information Management, National University of Kaohsiung, 700, Kaohsiung University Rd., Nanzih District, Kaohsiung 811, Taiwan.</span></p><p><span class=\"fm-affl\">Cheng-Tse Liu, </span><span class=\"fm-affl\" lang=\"en\" id=\"idm139637297455056\"> \\nDepartment of Information Management, National University of Kaohsiung, 700, Kaohsiung University Rd., Nanzih District, Kaohsiung 811, Taiwan.</span></p></div><div id=\"s5\" class=\"tsec bk-sec\"><div class=\"goto jig-ncbiinpagenav-goto-container\"><a class=\"tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading\" href=\"#\" title=\"Go to other sections in this page\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Go to:</a></div><h2 class=\"head no_bottom_margin ui-helper-clearfix\" id=\"s5title\">Funding</h2><!--/article/back/sec/--><p id=\"idm139637295655760\" class=\"p p-first-last\">This work is funded by National University of Kaohsiung and Ministry of Science and Technology of Taiwan (MOST 107-2218-E-390-009-MY3).</p></div><div id=\"s6\" class=\"tsec bk-sec\"><div class=\"goto jig-ncbiinpagenav-goto-container\"><a class=\"tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading\" href=\"#\" title=\"Go to other sections in this page\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Go to:</a></div><h2 class=\"head no_bottom_margin ui-helper-clearfix\" id=\"s6title\">Conflict of interest.</h2><!--/article/back/sec/--><p id=\"idm139637295654736\" class=\"p p-first-last\">The authors declare that they have no competing interests.</p></div><div id=\"ref1\" class=\"tsec sec\"><div class=\"goto jig-ncbiinpagenav-goto-container\"><a class=\"tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading\" href=\"#\" title=\"Go to other sections in this page\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Go to:</a></div><h2 class=\"head no_bottom_margin ui-helper-clearfix\" id=\"ref1title\">References</h2><div class=\"ref-list-sec sec\" id=\"reference-list\"><div class=\"ref-cit-blk half_rhythm\" id=\"R1\">1. <span class=\"mixed-citation\">\\nGebauer F., Hentze M.W. (2004) <span class=\"ref-title\">Molecular mechanisms of translational control</span>. <span class=\"ref-journal\"><em>Nat. Rev. Mol. Cell Biol.</em></span>, <span class=\"ref-vol\">5</span>, nrm1488. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC7097087/\">PMC free article</a>]</span> [<a href=\"/pubmed/15459663\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Mol.+Cell+Biol.&amp;title=Molecular+mechanisms+of+translational+control&amp;volume=5&amp;publication_year=2004&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R2\">2. <span class=\"mixed-citation\">\\nSonenberg N., Hinnebusch A.G. (2009) <span class=\"ref-title\">Regulation of translation initiation in eukaryotes: mechanisms and biological targets</span>. <span class=\"ref-journal\"><em>Cell</em></span>, <span class=\"ref-vol\">136</span>, 731–745. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC3610329/\">PMC free article</a>]</span> [<a href=\"/pubmed/19239892\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Regulation+of+translation+initiation+in+eukaryotes:+mechanisms+and+biological+targets&amp;volume=136&amp;publication_year=2009&amp;pages=731-745&amp;pmid=19239892&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R3\">3. <span class=\"mixed-citation\">\\nJohnson A.G., Grosely R., Petrov A.N.. \\net al. (2017) <span class=\"ref-title\">Dynamics of IRES-mediated translation</span>. <span class=\"ref-journal\"><em>Philos. Trans. R. Soc. B Biol. Sci.</em></span>, <span class=\"ref-vol\">372</span>, 20160177. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC5311923/\">PMC free article</a>]</span> [<a href=\"/pubmed/28138065\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Philos.+Trans.+R.+Soc.+B+Biol.+Sci.&amp;title=Dynamics+of+IRES-mediated+translation&amp;volume=372&amp;publication_year=2017&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R4\">4. <span class=\"mixed-citation\">\\nPelletier J., Sonenberg N. (1988) <span class=\"ref-title\">Internal initiation of translation of eukaryotic mRNA directed by a sequence derived from poliovirus RNA</span>. <span class=\"ref-journal\"><em>Nature</em></span>, <span class=\"ref-vol\">334</span>, 320. [<a href=\"/pubmed/2839775\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Internal+initiation+of+translation+of+eukaryotic+mRNA+directed+by+a+sequence+derived+from+poliovirus+RNA&amp;volume=334&amp;publication_year=1988&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R5\">5. <span class=\"mixed-citation\">\\nLukavsky P.J. (2009) <span class=\"ref-title\">Structure and function of HCV IRES domains</span>. <span class=\"ref-journal\"><em>Virus Res.</em></span>, <span class=\"ref-vol\">139</span>, 166–171. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC2726286/\">PMC free article</a>]</span> [<a href=\"/pubmed/18638512\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Virus+Res.&amp;title=Structure+and+function+of+HCV+IRES+domains&amp;volume=139&amp;publication_year=2009&amp;pages=166-171&amp;pmid=18638512&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R6\">6. <span class=\"mixed-citation\">\\nRay P.S., Grover R., Das S. (2006) <span class=\"ref-title\">Two internal ribosome entry sites mediate the translation of p53 isoforms</span>. <span class=\"ref-journal\"><em>EMBO Rep.</em></span>, <span class=\"ref-vol\">7</span>, 404–410. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC1456917/\">PMC free article</a>]</span> [<a href=\"/pubmed/16440000\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=EMBO+Rep.&amp;title=Two+internal+ribosome+entry+sites+mediate+the+translation+of+p53+isoforms&amp;volume=7&amp;publication_year=2006&amp;pages=404-410&amp;pmid=16440000&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R7\">7. <span class=\"mixed-citation\">\\nSharathchandra A., Katoch A., Das S. (2014) <span class=\"ref-title\">IRES mediated translational regulation of p53 isoforms</span>. <span class=\"ref-journal\"><em>Wiley Interdiscip. Rev. RNA</em></span>, <span class=\"ref-vol\">5</span>, 131–139. [<a href=\"/pubmed/24343861\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Wiley+Interdiscip.+Rev.+RNA&amp;title=IRES+mediated+translational+regulation+of+p53+isoforms&amp;volume=5&amp;publication_year=2014&amp;pages=131-139&amp;pmid=24343861&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R8\">8. <span class=\"mixed-citation\">\\nSharma D.K., Bressler K., Patel H.. \\net al. (2016) <span class=\"ref-title\">Role of eukaryotic initiation factors during cellular stress and cancer progression</span>. <span class=\"ref-journal\"><em>J. Nucleic Acids</em></span>, <span class=\"ref-vol\">2016</span>, 8235121. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC5204094/\">PMC free article</a>]</span> [<a href=\"/pubmed/28083147\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Nucleic+Acids&amp;title=Role+of+eukaryotic+initiation+factors+during+cellular+stress+and+cancer+progression&amp;volume=2016&amp;publication_year=2016&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R9\">9. <span class=\"mixed-citation\">\\nHolcik M., Sonenberg N. (2005) <span class=\"ref-title\">Translational control in stress and apoptosis</span>. <span class=\"ref-journal\"><em>Nat. Rev. Mol. Cell Biol.</em></span>, <span class=\"ref-vol\">6</span>, 318–327. [<a href=\"/pubmed/15803138\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Mol.+Cell+Biol.&amp;title=Translational+control+in+stress+and+apoptosis&amp;volume=6&amp;publication_year=2005&amp;pages=318-327&amp;pmid=15803138&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R10\">10. <span class=\"mixed-citation\">\\nSpriggs K.A., Stoneley M., Bushell M.. \\net al. (2008) <span class=\"ref-title\">Re-programming of translation following cell stress allows IRES-mediated translation to predominate</span>. <span class=\"ref-journal\"><em>Biol. Cell</em></span>, <span class=\"ref-vol\">100</span>, 27–38. [<a href=\"/pubmed/18072942\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Biol.+Cell&amp;title=Re-programming+of+translation+following+cell+stress+allows+IRES-mediated+translation+to+predominate&amp;volume=100&amp;publication_year=2008&amp;pages=27-38&amp;pmid=18072942&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R11\">11. <span class=\"mixed-citation\">\\nLeppek K., Das R., Barna M. (2018) <span class=\"ref-title\">Functional 5′UTR mRNA structures in eukaryotic translation regulation and how to find them</span>. <span class=\"ref-journal\"><em>Nat. Rev. Mol. Cell Biol.</em></span>, <span class=\"ref-vol\">19</span>, 158. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC5820134/\">PMC free article</a>]</span> [<a href=\"/pubmed/29165424\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Mol.+Cell+Biol.&amp;title=Functional+5′UTR+mRNA+structures+in+eukaryotic+translation+regulation+and+how+to+find+them&amp;volume=19&amp;publication_year=2018&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R12\">12. <span class=\"mixed-citation\">\\nWeingarten-Gabbay S., Elias-Kirma S., Nir R. et al. (2016) <span class=\"ref-title\">Systematic discovery of cap-independent translation sequences in human and viral genomes</span>. <span class=\"ref-journal\"><em>Science</em></span>, <span class=\"ref-vol\">351</span>, aad4939. [<a href=\"/pubmed/26816383\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Systematic+discovery+of+cap-independent+translation+sequences+in+human+and+viral+genomes&amp;volume=351&amp;publication_year=2016&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R13\">13. <span class=\"mixed-citation\">\\nYang T.-H., Wang -C.-C., Hung P.-C.. \\net al. (2014) <span class=\"ref-title\">cis MEP: an integrated repository of genomic epigenetic profiles and cis-regulatory modules in Drosophila</span>. <span class=\"ref-journal\"><em>BMC Syst. Biol.</em></span>, <span class=\"ref-vol\">8</span>, S8. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC4290730/\">PMC free article</a>]</span> [<a href=\"/pubmed/25521507\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Syst.+Biol.&amp;title=cis+MEP:+an+integrated+repository+of+genomic+epigenetic+profiles+and+cis-regulatory+modules+in+Drosophila&amp;volume=8&amp;publication_year=2014&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R14\">14. <span class=\"mixed-citation\">\\nYang T.-H., Wang -C.-C., Wang Y.-C.. \\net al. (2014) <span class=\"ref-title\">YTRP: a repository for yeast transcriptional regulatory pathways</span>. <span class=\"ref-journal\"><em>Database</em></span>, <span class=\"ref-vol\">2014</span>, bau014. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC3948430/\">PMC free article</a>]</span> [<a href=\"/pubmed/24608172\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Database&amp;title=YTRP:+a+repository+for+yeast+transcriptional+regulatory+pathways&amp;volume=2014&amp;publication_year=2014&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R15\">15. <span class=\"mixed-citation\">\\nHaeussler M., Zweig A.S., Tyner C.. \\net al. (2019) <span class=\"ref-title\">The UCSC genome browser database: 2019 update</span>. <span class=\"ref-journal\"><em>Nucleic Acids Res.</em></span>, <span class=\"ref-vol\">47</span>, D853–D858. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC6323953/\">PMC free article</a>]</span> [<a href=\"/pubmed/30407534\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+UCSC+genome+browser+database:+2019+update&amp;volume=47&amp;publication_year=2019&amp;pages=D853-D858&amp;pmid=30407534&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R16\">16. <span class=\"mixed-citation\">\\nZhao J., Wu J., Xu T.. \\net al. (2018) <span class=\"ref-title\">IRESfinder: identifying RNA internal ribosome entry site in eukaryotic cell using framed k-mer features</span>. <span class=\"ref-journal\"><em>J. Genet. Genomics</em></span>, <span class=\"ref-vol\">45</span>, 403–406. [<a href=\"/pubmed/30054216\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Genet.+Genomics&amp;title=IRESfinder:+identifying+RNA+internal+ribosome+entry+site+in+eukaryotic+cell+using+framed+k-mer+features&amp;volume=45&amp;publication_year=2018&amp;pages=403-406&amp;pmid=30054216&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R17\">17. <span class=\"mixed-citation\">\\nGrillo G., Licciulli F., Liuni S.. \\net al. (2003) <span class=\"ref-title\">PatSearch: a program for the detection of patterns and structural motifs in nucleotide sequences</span>. <span class=\"ref-journal\"><em>Nucleic Acids Res.</em></span>, <span class=\"ref-vol\">31</span>, 3608–3612. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC168955/\">PMC free article</a>]</span> [<a href=\"/pubmed/12824377\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=PatSearch:+a+program+for+the+detection+of+patterns+and+structural+motifs+in+nucleotide+sequences&amp;volume=31&amp;publication_year=2003&amp;pages=3608-3612&amp;pmid=12824377&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R18\">18. <span class=\"mixed-citation\">\\nWashietl S., Hofacker I.L., Stadler P.F. (2005) <span class=\"ref-title\">Fast and reliable prediction of noncoding RNAs</span>. <span class=\"ref-journal\"><em>Proc. Natl. Acad. Sci. USA</em></span>, <span class=\"ref-vol\">102</span>, 2454–2459. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC548974/\">PMC free article</a>]</span> [<a href=\"/pubmed/15665081\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Fast+and+reliable+prediction+of+noncoding+RNAs&amp;volume=102&amp;publication_year=2005&amp;pages=2454-2459&amp;pmid=15665081&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R19\">19. <span class=\"mixed-citation\">\\nKolekar P., Pataskar A., Kulkarni-Kale U.. \\net al. (2016) <span class=\"ref-title\">IRESPred: web server for prediction of cellular and viral internal ribosome entry site (IRES)</span>. <span class=\"ref-journal\"><em>Sci. Rep.</em></span>, <span class=\"ref-vol\">6</span>, 27436. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC4893748/\">PMC free article</a>]</span> [<a href=\"/pubmed/27264539\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=IRESPred:+web+server+for+prediction+of+cellular+and+viral+internal+ribosome+entry+site+(IRES)&amp;volume=6&amp;publication_year=2016&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R20\">20. <span class=\"mixed-citation\">\\nWang J., Gribskov M. (2019) <span class=\"ref-title\">IRESpy: an XGBoost model for prediction of internal ribosome entry sites</span>. <span class=\"ref-journal\"><em>BMC Bioinform.</em></span>, <span class=\"ref-vol\">20</span>, 409. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC6664791/\">PMC free article</a>]</span> [<a href=\"/pubmed/31362694\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Bioinform.&amp;title=IRESpy:+an+XGBoost+model+for+prediction+of+internal+ribosome+entry+sites&amp;volume=20&amp;publication_year=2019&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R21\">21. <span class=\"mixed-citation\">\\nZhao J., Li Y., Wang C.. \\net al. (2020) <span class=\"ref-title\">IRESbase: a comprehensive database of experimentally validated internal ribosome entry sites</span>. <span class=\"ref-journal\"><em>Genomics Proteomics Bioinf</em></span>, <span class=\"ref-vol\">18</span>, 129–139. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC7646085/\">PMC free article</a>]</span> [<a href=\"/pubmed/32512182\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genomics+Proteomics+Bioinf&amp;title=IRESbase:+a+comprehensive+database+of+experimentally+validated+internal+ribosome+entry+sites&amp;volume=18&amp;publication_year=2020&amp;pages=129-139&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R22\">22. <span class=\"mixed-citation\">\\nBaird S.D., Turcotte M., Korneluk R.G.. \\net al. (2006) <span class=\"ref-title\">Searching for IRES</span>. <span class=\"ref-journal\"><em>RNA</em></span>, <span class=\"ref-vol\">12</span>, 1755–1785. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC1581980/\">PMC free article</a>]</span> [<a href=\"/pubmed/16957278\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=RNA&amp;title=Searching+for+IRES&amp;volume=12&amp;publication_year=2006&amp;pages=1755-1785&amp;pmid=16957278&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R23\">23. <span class=\"mixed-citation\">\\nKomar A.A., Hatzoglou M. (2011) <span class=\"ref-title\">Cellular IRES-mediated translation: the war of ITAFs in pathophysiological states</span>. <span class=\"ref-journal\"><em>Cell Cycle</em></span>, <span class=\"ref-vol\">10</span>, 229–240. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC3048795/\">PMC free article</a>]</span> [<a href=\"/pubmed/21220943\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cell+Cycle&amp;title=Cellular+IRES-mediated+translation:+the+war+of+ITAFs+in+pathophysiological+states&amp;volume=10&amp;publication_year=2011&amp;pages=229-240&amp;pmid=21220943&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R24\">24. <span class=\"mixed-citation\">\\nEisenberg E., Levanon E.Y. (2013) <span class=\"ref-title\">Human housekeeping genes, revisited</span>. <span class=\"ref-journal\"><em>Trends Genet.</em></span>, <span class=\"ref-vol\">29</span>, 569–574. [<a href=\"/pubmed/23810203\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Trends+Genet.&amp;title=Human+housekeeping+genes,+revisited&amp;volume=29&amp;publication_year=2013&amp;pages=569-574&amp;pmid=23810203&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R25\">25. <span class=\"mixed-citation\">\\nGritsenko A.A., Weingarten-Gabbay S., Elias-Kirma S.. \\net al. (2017) <span class=\"ref-title\">Sequence features of viral and human Internal Ribosome Entry Sites predictive of their activity</span>. <span class=\"ref-journal\"><em>PLoS Comput. Biol.</em></span>, <span class=\"ref-vol\">13</span>, e1005734. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC5630158/\">PMC free article</a>]</span> [<a href=\"/pubmed/28922394\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+Comput.+Biol.&amp;title=Sequence+features+of+viral+and+human+Internal+Ribosome+Entry+Sites+predictive+of+their+activity&amp;volume=13&amp;publication_year=2017&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R26\">26. <span class=\"mixed-citation\">\\nDu X., Wang J., Zhu H.. \\net al. (2013) <span class=\"ref-title\">Second cistron in CACNA1A gene encodes a transcription factor mediating cerebellar development and SCA6</span>. <span class=\"ref-journal\"><em>Cell</em></span>, <span class=\"ref-vol\">154</span>, 118–133. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC3939801/\">PMC free article</a>]</span> [<a href=\"/pubmed/23827678\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Second+cistron+in+CACNA1A+gene+encodes+a+transcription+factor+mediating+cerebellar+development+and+SCA6&amp;volume=154&amp;publication_year=2013&amp;pages=118-133&amp;pmid=23827678&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R27\">27. <span class=\"mixed-citation\">\\nCornelis S., Bruynooghe Y., Denecker G.. \\net al. (2000) <span class=\"ref-title\">Identification and characterization of a novel cell cycle–regulated internal ribosome entry site</span>. <span class=\"ref-journal\"><em>Mol. Cell</em></span>, <span class=\"ref-vol\">5</span>, 597–605. [<a href=\"/pubmed/10882096\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=Identification+and+characterization+of+a+novel+cell+cycle–regulated+internal+ribosome+entry+site&amp;volume=5&amp;publication_year=2000&amp;pages=597-605&amp;pmid=10882096&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R28\">28. <span class=\"mixed-citation\">\\nCandeias M.M., Powell D.J., Roubalova E.. \\net al. (2006) <span class=\"ref-title\">Expression of p53 and p53/47 are controlled by alternative mechanisms of messenger RNA translation initiation</span>. <span class=\"ref-journal\"><em>Oncogene</em></span>, <span class=\"ref-vol\">25</span>, 6936–6947. [<a href=\"/pubmed/16983332\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Expression+of+p53+and+p53/47+are+controlled+by+alternative+mechanisms+of+messenger+RNA+translation+initiation&amp;volume=25&amp;publication_year=2006&amp;pages=6936-6947&amp;pmid=16983332&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R29\">29. <span class=\"mixed-citation\">\\nFaye M.D., Graber T.E., Liu P.. \\net al. (2013) <span class=\"ref-title\">Nucleotide composition of cellular internal ribosome entry sites defines dependence on NF45 and predicts a posttranscriptional mitotic regulon</span>. <span class=\"ref-journal\"><em>Mol. Cell. Biol.</em></span>, <span class=\"ref-vol\">33</span>, 307–318. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC3554113/\">PMC free article</a>]</span> [<a href=\"/pubmed/23129811\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol.&amp;title=Nucleotide+composition+of+cellular+internal+ribosome+entry+sites+defines+dependence+on+NF45+and+predicts+a+posttranscriptional+mitotic+regulon&amp;volume=33&amp;publication_year=2013&amp;pages=307-318&amp;pmid=23129811&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R30\">30. <span class=\"mixed-citation\">\\nStoneley M., Willis A.E. (2004) <span class=\"ref-title\">Cellular internal ribosome entry segments: structures, trans-acting factors and regulation of gene expression</span>. <span class=\"ref-journal\"><em>Oncogene</em></span>, <span class=\"ref-vol\">23</span>, 3200. [<a href=\"/pubmed/15094769\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Cellular+internal+ribosome+entry+segments:+structures,+trans-acting+factors+and+regulation+of+gene+expression&amp;volume=23&amp;publication_year=2004&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R31\">31. <span class=\"mixed-citation\">\\nWan Y., Kertesz M., Spitale R.C.. \\net al. (2011) <span class=\"ref-title\">Understanding the transcriptome through RNA structure</span>. <span class=\"ref-journal\"><em>Nat. Rev. Genet.</em></span>, <span class=\"ref-vol\">12</span>, 641–655. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC3858389/\">PMC free article</a>]</span> [<a href=\"/pubmed/21850044\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Understanding+the+transcriptome+through+RNA+structure&amp;volume=12&amp;publication_year=2011&amp;pages=641-655&amp;pmid=21850044&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R32\">32. <span class=\"mixed-citation\">\\nYang T.-H. (2019) <span class=\"ref-title\">Transcription factor regulatory modules provide the molecular mechanisms for functional redundancy observed among transcription factors in yeast</span>. <span class=\"ref-journal\"><em>BMC Bioinform.</em></span>, <span class=\"ref-vol\">20</span>, 1–16. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC6933673/\">PMC free article</a>]</span> [<a href=\"/pubmed/31881824\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Bioinform.&amp;title=Transcription+factor+regulatory+modules+provide+the+molecular+mechanisms+for+functional+redundancy+observed+among+transcription+factors+in+yeast&amp;volume=20&amp;publication_year=2019&amp;pages=1-16&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R33\">33. <span class=\"mixed-citation\">\\nSiepel A., Bejerano G., Pedersen J.S.. \\net al. (2005) <span class=\"ref-title\">Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes</span>. <span class=\"ref-journal\"><em>Genome Res.</em></span>, <span class=\"ref-vol\">15</span>, 1034–1050. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC1182216/\">PMC free article</a>]</span> [<a href=\"/pubmed/16024819\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=Evolutionarily+conserved+elements+in+vertebrate,+insect,+worm,+and+yeast+genomes&amp;volume=15&amp;publication_year=2005&amp;pages=1034-1050&amp;pmid=16024819&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R34\">34. <span class=\"mixed-citation\">\\nMargulies E.H., Blanchette M., Haussler D.et al. , N. C. S. Program . (2003) <span class=\"ref-title\">Identification and characterization of multi-species conserved sequences</span>. <span class=\"ref-journal\"><em>Genome Res.</em></span>, <span class=\"ref-vol\">13</span>, 2507–2518. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC403793/\">PMC free article</a>]</span> [<a href=\"/pubmed/14656959\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=Identification+and+characterization+of+multi-species+conserved+sequences&amp;volume=13&amp;publication_year=2003&amp;pages=2507-2518&amp;pmid=14656959&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R35\">35. <span class=\"mixed-citation\">\\nYang T.-H., Wu W.-S. (2013) <span class=\"ref-title\">Inferring functional transcription factor-gene binding pairs by integrating transcription factor binding data with transcription factor knockout data</span>. <span class=\"ref-journal\"><em>BMC Syst. Biol.</em></span>, <span class=\"ref-vol\">7</span>, S13. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC4029220/\">PMC free article</a>]</span> [<a href=\"/pubmed/24565265\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Syst.+Biol.&amp;title=Inferring+functional+transcription+factor-gene+binding+pairs+by+integrating+transcription+factor+binding+data+with+transcription+factor+knockout+data&amp;volume=7&amp;publication_year=2013&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R36\">36. <span class=\"mixed-citation\">\\nLee K.-M., Chen C.-J., Shih S.-R. (2017) <span class=\"ref-title\">Regulation mechanisms of viral IRES-driven translation</span>. <span class=\"ref-journal\"><em>Trends Microbiol.</em></span>, <span class=\"ref-vol\">25</span>, 546–561. [<a href=\"/pubmed/28242053\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Trends+Microbiol.&amp;title=Regulation+mechanisms+of+viral+IRES-driven+translation&amp;volume=25&amp;publication_year=2017&amp;pages=546-561&amp;pmid=28242053&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R37\">37. <span class=\"mixed-citation\">\\nSidrauski C., McGeachy A.M., Ingolia N.T.. \\net al. (2015) <span class=\"ref-title\">The small molecule ISRIB reverses the effects of eIF2α phosphorylation on translation and stress granule assembly</span>. <span class=\"ref-journal\"><em>Elife</em></span>, <span class=\"ref-vol\">4</span>, e05033. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC4341466/\">PMC free article</a>]</span> [<a href=\"/pubmed/25719440\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=The+small+molecule+ISRIB+reverses+the+effects+of+eIF2α+phosphorylation+on+translation+and+stress+granule+assembly&amp;volume=4&amp;publication_year=2015&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R38\">38. <span class=\"mixed-citation\">\\nMitchell S.A., Spriggs K.A., Coldwell M.J.. \\net al. (2003) <span class=\"ref-title\">The Apaf-1 internal ribosome entry segment attains the correct structural conformation for function via interactions with PTB and unr</span>. <span class=\"ref-journal\"><em>Mol. Cell</em></span>, <span class=\"ref-vol\">11</span>, 757–771. [<a href=\"/pubmed/12667457\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=The+Apaf-1+internal+ribosome+entry+segment+attains+the+correct+structural+conformation+for+function+via+interactions+with+PTB+and+unr&amp;volume=11&amp;publication_year=2003&amp;pages=757-771&amp;pmid=12667457&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R39\">39. <span class=\"mixed-citation\">\\nZhang H., Wang Y., Lu J. (2019) <span class=\"ref-title\">Function and evolution of upstream ORFs in eukaryotes</span>. <span class=\"ref-journal\"><em>Trends Biochem. Sci.</em></span>, <span class=\"ref-vol\">44</span>, 782–794. [<a href=\"/pubmed/31003826\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Trends+Biochem.+Sci.&amp;title=Function+and+evolution+of+upstream+ORFs+in+eukaryotes&amp;volume=44&amp;publication_year=2019&amp;pages=782-794&amp;pmid=31003826&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R40\">40. <span class=\"mixed-citation\">\\nChen T.-M., Shih Y.-H., Tseng J. T.. \\net al. (2014) <span class=\"ref-title\">Overexpression of FGF9 in colon cancer cells is mediated by hypoxia-induced translational activation</span>. <span class=\"ref-journal\"><em>Nucleic Acids Res.</em></span>, <span class=\"ref-vol\">42</span>, 2932–2944. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC3950685/\">PMC free article</a>]</span> [<a href=\"/pubmed/24334956\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Overexpression+of+FGF9+in+colon+cancer+cells+is+mediated+by+hypoxia-induced+translational+activation&amp;volume=42&amp;publication_year=2014&amp;pages=2932-2944&amp;pmid=24334956&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R41\">41. <span class=\"mixed-citation\">\\nWu W.-S., Jiang Y.-X., Chang J.-W.. \\net al. (2018) <span class=\"ref-title\">HRPDviewer: human ribosome profiling data viewer</span>. <span class=\"ref-journal\"><em>Database</em></span>, <span class=\"ref-vol\">2018</span>, bay074. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC6041748/\">PMC free article</a>]</span> [<a href=\"/pubmed/30010738\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Database&amp;title=HRPDviewer:+human+ribosome+profiling+data+viewer&amp;volume=2018&amp;publication_year=2018&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R42\">42. <span class=\"mixed-citation\">\\nMartin M. (2011) <span class=\"ref-title\">Cutadapt removes adapter sequences from high-throughput sequencing reads</span>. <span class=\"ref-journal\"><em>EMBnet.journal</em></span>, <span class=\"ref-vol\">17</span>, 10. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=EMBnet.journal&amp;title=Cutadapt+removes+adapter+sequences+from+high-throughput+sequencing+reads&amp;volume=17&amp;publication_year=2011&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R43\">43. <span class=\"mixed-citation\">\\nLi B., Dewey C.N. (2011) <span class=\"ref-title\">RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome</span>. <span class=\"ref-journal\"><em>BMC Bioinform.</em></span>, <span class=\"ref-vol\">12</span>, 1–16. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC3163565/\">PMC free article</a>]</span> [<a href=\"/pubmed/21816040\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Bioinform.&amp;title=RSEM:+accurate+transcript+quantification+from+RNA-Seq+data+with+or+without+a+reference+genome&amp;volume=12&amp;publication_year=2011&amp;pages=1-16&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R44\">44. <span class=\"mixed-citation\">\\nLe Quesne J.P.C., Stoneley M., Fraser G.A.. \\net al. (2001) <span class=\"ref-title\">Derivation of a structural model for the c-myc IRES</span>. <span class=\"ref-journal\"><em>J. Mol. Biol.</em></span>, <span class=\"ref-vol\">310</span>, 111–126. [<a href=\"/pubmed/11419940\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Biol.&amp;title=Derivation+of+a+structural+model+for+the+c-myc+IRES&amp;volume=310&amp;publication_year=2001&amp;pages=111-126&amp;pmid=11419940&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R45\">45. <span class=\"mixed-citation\">\\nYaman I., Fernandez J., Liu H.. \\net al. (2003) <span class=\"ref-title\">The zipper model of translational control: a small upstream ORF is the switch that controls structural remodeling of an mRNA leader</span>. <span class=\"ref-journal\"><em>Cell</em></span>, <span class=\"ref-vol\">113</span>, 519–531. [<a href=\"/pubmed/12757712\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The+zipper+model+of+translational+control:+a+small+upstream+ORF+is+the+switch+that+controls+structural+remodeling+of+an+mRNA+leader&amp;volume=113&amp;publication_year=2003&amp;pages=519-531&amp;pmid=12757712&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R46\">46. <span class=\"mixed-citation\">\\nLorenz R., Bernhart S.H., Zu Siederdissen C.H.. \\net al. (2011) <span class=\"ref-title\">ViennaRNA package 2.0</span>. <span class=\"ref-journal\"><em>Algorithms Mol. Biol.</em></span>, <span class=\"ref-vol\">6</span>, 26. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC3319429/\">PMC free article</a>]</span> [<a href=\"/pubmed/22115189\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Algorithms+Mol.+Biol.&amp;title=ViennaRNA+package+2.0&amp;volume=6&amp;publication_year=2011&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R47\">47. <span class=\"mixed-citation\">\\nHofacker I.L., Priwitzer B. and Stadler P.F. (2004) <span class=\"ref-title\">Prediction of locally stable RNA secondary structures for genome-wide surveys</span>. <span class=\"ref-journal\"><em>Bioinformatics</em></span>, <span class=\"ref-vol\">20</span>, 186–190. [<a href=\"/pubmed/14734309\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Prediction+of+locally+stable+RNA+secondary+structures+for+genome-wide+surveys&amp;volume=20&amp;publication_year=2004&amp;pages=186-190&amp;pmid=14734309&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R48\">48. <span class=\"mixed-citation\">\\nDeng F., Ledda M., Vaziri S.. \\net al. (2016) <span class=\"ref-title\">Data-directed RNA secondary structure prediction using probabilistic modeling</span>. <span class=\"ref-journal\"><em>RNA</em></span>, <span class=\"ref-vol\">22</span>, 1109–1109. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC4931104/\">PMC free article</a>]</span> [<a href=\"/pubmed/27251549\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=RNA&amp;title=Data-directed+RNA+secondary+structure+prediction+using+probabilistic+modeling&amp;volume=22&amp;publication_year=2016&amp;pages=1109-1109&amp;pmid=27251549&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R49\">49. <span class=\"mixed-citation\">\\nLu Z.J., Gloor J.W., Mathews D.H. (2009) <span class=\"ref-title\">Improved RNA secondary structure prediction by maximizing expected pair accuracy</span>. <span class=\"ref-journal\"><em>RNA</em></span>, <span class=\"ref-vol\">15</span>, 1805–1813. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC2743040/\">PMC free article</a>]</span> [<a href=\"/pubmed/19703939\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=RNA&amp;title=Improved+RNA+secondary+structure+prediction+by+maximizing+expected+pair+accuracy&amp;volume=15&amp;publication_year=2009&amp;pages=1805-1813&amp;pmid=19703939&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R50\">50. <span class=\"mixed-citation\">\\nBernhart S.H., Hofacker I.L., Will S.. \\net al. (2008) <span class=\"ref-title\">RNAalifold: improved consensus structure prediction for RNA alignments</span>. <span class=\"ref-journal\"><em>BMC Bioinform.</em></span>, <span class=\"ref-vol\">9</span>, 474. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC2621365/\">PMC free article</a>]</span> [<a href=\"/pubmed/19014431\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Bioinform.&amp;title=RNAalifold:+improved+consensus+structure+prediction+for+RNA+alignments&amp;volume=9&amp;publication_year=2008&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R51\">51. <span class=\"mixed-citation\">\\nReuter J.S., Mathews D.H. (2010) <span class=\"ref-title\">RNAstructure: software for RNA secondary structure prediction and analysis</span>. <span class=\"ref-journal\"><em>BMC Bioinform.</em></span>, <span class=\"ref-vol\">11</span>, 129. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC2984261/\">PMC free article</a>]</span> [<a href=\"/pubmed/20230624\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Bioinform.&amp;title=RNAstructure:+software+for+RNA+secondary+structure+prediction+and+analysis&amp;volume=11&amp;publication_year=2010&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R52\">52. <span class=\"mixed-citation\">\\nWill S., Otto C., Miladi M.. \\net al. (2015) <span class=\"ref-title\">SPARSE: quadratic time simultaneous alignment and folding of RNAs without sequence-based heuristics</span>. <span class=\"ref-journal\"><em>Bioinformatics</em></span>, <span class=\"ref-vol\">31</span>, 2489–2496. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC4514930/\">PMC free article</a>]</span> [<a href=\"/pubmed/25838465\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SPARSE:+quadratic+time+simultaneous+alignment+and+folding+of+RNAs+without+sequence-based+heuristics&amp;volume=31&amp;publication_year=2015&amp;pages=2489-2496&amp;pmid=25838465&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R53\">53. <span class=\"mixed-citation\">\\nTabei Y., Kiryu H., Kin T.. \\net al. (2008) <span class=\"ref-title\">A fast structural multiple alignment method for long RNA sequences</span>. <span class=\"ref-journal\"><em>BMC Bioinform.</em></span>, <span class=\"ref-vol\">9</span>, 33. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC2375124/\">PMC free article</a>]</span> [<a href=\"/pubmed/18215258\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Bioinform.&amp;title=A+fast+structural+multiple+alignment+method+for+long+RNA+sequences&amp;volume=9&amp;publication_year=2008&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R54\">54. <span class=\"mixed-citation\">\\nGlouzon J.-P.S. and Ouangraoua A. (2018) <span class=\"ref-title\">aliFreeFold: an alignment-free approach to predict secondary structure from homologous RNA sequences</span>. <span class=\"ref-journal\"><em>Bioinformatics</em></span>, <span class=\"ref-vol\">34</span>, i70–i78. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC6022685/\">PMC free article</a>]</span> [<a href=\"/pubmed/29949960\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=aliFreeFold:+an+alignment-free+approach+to+predict+secondary+structure+from+homologous+RNA+sequences&amp;volume=34&amp;publication_year=2018&amp;pages=i70-i78&amp;pmid=29949960&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R55\">55. <span class=\"mixed-citation\">\\nWill S., Joshi T., Hofacker I.L.. \\net al. (2012) <span class=\"ref-title\">LocARNA-P: accurate boundary prediction and improved detection of structural RNAs</span>. <span class=\"ref-journal\"><em>RNA</em></span>, <span class=\"ref-vol\">18</span>, 900–914. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC3334699/\">PMC free article</a>]</span> [<a href=\"/pubmed/22450757\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=RNA&amp;title=LocARNA-P:+accurate+boundary+prediction+and+improved+detection+of+structural+RNAs&amp;volume=18&amp;publication_year=2012&amp;pages=900-914&amp;pmid=22450757&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R56\">56. <span class=\"mixed-citation\">\\nJi Y., Xu X., Stormo G.D. (2004) <span class=\"ref-title\">A graph theoretical approach for predicting common RNA secondary structure motifs including pseudoknots in unaligned sequences</span>. <span class=\"ref-journal\"><em>Bioinformatics</em></span>, <span class=\"ref-vol\">20</span>, 1591–1602. [<a href=\"/pubmed/14962926\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A+graph+theoretical+approach+for+predicting+common+RNA+secondary+structure+motifs+including+pseudoknots+in+unaligned+sequences&amp;volume=20&amp;publication_year=2004&amp;pages=1591-1602&amp;pmid=14962926&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R57\">57. <span class=\"mixed-citation\">\\nWu Y., Shi B., Ding X.. \\net al. (2015) <span class=\"ref-title\">Improved prediction of RNA secondary structure by integrating the free energy model with restraints derived from experimental probing data</span>. <span class=\"ref-journal\"><em>Nucleic Acids Res.</em></span>, <span class=\"ref-vol\">43</span>, 7247–7259. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC4551937/\">PMC free article</a>]</span> [<a href=\"/pubmed/26170232\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Improved+prediction+of+RNA+secondary+structure+by+integrating+the+free+energy+model+with+restraints+derived+from+experimental+probing+data&amp;volume=43&amp;publication_year=2015&amp;pages=7247-7259&amp;pmid=26170232&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R58\">58. <span class=\"mixed-citation\">\\nLin Y., Liu T., Cui T.. \\net al. (2020) <span class=\"ref-title\">RNAInter in 2020: RNA interactome repository with increased coverage and annotation</span>. <span class=\"ref-journal\"><em>Nucleic Acids Res.</em></span>, <span class=\"ref-vol\">48</span>, D189–D197. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC6943043/\">PMC free article</a>]</span> [<a href=\"/pubmed/31906603\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=RNAInter+in+2020:+RNA+interactome+repository+with+increased+coverage+and+annotation&amp;volume=48&amp;publication_year=2020&amp;pages=D189-D197&amp;pmid=31906603&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R59\">59. <span class=\"mixed-citation\">\\nGardner P.P., Wilm A., Washietl S. (2005) <span class=\"ref-title\">A benchmark of multiple sequence alignment programs upon structural RNAs</span>. <span class=\"ref-journal\"><em>Nucleic Acids Res.</em></span>, <span class=\"ref-vol\">33</span>, 2433–2439. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC1087786/\">PMC free article</a>]</span> [<a href=\"/pubmed/15860779\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=A+benchmark+of+multiple+sequence+alignment+programs+upon+structural+RNAs&amp;volume=33&amp;publication_year=2005&amp;pages=2433-2439&amp;pmid=15860779&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R60\">60. <span class=\"mixed-citation\">\\nJiang T., Lin G., Ma B.. \\net al. (2002) <span class=\"ref-title\">A general edit distance between RNA structures</span>. <span class=\"ref-journal\"><em>J. Comput. Biol.</em></span>, <span class=\"ref-vol\">9</span>, 371–388. [<a href=\"/pubmed/12015887\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Comput.+Biol.&amp;title=A+general+edit+distance+between+RNA+structures&amp;volume=9&amp;publication_year=2002&amp;pages=371-388&amp;pmid=12015887&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R61\">61. <span class=\"mixed-citation\">\\nYang T.H., Wu W.S. (2012) <span class=\"ref-title\">Identifying biologically interpretable transcription factor knockout targets by jointly analyzing the transcription factor knockout microarray and the ChIP-chip data</span>. <span class=\"ref-journal\"><em>BMC Syst. Biol.</em></span>, <span class=\"ref-vol\">6</span>, 102. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC3465233/\">PMC free article</a>]</span> [<a href=\"/pubmed/22898448\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Syst.+Biol.&amp;title=Identifying+biologically+interpretable+transcription+factor+knockout+targets+by+jointly+analyzing+the+transcription+factor+knockout+microarray+and+the+ChIP-chip+data&amp;volume=6&amp;publication_year=2012&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R62\">62. <span class=\"mixed-citation\">\\nBlin K., Dieterich C., Wurmus R.. \\net al. (2015) <span class=\"ref-title\">DoRiNA 2.0—upgrading the doRiNA database of RNA interactions in post-transcriptional regulation</span>. <span class=\"ref-journal\"><em>Nucleic Acids Res.</em></span>, <span class=\"ref-vol\">43</span>, D160–D167. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC4383974/\">PMC free article</a>]</span> [<a href=\"/pubmed/25416797\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=DoRiNA+2.0—upgrading+the+doRiNA+database+of+RNA+interactions+in+post-transcriptional+regulation&amp;volume=43&amp;publication_year=2015&amp;pages=D160-D167&amp;pmid=25416797&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R63\">63. <span class=\"mixed-citation\">\\nTouriol C., Bornes S., Bonnal S.. \\net al. (2003) <span class=\"ref-title\">Generation of protein isoform diversity by alternative initiation of translation at non-AUG codons</span>. <span class=\"ref-journal\"><em>Biol. Cell</em></span>, <span class=\"ref-vol\">95</span>, 169–178. [<a href=\"/pubmed/12867081\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Biol.+Cell&amp;title=Generation+of+protein+isoform+diversity+by+alternative+initiation+of+translation+at+non-AUG+codons&amp;volume=95&amp;publication_year=2003&amp;pages=169-178&amp;pmid=12867081&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R64\">64. <span class=\"mixed-citation\">\\nLee S., Liu B., Lee S.. \\net al. (2012) <span class=\"ref-title\">Global mapping of translation initiation sites in mammalian cells at single-nucleotide resolution</span>. <span class=\"ref-journal\"><em>Proc. Natl. Acad. Sci. USA</em></span>, <span class=\"ref-vol\">109</span>, E2424–E2432. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC3443142/\">PMC free article</a>]</span> [<a href=\"/pubmed/22927429\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Global+mapping+of+translation+initiation+sites+in+mammalian+cells+at+single-nucleotide+resolution&amp;volume=109&amp;publication_year=2012&amp;pages=E2424-E2432&amp;pmid=22927429&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R65\">65. <span class=\"mixed-citation\">\\nWan J. and Qian S.-B. (2013) <span class=\"ref-title\">TISdb: a database for alternative translation initiation in mammalian cells</span>. <span class=\"ref-journal\"><em>Nucleic Acids Res.</em></span>, <span class=\"ref-vol\">42</span>, D845–D850. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC3965020/\">PMC free article</a>]</span> [<a href=\"/pubmed/24203712\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=TISdb:+a+database+for+alternative+translation+initiation+in+mammalian+cells&amp;volume=42&amp;publication_year=2013&amp;pages=D845-D850&amp;pmid=24203712&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R66\">66. <span class=\"mixed-citation\">\\nLangmead B., Trapnell C., Pop M.. \\net al. (2009) <span class=\"ref-title\">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</span>. <span class=\"ref-journal\"><em>Genome Biol.</em></span>, <span class=\"ref-vol\">10</span>, R25. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC2690996/\">PMC free article</a>]</span> [<a href=\"/pubmed/19261174\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Biol.&amp;title=Ultrafast+and+memory-efficient+alignment+of+short+DNA+sequences+to+the+human+genome&amp;volume=10&amp;publication_year=2009&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R67\">67. <span class=\"mixed-citation\">\\nReuter K., Biehl A., Koch L.. \\net al. (2016) <span class=\"ref-title\">PreTIS: a tool to predict non-canonical 5’UTR translational initiation sites in human and mouse</span>. <span class=\"ref-journal\"><em>PLoS Comput. Biol.</em></span>, <span class=\"ref-vol\">12</span>, e1005170. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC5074520/\">PMC free article</a>]</span> [<a href=\"/pubmed/27768687\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+Comput.+Biol.&amp;title=PreTIS:+a+tool+to+predict+non-canonical+5’UTR+translational+initiation+sites+in+human+and+mouse&amp;volume=12&amp;publication_year=2016&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R68\">68. <span class=\"mixed-citation\">\\nSomers J., Pöyry T. and Willis A.E. (2013) <span class=\"ref-title\">A perspective on mammalian upstream open reading frame function</span>. <span class=\"ref-journal\"><em>Int. J. Biochem. Cell Biol.</em></span>, <span class=\"ref-vol\">45</span>, 1690–1700. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC7172355/\">PMC free article</a>]</span> [<a href=\"/pubmed/23624144\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int.+J.+Biochem.+Cell+Biol.&amp;title=A+perspective+on+mammalian+upstream+open+reading+frame+function&amp;volume=45&amp;publication_year=2013&amp;pages=1690-1700&amp;pmid=23624144&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R69\">69. <span class=\"mixed-citation\">\\nFernandez J., Yaman I., Huang C.. \\net al. (2005) <span class=\"ref-title\">Ribosome stalling regulates IRES-mediated translation in eukaryotes, a parallel to prokaryotic attenuation</span>. <span class=\"ref-journal\"><em>Mol. Cell</em></span>, <span class=\"ref-vol\">17</span>, 405–416. [<a href=\"/pubmed/15694341\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=Ribosome+stalling+regulates+IRES-mediated+translation+in+eukaryotes,+a+parallel+to+prokaryotic+attenuation&amp;volume=17&amp;publication_year=2005&amp;pages=405-416&amp;pmid=15694341&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R70\">70. <span class=\"mixed-citation\">\\nYe Y., Liang Y., Yu Q.. \\net al. (2015) <span class=\"ref-title\">Analysis of human upstream open reading frames and impact on gene expression</span>. <span class=\"ref-journal\"><em>Hum. Genet.</em></span>, <span class=\"ref-vol\">134</span>, 605–612. [<a href=\"/pubmed/25800702\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=Analysis+of+human+upstream+open+reading+frames+and+impact+on+gene+expression&amp;volume=134&amp;publication_year=2015&amp;pages=605-612&amp;pmid=25800702&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R71\">71. <span class=\"mixed-citation\">\\nMokrejš M., Mašek T., Vopálenský V.. \\net al. (2009) <span class=\"ref-title\">IRESite—a tool for the examination of viral and cellular internal ribosome entry sites</span>. <span class=\"ref-journal\"><em>Nucleic Acids Res.</em></span>, <span class=\"ref-vol\">38</span>, D131–D136. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC2808886/\">PMC free article</a>]</span> [<a href=\"/pubmed/19917642\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=IRESite—a+tool+for+the+examination+of+viral+and+cellular+internal+ribosome+entry+sites&amp;volume=38&amp;publication_year=2009&amp;pages=D131-D136&amp;pmid=19917642&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R72\">72. <span class=\"mixed-citation\">\\nColvin J.S., Green R.P., Schmahl J.. \\net al. (2001) <span class=\"ref-title\">Male-to-female sex reversal in mice lacking fibroblast growth factor 9</span>. <span class=\"ref-journal\"><em>Cell</em></span>, <span class=\"ref-vol\">104</span>, 875–889. [<a href=\"/pubmed/11290325\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Male-to-female+sex+reversal+in+mice+lacking+fibroblast+growth+factor+9&amp;volume=104&amp;publication_year=2001&amp;pages=875-889&amp;pmid=11290325&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R73\">73. <span class=\"mixed-citation\">\\nRobinson D., Hasharoni A., Evron Z.. \\net al. (2000) <span class=\"ref-title\">Synovial chondromatosis: the possible role of FGF 9 and FGF receptor 3 in its pathology</span>. <span class=\"ref-journal\"><em>Int. J. Exp. Pathol.</em></span>, <span class=\"ref-vol\">81</span>, 183–189. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC2517723/\">PMC free article</a>]</span> [<a href=\"/pubmed/10971739\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int.+J.+Exp.+Pathol.&amp;title=Synovial+chondromatosis:+the+possible+role+of+FGF+9+and+FGF+receptor+3+in+its+pathology&amp;volume=81&amp;publication_year=2000&amp;pages=183-189&amp;pmid=10971739&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R74\">74. <span class=\"mixed-citation\">\\nArai D., Hegab A.E., Soejima K.. \\net al. (2015) <span class=\"ref-title\">Characterization of the cell of origin and propagation potential of the fibroblast growth factor 9-induced mouse model of lung adenocarcinoma</span>. <span class=\"ref-journal\"><em>J. Pathol.</em></span>, <span class=\"ref-vol\">235</span>, 593–605. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC4329097/\">PMC free article</a>]</span> [<a href=\"/pubmed/25413587\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Pathol.&amp;title=Characterization+of+the+cell+of+origin+and+propagation+potential+of+the+fibroblast+growth+factor+9-induced+mouse+model+of+lung+adenocarcinoma&amp;volume=235&amp;publication_year=2015&amp;pages=593-605&amp;pmid=25413587&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R75\">75. <span class=\"mixed-citation\">\\nColvin J.S., Feldman B., Nadeau J.H.. \\net al. (1999) <span class=\"ref-title\">Genomic organization and embryonic expression of the mouse fibroblast growth factor 9 gene</span>. <span class=\"ref-journal\"><em>Dev. Dyn. An Off. Publ. Am. Assoc. Anat.</em></span>, <span class=\"ref-vol\">216</span>, 72–88. [<a href=\"/pubmed/10474167\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Dev.+Dyn.+An+Off.+Publ.+Am.+Assoc.+Anat.&amp;title=Genomic+organization+and+embryonic+expression+of+the+mouse+fibroblast+growth+factor+9+gene&amp;volume=216&amp;publication_year=1999&amp;pages=72-88&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R76\">76. <span class=\"mixed-citation\">\\nChen T.-M., Hsu C.-H., Tsai S.-J.. \\net al. (2010) <span class=\"ref-title\">AUF1 p42 isoform selectively controls both steady-state and PGE2-induced FGF9 mRNA decay</span>. <span class=\"ref-journal\"><em>Nucleic Acids Res.</em></span>, <span class=\"ref-vol\">38</span>, 8061–8071. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC3001084/\">PMC free article</a>]</span> [<a href=\"/pubmed/20716519\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=AUF1+p42+isoform+selectively+controls+both+steady-state+and+PGE2-induced+FGF9+mRNA+decay&amp;volume=38&amp;publication_year=2010&amp;pages=8061-8071&amp;pmid=20716519&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R77\">77. <span class=\"mixed-citation\">\\nChen W.-M. (2009) <span class=\"ref-journal\"><em>IRES Omnibus: An Integrated Web Server for Predicting and Searching the Human Genes Bearing the IRES Elements</em></span>. National Cheng Kung University, Tainan. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=IRES+Omnibus:+An+Integrated+Web+Server+for+Predicting+and+Searching+the+Human+Genes+Bearing+the+IRES+Elements&amp;publication_year=2009&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R78\">78. <span class=\"mixed-citation\">\\nIdeker T., Galitski T., Hood L. (2001) <span class=\"ref-title\">A new approach to decoding life: systems biology</span>. <span class=\"ref-journal\"><em>Annu. Rev. Genomics Hum. Genet.</em></span>, <span class=\"ref-vol\">2</span>, 343–372. [<a href=\"/pubmed/11701654\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Genomics+Hum.+Genet.&amp;title=A+new+approach+to+decoding+life:+systems+biology&amp;volume=2&amp;publication_year=2001&amp;pages=343-372&amp;pmid=11701654&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"R79\">79. <span class=\"mixed-citation\">\\nHwang D., Rust A.G., Ramsey S.. \\net al. (2005) <span class=\"ref-title\">A data integration methodology for systems biology</span>. <span class=\"ref-journal\"><em>Proc. Natl. Acad. Sci. USA</em></span>, <span class=\"ref-vol\">102</span>, 17296–17301. <span class=\"nowrap\">[<a class=\"int-reflink\" href=\"/pmc/articles/PMC1297682/\">PMC free article</a>]</span> [<a href=\"/pubmed/16301537\" ref=\"reftype=pubmed&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=A+data+integration+methodology+for+systems+biology&amp;volume=102&amp;publication_year=2005&amp;pages=17296-17301&amp;pmid=16301537&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=8094437&amp;issue-id=372510&amp;journal-id=973&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\" role=\"button\" aria-expanded=\"false\" aria-haspopup=\"true\">Google Scholar</a>]</span></span></div></div></div><div style=\"display: none; width: 200px; top: -100px; left: -100px;\" aria-live=\"assertive\" aria-hidden=\"true\" class=\"ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu\"><ul id=\"ui-ncbiinpagenav-2\"><li><a href=\"#idm139637299406304title\">Abstract</a></li><li><a href=\"#s1title\">Background</a></li><li><a href=\"#s2title\">Construction and contents</a></li><li><a href=\"#s3title\">Utility and discussion</a></li><li><a href=\"#s4title\">Conclusions</a></li><li><a href=\"#ack1title\">Acknowledgements</a></li><li><a href=\"#idm139637292208448aff-infotitle\">Contributor Information</a></li><li><a href=\"#s5title\">Funding</a></li><li><a href=\"#s6title\">Conflict of interest.</a></li><li><a href=\"#ref1title\">References</a></li></ul></div></div><!--post-content--><div class=\"courtesy-note whole_rhythm small\"><hr><div class=\"half_rhythm\">Articles from <span class=\"acknowledgment-journal-title\">Database: The Journal of Biological Databases and Curation</span> are provided here courtesy of <strong>Oxford University Press</strong></div><hr></div></div>'"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "content.get_attribute('outerHTML')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7d636a47",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
